# Main Findings
main_findings='The study investigated the role of macrophages in colorectal liver metastasis (CLM), focusing on the distinct roles of SERPINB2-positive monocyte-derived macrophages (MoM f) and GPNMB-positive tumor-associated macrophages (TAMs) in the tumor microenvironment (TME). Through single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics, the study found that high density of SERPINB2-positive MoM f is protective, whereas a high density of GPNMB-positive TAMs is detrimental for disease-free survival (DFS) in CLM patients. This was independent of neoadjuvant systemic chemotherapy and RAS status. The spatial distribution analysis revealed that SERPINB2-positive MoM f are evenly spread out in the invasive margin (IM), while GPNMB-positive TAMs tend to accumulate at the edge of the tumor, potentially exposing them more to tumor immunosuppressive signals. This suggests that the spatial localization of these macrophage populations within the TME is critically linked to their function and impacts patient clinical outcomes. Additionally, the study demonstrated the heterogeneous association of these macrophage populations with patient clinical outcomes, highlighting the complexity of the TME and the need for targeted therapeutic strategies.'
# Computational Methods
computational_methods="Statistical analysis was performed using Stata (version 16) and GraphPad Prism 8. Differences between two groups were calculated by Mannâ\x80\x93Whitney test, and for comparing immune variables from the same patient, paired t-test was used. Statistical significance of signature enrichment score between multiple cell types was determined using one-way ANOVA by Kruskalâ\x80\x93Wallis test. Survival analysis of The Cancer Genome Atlas (TCGA) colon cancer gene expression dataset was performed using GEPIA2. The raw scRNA-seq data have been deposited in Gene Expression Omnibus under the accession numbers GSE200068 and GSE200253. NicheNet method was applied to predict ligandâ\x80\x93receptors interactions. Seurat normalized data was imputed of missed counts with ALRA function. CellPhoneDB was used to analyze cellular interactions between MP in the CD45+ dataset and other CD45+ cell populations. Gene signature enrichment analysis was performed using the Seurat function 'AddModuleScore'. The analysis of regulon activity was performed with SCENIC R package. Single-cell pseudotime trajectory was generated using Monocle version 2.8."
# Experimental Methods
experimental_methods='Spatial transcriptomics, scRNA-seq, Flow cytometry staining and cell sorting, NicheNet interaction analysis, CellPhoneDB interaction analysis, Gene signature enrichment analysis, Gene regulatory network analysis'
# Samples
samples='GSE200068, GSE200253'
