import os
from enum import Enum
import inspect
from schema_agents.provider.openai_api import retry
import asyncio
from paperqa import Docs
from typing import List, Optional, Union, Type, Any, get_type_hints, Tuple, Literal
from pydantic import BaseModel, Field, validator, create_model
from bioimageio_chatbot.utils import ChatbotExtension
import asyncio
from schema_agents.role import Message
from schema_agents import schema_tool, Role
import time
import urllib.request

def call_api(url):
    time.sleep(1)
    url = url.replace(' ', '+')
    print(url)

    req = urllib.request.Request(url) 
    with urllib.request.urlopen(req) as response:
        call = response.read()

    # return call.decode()
    return call

@schema_tool
async def get_geo_api_info() -> str:
    """Returns details about the usage of the NCBI Web API for usage of the GEO database"""

    s = """Use the NCBI Web APIs to explore and analyze the GEO database which is a public repository for high-throughput gene expression data and other functional genomics datasets
You can use Eutils by: "[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/{esearch|efetch|esummary}.fcgi?db={gene|snp|omim}&retmax={}&{term|id}={term|id}]".
esearch: input is a search term and output is database id(s).
efectch/esummary: input is database id(s) and output is full records or summaries that contain name, chromosome location, and other information.
Normally, you need to first call esearch to get the database id(s) of the search term, and then call efetch/esummary to get the information with the database id(s).
Database: gene is for genes, snp is for SNPs, and omim is for genetic diseases.

Here are some examples:

(1) Searching for GEO Datasets
Question: What are the GEO datasets (IDs) related to breast cancer?
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=gds&term=breast+cancer&retmax=10] -> [b'{"header":{"type":"esearch","version":"0.3"},"esearchresult":{"count":"176858","retmax":"10","retstart":"0","idlist":["200168121","200249972","200249971","200249969","200249967","200217204","200202309","200202307","200202303","200197901"],"translationset":[{"from":"breast cancer","to":"\\"breast neoplasms\\"[MeSH Terms] OR breast cancer[All Fields]"}],"translationstack":[{"term":"\\"breast neoplasms\\"[MeSH Terms]","field":"MeSH Terms","count":"176858","explode":"Y"},{"term":"breast cancer[All Fields]","field":"All Fields","count":"126335","explode":"N"},"OR","GROUP"],"querytranslation":"\\"breast neoplasms\\"[MeSH Terms] OR breast cancer[All Fields]"}}\n']
Answer: 200168121,200249972,200249971,200249969,200249967,200217204,200202309,200202307,200202303,200197901

(2) Retrieving Dataset Details
Question: What are the details of the GEO dataset with ID 200?
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=gds&id=200&retmode=json] -> [b'{"header":{"type":"esummary","version":"0.3"},"result":{"uids":["200"],"200":{"uid":"200","accession":"GDS200","gds":"200","title":"Contextual fear conditioning paradigm","summary":"Mice subject to novel spatial context with 0.5 mA shock, shock alone or control. Alzheimer\'s mice (50% C57BL/6, 50% SJL) and MK-801-injected mice also examined. Dentate gyrus and CA1 hippocampus tissue examined 1, 2, 4 or 6 hours after treatment.","gpl":"81","gse":"1782","taxon":"Mus musculus","entrytype":"GDS","gdstype":"Expression profiling by array","ptechtype":"","valtype":"count","ssinfo":"protocol;protocol;protocol;tissue;tissue;time;time;time;time;strain;strain;strain","subsetinfo":"3 protocol, 3 strain, 4 time, 2 tissue","pdat":"2005/03/30","suppfile":"","samples":[{"accession":"GSM4549","title":"1hr S-CA1"},{"accession":"GSM4550","title":"2hr S-CA1"},{"accession":"GSM4565","title":"MK 2hr S-CA1"},{"accession":"GSM4551","title":"4hr S-CA1"},{"accession":"GSM4553","title":"6hr S-CA1"},{"accession":"GSM4554","title":"1hr S-DG"},{"accession":"GSM4555","title":"2hr S-DG"},{"accession":"GSM4567","title":"MK 2hr S-DG"},{"accession":"GSM4556","title":"4hr S-DG"},{"accession":"GSM4557","title":"6hr S-DG"},{"accession":"GSM4519","title":"1st 1hr C+S-CA1"},{"accession":"GSM4525","title":"2nd 1hr C+S-CA1"},{"accession":"GSM4529","title":"3rd 1hr C+S-CA1"},{"accession":"GSM4521","title":"1st 2hr C+S-CA1"},{"accession":"GSM4526","title":"2nd 2hr C+S-CA1"},{"accession":"GSM4531","title":"3rd 2hr C+S-CA1"},{"accession":"GSM4560","title":"AD 2hr C+S-CA1"},{"accession":"GSM4522","title":"1st 4hr C+S-CA1"},{"accession":"GSM4527","title":"2nd 4hr C+S-CA1"},{"accession":"GSM4532","title":"3rd 4hr C+S-CA1"},{"accession":"GSM4523","title":"1st 6hr C+S-CA1"},{"accession":"GSM4528","title":"2nd 6hr C+S-CA1"},{"accession":"GSM4533","title":"3rd 6hr C+S-CA1"},{"accession":"GSM4535","title":"1st 1hr C+S-DG"},{"accession":"GSM4539","title":"2nd 1hr C+S-DG"},{"accession":"GSM4544","title":"3rd 1hr C+S-DG"},{"accession":"GSM4536","title":"1st 2hr C+S-DG"},{"accession":"GSM4540","title":"2nd 2hr C+S-DG"},{"accession":"GSM4545","title":"3rd 2hr C+S-DG"},{"accession":"GSM4562","title":"AD 2hr C+S-DG"},{"accession":"GSM4537","title":"1st 4hr C+S-DG"},{"accession":"GSM4542","title":"2nd 4hr C+S-DG"},{"accession":"GSM4546","title":"3rd 4hr C+S-DG"},{"accession":"GSM4538","title":"1st 6hr C+S-DG"},{"accession":"GSM4543","title":"2nd 6hr C+S-DG"},{"accession":"GSM4548","title":"3rd 6hr C+S-DG"},{"accession":"GSM4493","title":"1st 1hr C-CA1"},{"accession":"GSM4497","title":"2nd 1hr C-CA1"},{"accession":"GSM4501","title":"3rd 1hr C-CA1"},{"accession":"GSM4494","title":"1st 2hr C-CA1"},{"accession":"GSM4498","title":"2nd 2hr C-CA1"},{"accession":"GSM4502","title":"3rd 2hr C-CA1"},{"accession":"GSM4558","title":"AD 2hr C-CA1"},{"accession":"GSM4563","title":"MK 2hr C-CA1"},{"accession":"GSM4495","title":"1st 4hr C-CA1"},{"accession":"GSM4499","title":"2nd 4hr C-CA1"},{"accession":"GSM4503","title":"3rd 4hr C-CA1"},{"accession":"GSM4496","title":"1st 6hr C-CA1"},{"accession":"GSM4500","title":"2nd 6hr C-CA1"},{"accession":"GSM4504","title":"3rd 6hr C-CA1"},{"accession":"GSM4505","title":"1st 1hr C-DG"},{"accession":"GSM4510","title":"2nd 1hr C-DG"},{"accession":"GSM4514","title":"3rd 1hr C-DG"},{"accession":"GSM4506","title":"1st 2hr C-DG"},{"accession":"GSM4511","title":"2nd 2hr C-DG"},{"accession":"GSM4516","title":"3rd 2hr C-DG"},{"accession":"GSM4561","title":"AD 2hr C-DG"},{"accession":"GSM4566","title":"MK 2hr C-DG"},{"accession":"GSM4507","title":"1st 4hr C-DG"},{"accession":"GSM4512","title":"2nd 4hr C-DG"},{"accession":"GSM4517","title":"3rd 4hr C-DG"},{"accession":"GSM4509","title":"1st 6hr C-DG"},{"accession":"GSM4513","title":"2nd 6hr C-DG"},{"accession":"GSM4518","title":"3rd 6hr C-DG"}],"relations":[],"extrelations":[],"n_samples":64,"seriestitle":"A Bioinformatics Analysis of Memory Consolidation Reveals Involvement of the Transcription Factor c-Rel","platformtitle":"[MG_U74Av2] Affymetrix Murine Genome U74A Version 2 Array","platformtaxa":"Mus musculus","samplestaxa":"Mus musculus","pubmedids":["15102909"],"projects":[],"ftplink":"ftp://ftp.ncbi.nlm.nih.gov/geo/datasets/GDSnnn/GDS200/","geo2r":"","bioproject":"PRJNA90577"}}}\n']
Answer: The entire dataset details are contained in the json object returned from the query. For example, the dataset files can be downloaded via the ftp link 'ftp://ftp.ncbi.nlm.nih.gov/geo/datasets/GDSnnn/GDS200/'
"""

@schema_tool
async def get_genomic_api_info() -> str:
    """Returns details about the usage of the NCBI Web API for genomic questions"""

    s = """Use the NCBI Web APIs to answer genomic questions.
You can call Eutils by: "[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/{esearch|efetch|esummary}.fcgi?db={gene|snp|omim}&retmax={}&{term|id}={term|id}]".
esearch: input is a search term and output is database id(s).
efectch/esummary: input is database id(s) and output is full records or summaries that contain name, chromosome location, and other information.
Normally, you need to first call esearch to get the database id(s) of the search term, and then call efetch/esummary to get the information with the database id(s).
Database: gene is for genes, snp is for SNPs, and omim is for genetic diseases.

For DNA sequences, you can use BLAST by: "[https://blast.ncbi.nlm.nih.gov/blast/Blast.cgi?CMD={Put|Get}&PROGRAM=blastn&MEGABLAST=on&DATABASE=nt&FORMAT_TYPE={XML|Text}&QUERY={sequence}&HITLIST_SIZE={max_hit_size}]".
BLAST maps a specific DNA {sequence} to its chromosome location among different species.
You need to first PUT the BLAST request and then GET the results using the RID returned by PUT.

Here are some examples:

Question: What is the official gene symbol of LMP10?
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=gene&retmax=5&retmode=json&sort=relevance&term=LMP10]->[b'{"header":{"type":"esearch","version":"0.3"},"esearchresult":{"count":"3","retmax":"3","retstart":"0","idlist":["5699","8138","19171"],"translationset":[],"translationstack":[{"term":"LMP10[All Fields]","field":"All Fields","count":"3","explode":"N"},"GROUP"],"querytranslation":"LMP10[All Fields]"}}\n']
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=gene&retmax=5&retmode=json&id=19171,5699,8138]->[b'\n1. Psmb10\nOfficial Symbol: Psmb10 and Name: proteasome (prosome, macropain) subunit, beta type 10 [Mus musculus (house mouse)]\nOther Aliases: Mecl-1, Mecl1\nOther Designations: proteasome subunit beta type-10; low molecular mass protein 10; macropain subunit MECl-1; multicatalytic endopeptidase complex subunit MECl-1; prosome Mecl1; proteasome (prosomome, macropain) subunit, beta type 10; proteasome MECl-1; proteasome subunit MECL1; proteasome subunit beta-2i\nChromosome: 8; Location: 8 53.06 cM\nAnnotation: Chromosome 8 NC_000074.7 (106662360..106665024, complement)\nID: 19171\n\n2. PSMB10\nOfficial Symbol: PSMB10 and Name: proteasome 20S subunit beta 10 [Homo sapiens (human)]\nOther Aliases: LMP10, MECL1, PRAAS5, beta2i\nOther Designations: proteasome subunit beta type-10; low molecular mass protein 10; macropain subunit MECl-1; multicatalytic endopeptidase complex subunit MECl-1; proteasome (prosome, macropain) subunit, beta type, 10; proteasome MECl-1; proteasome catalytic subunit 2i; proteasome subunit MECL1; proteasome subunit beta 10; proteasome subunit beta 7i; proteasome subunit beta-2i; proteasome subunit beta2i\nChromosome: 16; Location: 16q22.1\nAnnotation: Chromosome 16 NC_000016.10 (67934506..67936850, complement)\nMIM: 176847\nID: 5699\n\n3. MECL1\nProteosome subunit MECL1 [Homo sapiens (human)]\nOther Aliases: LMP10, PSMB10\nThis record was replaced with GeneID: 5699\nID: 8138\n\n']
Answer: PSMB10

Question: Which gene is SNP rs1217074595 associated with?
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=snp&retmax=10&retmode=json&id=1217074595]->[b'{"header":{"type":"esummary","version":"0.3"},"result":{"uids":["1217074595"],"1217074595":{"uid":"1217074595","snp_id":1217074595,"allele_origin":"","global_mafs":[{"study":"GnomAD","freq":"A=0.000007/1"},{"study":"TOPMED","freq":"A=0.000004/1"},{"study":"ALFA","freq":"A=0./0"}],"global_population":"","global_samplesize":"","suspected":"","clinical_significance":"","genes":[{"name":"LINC01270","gene_id":"284751"}],"acc":"NC_000020.11","chr":"20","handle":"GNOMAD,TOPMED","spdi":"NC_000020.11:50298394:G:A","fxn_class":"non_coding_transcript_variant","validated":"by-frequency,by-alfa,by-cluster","docsum":"HGVS=NC_000020.11:g.50298395G>A,NC_000020.10:g.48914932G>A,NR_034124.1:n.351G>A,NM_001025463.1:c.*4G>A|SEQ=[G/A]|LEN=1|GENE=LINC01270:284751","tax_id":9606,"orig_build":155,"upd_build":156,"createdate":"2017/11/09 09:55","updatedate":"2022/10/13 17:11","ss":"4354715686,5091242333","allele":"R","snp_class":"snv","chrpos":"20:50298395","chrpos_prev_assm":"20:48914932","text":"","snp_id_sort":"1217074595","clinical_sort":"0","cited_sort":"","chrpos_sort":"0050298395","merged_sort":"0"}}}\n']
Answer: LINC01270

Question: What are genes related to Meesmann corneal dystrophy?
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=omim&retmax=20&retmode=json&sort=relevance&term=Meesmann+corneal+dystrophy]->[b'{"header":{"type":"esearch","version":"0.3"},"esearchresult":{"count":"5","retmax":"5","retstart":"0","idlist":["122100","618767","300778","601687","148043"],"translationset":[],"translationstack":[{"term":"Meesmann[All Fields]","field":"All Fields","count":"5","explode":"N"},{"term":"corneal[All Fields]","field":"All Fields","count":"565","explode":"N"},"AND",{"term":"dystrophy[All Fields]","field":"All Fields","count":"1355","explode":"N"},"AND","GROUP"],"querytranslation":"Meesmann[All Fields] AND corneal[All Fields] AND dystrophy[All Fields]"}}\n']
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=omim&retmax=20&retmode=json&id=618767,601687,300778,148043,122100]->[b'{"header":{"type":"esummary","version":"0.3"},"result":{"uids":["618767","601687","300778","148043","122100"],"618767":{"uid":"618767","oid":"#618767","title":"CORNEAL DYSTROPHY, MEESMANN, 2; MECD2","alttitles":"","locus":"602767"},"601687":{"uid":"601687","oid":"*601687","title":"KERATIN 12, TYPE I; KRT12","alttitles":"","locus":""},"300778":{"uid":"300778","oid":"%300778","title":"CORNEAL DYSTROPHY, LISCH EPITHELIAL; LECD","alttitles":"","locus":"#6####"},"148043":{"uid":"148043","oid":"*148043","title":"KERATIN 3, TYPE II; KRT3","alttitles":"","locus":"mutation"},"122100":{"uid":"122100","oid":"#122100","title":"CORNEAL DYSTROPHY, MEESMANN, 1; MECD1","alttitles":"","locus":"P#####"}}}\n']
Answer: KRT12, KRT3

Question: Align the DNA sequence to the human genome:ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACCCTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGAAAAAAAAAAAAGTATTTCTCT
[https://blast.ncbi.nlm.nih.gov/blast/Blast.cgi?CMD=Put&PROGRAM=blastn&MEGABLAST=on&DATABASE=nt&FORMAT_TYPE=XML&QUERY=ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACCCTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGAAAAAAAAAAAAGTATTTCTCT&HITLIST_SIZE=5]->[YBWG5AYT013]
[https://blast.ncbi.nlm.nih.gov/blast/Blast.cgi?CMD=Get&FORMAT_TYPE=Text&RID=YBWG5AYT013]->[b'<p><!--\nQBlastInfoBegin\n\tStatus=READY\nQBlastInfoEnd\n--><p>\n<PRE>\nBLASTN 2.15.0+\nReference: Zheng Zhang, Scott Schwartz, Lukas Wagner, and\nWebb Miller (2000), "A greedy algorithm for aligning DNA\nsequences", J Comput Biol 2000; 7(1-2):203-14.\n\n\nReference for database indexing: Aleksandr Morgulis, George\nCoulouris, Yan Raytselis, Thomas L. Madden, Richa Agarwala,\nAlejandro A. Schaffer (2008), "Database Indexing for\nProduction MegaBLAST Searches", Bioinformatics 24:1757-1764.\n\n\nRID: YBWG5AYT013\n\n\nDatabase: Nucleotide collection (nt)\n           103,965,835 sequences; 1,574,905,710,618 total letters\nQuery= \nLength=128\n\n\n                                                                   Score     E     Max\nSequences producing significant alignments:                       (Bits)  Value  Ident\n\nCP034493.1 Eukaryotic synthetic construct chromosome 15            237     5e-58  100%      \nCP139551.1 Homo sapiens isolate NA24385 chromosome 15              237     5e-58  100%      \nNG_132175.1 Homo sapiens H3K27ac-H3K4me1 hESC enhancer GRCh37_ 237     5e-58  100%      \nCP068263.2 Homo sapiens isolate CHM13 chromosome 15                237     5e-58  100%      \nAP023475.1 Homo sapiens DNA, chromosome 15, nearly complete ge 237     5e-58  100%      \n\nALIGNMENTS\n>CP034493.1 Eukaryotic synthetic construct chromosome 15\n CP034518.1 Eukaryotic synthetic construct chromosome 15\nLength=82521392\n\n Score = 237 bits (128),  Expect = 5e-58\n Identities = 128/128 (100%), Gaps = 0/128 (0%)\n Strand=Plus/Plus\n\nQuery  1         ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  60\n                 ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  72494035  ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  72494094\n\nQuery  61        CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGaaaaaaaaaaaaGT  120\n                 ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  72494095  CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGAAAAAAAAAAAAGT  72494154\n\nQuery  121       ATTTCTCT  128\n                 ||||||||\nSbjct  72494155  ATTTCTCT  72494162\n\n\n>CP139551.1 Homo sapiens isolate NA24385 chromosome 15\nLength=96257017\n\n Score = 237 bits (128),  Expect = 5e-58\n Identities = 128/128 (100%), Gaps = 0/128 (0%)\n Strand=Plus/Plus\n\nQuery  1         ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  60\n                 ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  86191593  ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  86191652\n\nQuery  61        CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGaaaaaaaaaaaaGT  120\n                 ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  86191653  CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGAAAAAAAAAAAAGT  86191712\n\nQuery  121       ATTTCTCT  128\n                 ||||||||\nSbjct  86191713  ATTTCTCT  86191720\n\n\n>NG_132175.1 Homo sapiens H3K27ac-H3K4me1 hESC enhancer GRCh37_chr15:92493309-92494181 \n(LOC127830695) on chromosome 15\nLength=1073\n\n Score = 237 bits (128),  Expect = 5e-58\n Identities = 128/128 (100%), Gaps = 0/128 (0%)\n Strand=Plus/Plus\n\nQuery  1    ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  60\n            ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  827  ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  886\n\nQuery  61   CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGaaaaaaaaaaaaGT  120\n            ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  887  CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGAAAAAAAAAAAAGT  946\n\nQuery  121  ATTTCTCT  128\n            ||||||||\nSbjct  947  ATTTCTCT  954\n\n\n>CP068263.2 Homo sapiens isolate CHM13 chromosome 15\nLength=99753195\n\n Score = 237 bits (128),  Expect = 5e-58\n Identities = 128/128 (100%), Gaps = 0/128 (0%)\n Strand=Plus/Plus\n\nQuery  1         ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  60\n                 ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  89712558  ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  89712617\n\nQuery  61        CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGaaaaaaaaaaaaGT  120\n                 ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  89712618  CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGAAAAAAAAAAAAGT  89712677\n\nQuery  121       ATTTCTCT  128\n                 ||||||||\nSbjct  89712678  ATTTCTCT  89712685\n\n\n>AP023475.1 Homo sapiens DNA, chromosome 15, nearly complete genome\nLength=95537968\n\n Score = 237 bits (128),  Expect = 5e-58\n Identities = 128/128 (100%), Gaps = 0/128 (0%)\n Strand=Plus/Plus\n\nQuery  1         ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  60\n                 ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  85572367  ATTCTGCCTTTAGTAATTTGATGACAGAGACTTCTTGGGAACCACAGCCAGGGAGCCACC  85572426\n\nQuery  61        CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGaaaaaaaaaaaaGT  120\n                 ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||\nSbjct  85572427  CTTTACTCCACCAACAGGTGGCTTATATCCAATCTGAGAAAGAAAGAAAAAAAAAAAAGT  85572486\n\nQuery  121       ATTTCTCT  128\n                 ||||||||\nSbjct  85572487  ATTTCTCT  85572494\n\n\n  Database: Nucleotide collection (nt)\n    Posted date:  Mar 1, 2024  12:40 PM\n  Number of letters in database: 1,574,905,710,618\n  Number of sequences in database:  103,965,835\n\nLambda      K        H\n    1.33    0.621     1.12 \nGapped\nLambda      K        H\n    1.28    0.460    0.850 \nMatrix: blastn matrix:1 -2\nGap Penalties: Existence: 0, Extension: 0\nNumber of Sequences: 103965835\nNumber of Hits to DB: 0\nNumber of extensions: 0\nNumber of successful extensions: 0\nNumber of sequences better than 10: 1\nNumber of HSP\'s better than 10 without gapping: 0\nNumber of HSP\'s gapped: 1\nNumber of HSP\'s successfully gapped: 1\nLength of query: 128\nLength of database: 1574905710618\nLength adjustment: 35\nEffective length of query: 93\nEffective length of database: 1571266906393\nEffective search space: 146127822294549\nEffective search space used: 146127822294549\nA: 0\nX1: 13 (25.0 bits)\nX2: 32 (59.1 bits)\nX3: 54 (99.7 bits)\nS1: 13 (25.1 bits)\nS2: 24 (45.4 bits)\n\n\n']
Answer: chr15:91950805-91950932"""

    return s

@schema_tool
async def get_pubmed_api_info() -> str:
    """Returns details about the usage of the NCBI Web API for usage of the PubMed Central (PMC) database. PubMed Central is a free digital repository that offers access to articles on life sciences and biomedical topics. The API offers functionality including searching the PMC database, retrieving article details, and downloading article content."""

    s = """Use the NCBI Web APIs to explore and analyze the PubMed Central (PMC) database which is a a free digital repository that offers access to articles on life sciences and biomedical topics. The API offers functionality including searching the PMC database, retrieving article details, and downloading article content.
You can use Eutils by: "[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/{esearch|efetch|esummary}.fcgi?db={gene|snp|omim}&retmax={}&{term|id}={term|id}]".
esearch: input is a search term and output is database id(s).
efectch/esummary: input is database id(s) and output is full records or summaries that contain name, chromosome location, and other information.
Normally, you need to first call esearch to get the database id(s) of the search term, and then call efetch/esummary to get the information with the database id(s).
Database: gene is for genes, snp is for SNPs, and omim is for genetic diseases.

Here are some examples:

Here are some examples:

(1) Searching the PMC Database
Query: Search for article IDs for PMC articles on COVID-19 published in 2020
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pmc&term=COVID-19+AND+2020[pdat]&retmax=10&retmode=json] -> [b'{"header":{"type":"esearch","version":"0.3"},"esearchresult":{"count":"105731","retmax":"10","retstart":"0","idlist":["7615674","10895950","10846889","10846884","10905971","7163933","10868656","10846888","10838546","10824537"],"translationset":[{"from":"COVID-19","to":"\\"COVID-19\\"[All Fields] OR \\"COVID-19\\"[MeSH Terms] OR \\"COVID-19 Vaccines\\"[All Fields] OR \\"COVID-19 Vaccines\\"[MeSH Terms] OR \\"COVID-19 serotherapy\\"[All Fields] OR \\"COVID-19 Nucleic Acid Testing\\"[All Fields] OR \\"covid-19 nucleic acid testing\\"[MeSH Terms] OR \\"COVID-19 Serological Testing\\"[All Fields] OR \\"covid-19 serological testing\\"[MeSH Terms] OR \\"COVID-19 Testing\\"[All Fields] OR \\"covid-19 testing\\"[MeSH Terms] OR \\"SARS-CoV-2\\"[All Fields] OR \\"sars-cov-2\\"[MeSH Terms] OR \\"Severe Acute Respiratory Syndrome Coronavirus 2\\"[All Fields] OR \\"NCOV\\"[All Fields] OR \\"2019 NCOV\\"[All Fields] OR ((\\"coronavirus\\"[MeSH Terms] OR \\"coronavirus\\"[All Fields] OR \\"COV\\"[All Fields]) AND 2019/11/01[PubDate] : 3000/12/31[PubDate])"}],"translationstack":[{"term":"\\"COVID-19\\"[All Fields]","field":"All Fields","count":"625718","explode":"N"},{"term":"\\"COVID-19\\"[MeSH Terms]","field":"MeSH Terms","count":"163462","explode":"Y"},"OR",{"term":"\\"COVID-19 Vaccines\\"[All Fields]","field":"All Fields","count":"46422","explode":"N"},"OR",{"term":"\\"COVID-19 Vaccines\\"[MeSH Terms]","field":"MeSH Terms","count":"15681","explode":"Y"},"OR",{"term":"\\"COVID-19 serotherapy\\"[All Fields]","field":"All Fields","count":"654","explode":"N"},"OR",{"term":"\\"COVID-19 Nucleic Acid Testing\\"[All Fields]","field":"All Fields","count":"1510","explode":"N"},"OR",{"term":"\\"covid-19 nucleic acid testing\\"[MeSH Terms]","field":"MeSH Terms","count":"1333","explode":"Y"},"OR",{"term":"\\"COVID-19 Serological Testing\\"[All Fields]","field":"All Fields","count":"1065","explode":"N"},"OR",{"term":"\\"covid-19 serological testing\\"[MeSH Terms]","field":"MeSH Terms","count":"905","explode":"Y"},"OR",{"term":"\\"COVID-19 Testing\\"[All Fields]","field":"All Fields","count":"22826","explode":"N"},"OR",{"term":"\\"covid-19 testing\\"[MeSH Terms]","field":"MeSH Terms","count":"8595","explode":"Y"},"OR",{"term":"\\"SARS-CoV-2\\"[All Fields]","field":"All Fields","count":"315737","explode":"N"},"OR",{"term":"\\"sars-cov-2\\"[MeSH Terms]","field":"MeSH Terms","count":"115444","explode":"Y"},"OR",{"term":"\\"Severe Acute Respiratory Syndrome Coronavirus 2\\"[All Fields]","field":"All Fields","count":"120943","explode":"N"},"OR",{"term":"\\"NCOV\\"[All Fields]","field":"All Fields","count":"68377","explode":"N"},"OR",{"term":"\\"2019 NCOV\\"[All Fields]","field":"All Fields","count":"53805","explode":"N"},"OR",{"term":"\\"coronavirus\\"[MeSH Terms]","field":"MeSH Terms","count":"123002","explode":"Y"},{"term":"\\"coronavirus\\"[All Fields]","field":"All Fields","count":"386938","explode":"N"},"OR",{"term":"\\"COV\\"[All Fields]","field":"All Fields","count":"331919","explode":"N"},"OR","GROUP",{"term":"2019/11/01[PubDate]","field":"PubDate","count":"0","explode":"N"},{"term":"3000/12/31[PubDate]","field":"PubDate","count":"0","explode":"N"},"RANGE","AND","GROUP","OR","GROUP",{"term":"2020[pdat]","field":"pdat","count":"747900","explode":"N"},"AND"],"querytranslation":"(\\"COVID-19\\"[All Fields] OR \\"COVID-19\\"[MeSH Terms] OR \\"COVID-19 Vaccines\\"[All Fields] OR \\"COVID-19 Vaccines\\"[MeSH Terms] OR \\"COVID-19 serotherapy\\"[All Fields] OR \\"COVID-19 Nucleic Acid Testing\\"[All Fields] OR \\"covid-19 nucleic acid testing\\"[MeSH Terms] OR \\"COVID-19 Serological Testing\\"[All Fields] OR \\"covid-19 serological testing\\"[MeSH Terms] OR \\"COVID-19 Testing\\"[All Fields] OR \\"covid-19 testing\\"[MeSH Terms] OR \\"SARS-CoV-2\\"[All Fields] OR \\"sars-cov-2\\"[MeSH Terms] OR \\"Severe Acute Respiratory Syndrome Coronavirus 2\\"[All Fields] OR \\"NCOV\\"[All Fields] OR \\"2019 NCOV\\"[All Fields] OR ((\\"coronavirus\\"[MeSH Terms] OR \\"coronavirus\\"[All Fields] OR \\"COV\\"[All Fields]) AND 2019/11/01[PubDate] : 3000/12/31[PubDate])) AND 2020[pdat]"}}\n']
Answer: 7615674, 10895950, 10846889, 10846884, 10905971, 7163933, 10868656, 10846888, 10838546, 10824537

(2) Fetch Article Details
Query: Fetch the details of the Pubmed Central article with the ID 7615674
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7615674&retmode=json] -> [b'<?xml version="1.0" ?>\n<!DOCTYPE pmc-articleset PUBLIC "-//NLM//DTD ARTICLE SET 2.0//EN" "https://dtd.nlm.nih.gov/ncbi/pmc/articleset/nlm-articleset-2.0.dtd">\n<pmc-articleset><article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" dtd-version="1.3" xml:lang="zh" article-type="research-article">\n  <!--The publisher of this article does not allow downloading of the full text in XML form.-->\n  <?properties manuscript?>\n  <processing-meta base-tagset="archiving" mathml-version="3.0" table-model="xhtml" tagset-family="jats">\n    <restricted-by>pmc</restricted-by>\n  </processing-meta>\n  <front>\n    <journal-meta>\n      <journal-id journal-id-type="nlm-journal-id">9918733285406676</journal-id>\n      <journal-id journal-id-type="nlm-ta">Feng Jing Yuan Lin</journal-id>\n      <journal-id journal-id-type="iso-abbrev">Feng Jing Yuan Lin</journal-id>\n      <journal-title-group>\n        <journal-title>Feng jing yuan lin = Landscape architecture</journal-title>\n      </journal-title-group>\n      <issn pub-type="epub">1673-1530</issn>\n    </journal-meta>\n    <article-meta>\n      <article-id pub-id-type="pmid">38420146</article-id>\n      <article-id pub-id-type="pmc">7615674</article-id>\n      <article-id pub-id-type="manuscript">EMS193589</article-id>\n      <article-id pub-id-type="doi">10.14085/j.fjyl.2020.11.0096.08</article-id>\n      <article-categories>\n        <subj-group subj-group-type="heading">\n          <subject>Article</subject>\n        </subj-group>\n      </article-categories>\n      <title-group>\n        <article-title>&#x5065;&#x5EB7;&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x8BC4;&#x4EF7;&#x4F53;&#x7CFB;&#x6784;&#x5EFA;&#x63A2;&#x6790;&#x2014;&#x57FA;&#x4E8E;&#x57CE;&#x5E02;&#x89C4;&#x5212;&#x4E0E;&#x516C;&#x5171;&#x5065;&#x5EB7;&#x7684;&#x7ED3;&#x5408;&#x89C6;&#x89D2;</article-title>\n        <trans-title-group xml:lang="en">\n          <trans-title>Research on Healthy Neighborhood Evaluation System Based on the Combined Perspectives of Urban Planning and Public Health</trans-title>\n        </trans-title-group>\n      </title-group>\n      <contrib-group>\n        <contrib contrib-type="author">\n          <name>\n            <surname>&#x96E8;&#x6D0B;</surname>\n            <given-names>&#x5F20;</given-names>\n          </name>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>&#x5B81;&#x777F;</surname>\n            <given-names>&#x5218;</given-names>\n          </name>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>&#x701B;</surname>\n            <given-names>&#x9F99;</given-names>\n          </name>\n        </contrib>\n      </contrib-group>\n      <contrib-group>\n        <contrib contrib-type="author">\n          <name xml:lang="en">\n            <surname>Yuyang</surname>\n            <given-names>Zhang</given-names>\n          </name>\n        </contrib>\n        <contrib contrib-type="author">\n          <name xml:lang="en">\n            <surname>Ningrui</surname>\n            <given-names>Liu</given-names>\n          </name>\n        </contrib>\n        <contrib contrib-type="author">\n          <name xml:lang="en">\n            <surname>Ying</surname>\n            <given-names>Long</given-names>\n          </name>\n        </contrib>\n      </contrib-group>\n      <aff id="A1">&#x5F20;&#x96E8;&#x6D0B; / &#x7537; / &#x6E05;&#x534E;&#x5927;&#x5B66;&#x5EFA;&#x7B51;&#x5B66;&#x9662;&#x5728;&#x7AD9;&#x535A;&#x58EB;&#x540E; / &#x7814;&#x7A76;&#x65B9;&#x5411;&#x4E3A;&#x5065;&#x5EB7;&#x57CE;&#x5E02;&#x4E0E;&#x5065;&#x5EB7;&#x793E;&#x533A;</aff>\n      <aff xml:lang="en" id="A2">ZHANG Yuyang is a postdoctoral researcher in the School of Architecture, Tsinghua University. His research focuses on healthy city and healthy neighborhood</aff>\n      <aff id="A3">&#x5218;&#x5B81;&#x777F; / &#x7537; / &#x6E05;&#x534E;&#x5927;&#x5B66;&#x5EFA;&#x7B51;&#x5B66;&#x9662;&#x5EFA;&#x7B51;&#x6280;&#x672F;&#x79D1; &#x5B66;&#x7CFB;&#x5728;&#x8BFB;&#x535A;&#x58EB;&#x7814;&#x7A76;&#x751F; / &#x7814;&#x7A76;&#x65B9;&#x5411;&#x4E3A;&#x5BA4;&#x5185;&#x7A7A;&#x6C14; &#x54C1;&#x8D28;&#x3001;&#x5065;&#x5EB7;&#x5EFA;&#x7B51;&#x4E0E;&#x5065;&#x5EB7;&#x57CE;&#x5E02;</aff>\n      <aff xml:lang="en" id="A4">LIU Ningrui is a Ph.D. candidate in the Department of Building Science, School of Architecture, Tsinghua University. His research focuses on indoor air quality, healthy building and healthy city</aff>\n      <aff id="A5">&#x9F99;&#x701B; / &#x7537; / &#x535A;&#x58EB; / &#x6E05;&#x534E;&#x5927;&#x5B66;&#x5EFA;&#x7B51;&#x5B66;&#x9662;&#x7814;&#x7A76;&#x5458; / &#x7814;&#x7A76;&#x65B9;&#x5411;&#x4E3A;&#x57CE;&#x4E61;&#x89C4;&#x5212;&#x6280;&#x672F;&#x79D1;&#x5B66; &#x901A;&#x4FE1;&#x4F5C;&#x8005;&#x90AE;&#x7BB1;&#xFF08;Corresponding author Email&#xFF09;&#xFF1A;</aff>\n      <aff xml:lang="en" id="A6">LONG Ying, Ph.D., is a research fellow in the School of Architecture, Tsinghua University. His research focuses on urban and rural planning technical science</aff>\n      <author-notes>\n        <corresp id="CR1"><label>*</label>(Corresponding author Email): <email>ylong@mails.tsinghua.edu.cn</email></corresp>\n      </author-notes>\n      <pub-date pub-type="ppub">\n        <year>2020</year>\n      </pub-date>\n      <pub-date pub-type="nihms-submitted">\n        <day>23</day>\n        <month>1</month>\n        <year>2024</year>\n      </pub-date>\n      <pub-date pub-type="pmc-release">\n        <day>28</day>\n        <month>2</month>\n        <year>2024</year>\n      </pub-date>\n      <volume>27</volume>\n      <issue>11</issue>\n      <fpage>96</fpage>\n      <lpage>103</lpage>\n      <abstract>\n        <p id="P1">&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x4F5C;&#x4E3A;&#x57CE;&#x5E02;&#x5C45;&#x6C11;&#x4F4F;&#x6240;&#x6240;&#x5728;&#x5730;&#xFF0C;&#x662F;&#x57CE;&#x5E02;&#x5C45;&#x6C11;&#x65E5;&#x5E38;&#x751F;&#x6D3B;&#x4E2D;&#x6240;&#x5904;&#x65F6;&#x95F4;&#x6700;&#x957F;&#x7684;&#x573A;&#x6240;&#xFF0C;&#x5BF9;&#x5C45;&#x6C11;&#x7684;&#x5065;&#x5EB7;&#x53EF;&#x4EE5;&#x4EA7; &#x751F;&#x51B3;&#x5B9A;&#x6027;&#x7684;&#x5F71;&#x54CD;&#x3002;&#x5F53;&#x4E0B;&#x4E2D;&#x56FD;&#x57CE;&#x5E02;&#x7684;&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#xFF0C;&#x9762;&#x4E34;&#x8001;&#x65E7;&#x6539;&#x9020;&#x3001;&#x73B0;&#x6709;&#x7EF4;&#x62A4;&#x548C;&#x672A;&#x6765;&#x65B0;&#x5EFA;&#x4E09;&#x5927;&#x91CD;&#x8981;&#x4EFB;&#x52A1;&#xFF0C;&#x6025;&#x9700;&#x6784;&#x5EFA;&#x79D1; &#x5B66;&#x5408;&#x7406;&#x7684;&#x8BC4;&#x4EF7;&#x6807;&#x51C6;&#x4EE5;&#x6307;&#x5BFC;&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x7684;&#x5065;&#x5EB7;&#x5316;&#x53D1;&#x5C55;&#x3002;&#x4E3A;&#x6784;&#x5EFA;&#x8BC4;&#x4EF7;&#x4F53;&#x7CFB;&#xFF0C;&#x9996;&#x5148;&#xFF0C;&#x660E;&#x786E;&#x6307;&#x6807;&#x7684;&#x4E09;&#x5927;&#x9009;&#x53D6;&#x539F;&#x5219;&#xFF1A;&#x6307;&#x6807;&#x9009; &#x53D6;&#x987B;&#x4ECE;&#x4EBA;&#x672C;&#x89C6;&#x89D2;&#x51FA;&#x53D1;&#xFF1B;&#x6307;&#x6807;&#x9009;&#x53D6;&#x5E94;&#x91CD;&#x89C6;&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x4E0E;&#x5065;&#x5EB7;&#x7ED3;&#x5C40;&#x95F4;&#x7684;&#x4F20;&#x5BFC;&#x8FC7;&#x7A0B;&#xFF1B;&#x6307;&#x6807;&#x5E94;&#x4ECE;&#x591A;&#x5C3A;&#x5EA6;&#x9009;&#x53D6;&#xFF0C;&#x5E76;&#x4F53;&#x73B0;&#x5C0F;&#x533A; &#x5185;&#x5916;&#x6709;&#x522B;&#x3002;&#x5176;&#x6B21;&#xFF0C;&#x521B;&#x65B0;&#x6027;&#x5730;&#x7ED3;&#x5408;&#x57CE;&#x5E02;&#x89C4;&#x5212;&#x4E0E;&#x516C;&#x5171;&#x5065;&#x5EB7;&#x5B66;&#x79D1;&#x89C6;&#x89D2;&#xFF0C;&#x786E;&#x5B9A;&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x5F71;&#x54CD;&#x5C45;&#x6C11;&#x5065;&#x5EB7;&#x7684;&#x6F5C;&#x5728;&#x6307;&#x6807;&#xFF1B;&#x901A;&#x8FC7; &#x8D28;&#x91CF;&#x5BA1;&#x67E5;&#x73AF;&#x8282;&#x68C0;&#x7D22;&#x6587;&#x732E;&#x5F97;&#x5230;&#x4E0E;&#x5065;&#x5EB7;&#x7ED3;&#x5C40;&#x76F8;&#x5173;&#x7684;&#x8BC1;&#x636E;&#xFF0C;&#x652F;&#x6491;&#x6307;&#x6807;&#x7684;&#x51C6;&#x786E;&#x6027;&#x4E0E;&#x6709;&#x6548;&#x6027;&#x3002;&#x6700;&#x540E;&#xFF0C;&#x5BF9;&#x672A;&#x6765;&#x5065;&#x5EB7;&#x5C45;&#x4F4F;&#x5C0F;&#x533A; &#x7684;&#x8BC4;&#x4EF7;&#x4E0E;&#x53D1;&#x5C55;&#x8FDB;&#x884C;&#x5C55;&#x671B;&#xFF1A;1&#xFF09;&#x5E94;&#x7ED3;&#x5408;&#x4E2D;&#x56FD;&#x5B9E;&#x9645;&#x60C5;&#x51B5;&#xFF0C;&#x5B8C;&#x5584;&#x4E0E;&#x5E94;&#x7528;&#x5065;&#x5EB7;&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x7684;&#x8BC4;&#x4EF7;&#x4F53;&#x7CFB;&#xFF1B;2&#xFF09;&#x672A;&#x6765;&#x5E94;&#x52A0;&#x5F3A;&#x65B0;&#x6280; &#x672F;&#x5728;&#x5065;&#x5EB7;&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x5185;&#x7684;&#x5E94;&#x7528;&#xFF1B;3&#xFF09;&#x5728;&#x5065;&#x5EB7;&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x7684;&#x53D1;&#x5C55;&#x4E2D;&#x52A0;&#x5165;&#x5BF9;&#x57CE;&#x5E02;&#x65B0;&#x53D8;&#x5316;&#x7684;&#x601D;&#x8003;&#x3002;&#x5E0C;&#x671B;&#x901A;&#x8FC7;&#x6784;&#x5EFA;&#x76F8;&#x5BF9;&#x5B8C;&#x6574;&#x3001; &#x5F00;&#x653E;&#x7684;&#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x8BC4;&#x4EF7;&#x6307;&#x6807;&#x4F53;&#x7CFB;&#xFF0C;&#x4F7F;&#x66F4;&#x591A;&#x76F8;&#x5173;&#x7814;&#x7A76;&#x4EBA;&#x5458;&#x3001;&#x653F;&#x5E9C;&#x9886;&#x5BFC;&#x8BA4;&#x8BC6;&#x5230;&#x8FD9;&#x4E00;&#x95EE;&#x9898;&#x7684;&#x91CD;&#x8981;&#x6027;&#x4E0E;&#x7D27;&#x8FEB;&#x6027;&#xFF0C;&#x4FC3;&#x8FDB;&#x4E2D;&#x56FD; &#x5C45;&#x4F4F;&#x5C0F;&#x533A;&#x7684;&#x5065;&#x5EB7;&#x5316;&#x53D1;&#x5C55;&#x3002;</p>\n      </abstract>\n      <trans-abstract xml:lang="en">\n        <p id="P2">Neighborhoods are places where people spend the most time in their lives. Neighborhoods have a decisive impact on the residents&#x2019; health. With several important tasks, including the transformation of old neighborhoods, the maintenance of existing neighborhoods, and the construction of new neighborhoods in the future, a scientific and reasonable evaluation standard is urgently needed to guide the development of healthy neighborhoods. To build the evaluation system, this paper first clarifies the principles for selecting evaluation indicators, which include: 1) the indicators are selected from a humanistic perspective; 2) the pathways between neighborhoods environment and health outcomes are deeply considered; 3) the indicators are selected from multiple scales. Secondly, based on the combined perspectives of urban planning and public health, it identifies the indicators that affect the residents&#x2019; health in neighborhoods and searches the literature through the quality assessment to provide evidence to support the accuracy and effectiveness of the indicators. Finally, it proposes prospect to the evaluation, including 1) it is urgent to improve and utilize the healthy neighborhoods based on the Chinese condition; 2) advanced technologies need to be widely applied in neighborhoods in the future; 3) the transitions in cities should be considered in the future development of neighborhoods. It hopes that relevant researchers and government leaders to realize the importance and urgency of healthy neighborhoods to build more healthy neighborhoods in China.</p>\n      </trans-abstract>\n      <kwd-group>\n        <kwd>&#x4F53;&#x80B2;&#x953B;&#x70BC;</kwd>\n        <kwd>&#x4F20;&#x5BFC;&#x8FC7;&#x7A0B;</kwd>\n        <kwd>&#x4EBA;&#x672C;&#x89C6;&#x89D2;</kwd>\n        <kwd>&#x5065;&#x5EB7;&#x7ED3;&#x5C40;</kwd>\n        <kwd>&#x516C;&#x5171;&#x5065;&#x5EB7;</kwd>\n      </kwd-group>\n      <kwd-group xml:lang="en">\n        <kwd>physical activities</kwd>\n        <kwd>pathways</kwd>\n        <kwd>humanistic perspective</kwd>\n        <kwd>health outcomes</kwd>\n        <kwd>public health </kwd>\n      </kwd-group>\n    </article-meta>\n  </front>\n</article>\n</pmc-articleset>']

(3) Accessing Full Article Content in XML Form
Query A: Get the full article text in xml form for the Pubmed Article corresponding to ID 7615674 if it exists
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10838546] -> [b'<?xml version="1.0" ?>\n<!DOCTYPE pmc-articleset PUBLIC "-//NLM//DTD ARTICLE SET 2.0//EN" "https://dtd.nlm.nih.gov/ncbi/pmc/articleset/nlm-articleset-2.0.dtd">\n<pmc-articleset><article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" dtd-version="1.3" xml:lang="en" article-type="research-article">\n  <!--The publisher of this article does not allow downloading of the full text in XML form.-->\n  <?properties manuscript?>\n  <processing-meta base-tagset="archiving" mathml-version="3.0" table-model="xhtml" tagset-family="jats">\n    <restricted-by>pmc</restricted-by>\n  </processing-meta>\n  <front>\n    <journal-meta>\n      <journal-id journal-id-type="nlm-journal-id">9604648</journal-id>\n      <journal-id journal-id-type="pubmed-jr-id">20305</journal-id>\n      <journal-id journal-id-type="nlm-ta">Nat Biotechnol</journal-id>\n      <journal-id journal-id-type="iso-abbrev">Nat Biotechnol</journal-id>\n      <journal-title-group>\n        <journal-title>Nature biotechnology</journal-title>\n      </journal-title-group>\n      <issn pub-type="ppub">1087-0156</issn>\n      <issn pub-type="epub">1546-1696</issn>\n    </journal-meta>\n    <article-meta>\n      <article-id pub-id-type="pmid">33273732</article-id>\n      <article-id pub-id-type="pmc">10838546</article-id>\n      <article-id pub-id-type="doi">10.1038/s41587-020-00758-7</article-id>\n      <article-id pub-id-type="manuscript">NIHMS1960342</article-id>\n      <article-categories>\n        <subj-group subj-group-type="heading">\n          <subject>Article</subject>\n        </subj-group>\n      </article-categories>\n      <title-group>\n        <article-title>The Art of Virtual Mentoring in the 21<sup>st</sup> Century for STEM Majors and Beyond</article-title>\n      </title-group>\n      <contrib-group>\n        <contrib contrib-type="author">\n          <name>\n            <surname>McReynolds</surname>\n            <given-names>Melanie R.</given-names>\n          </name>\n          <degrees>Ph.D.</degrees>\n          <xref rid="A1" ref-type="aff">1</xref>\n          <xref rid="FN1" ref-type="author-notes">*</xref>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>Termini</surname>\n            <given-names>Christina M.</given-names>\n          </name>\n          <degrees>Ph.D.</degrees>\n          <xref rid="A2" ref-type="aff">2</xref>\n          <xref rid="FN1" ref-type="author-notes">*</xref>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>Hinton</surname>\n            <given-names>Antentor</given-names>\n            <suffix>Jr.</suffix>\n          </name>\n          <degrees>Ph.D.</degrees>\n          <xref rid="A3" ref-type="aff">3</xref>\n          <xref rid="A4" ref-type="aff">4</xref>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>Taylor</surname>\n            <given-names>Brittany L.</given-names>\n          </name>\n          <degrees>Ph.D.</degrees>\n          <xref rid="A5" ref-type="aff">5</xref>\n          <xref rid="A6" ref-type="aff">6</xref>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>Vue</surname>\n            <given-names>Zer</given-names>\n          </name>\n          <degrees>Ph.D.</degrees>\n          <xref rid="A7" ref-type="aff">7</xref>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>Huang</surname>\n            <given-names>Sunny C.</given-names>\n          </name>\n          <xref rid="A4" ref-type="aff">4</xref>\n          <xref rid="A10" ref-type="aff">10</xref>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>Roby</surname>\n            <given-names>ReAnna S.</given-names>\n          </name>\n          <degrees>Ph.D.</degrees>\n          <xref rid="A8" ref-type="aff">8</xref>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>Shuler</surname>\n            <given-names>Haysetta</given-names>\n          </name>\n          <degrees>Ph.D.</degrees>\n          <xref rid="A9" ref-type="aff">9</xref>\n          <xref rid="CR1" ref-type="corresp">#</xref>\n        </contrib>\n        <contrib contrib-type="author">\n          <name>\n            <surname>Carter</surname>\n            <given-names>Calvin S.</given-names>\n          </name>\n          <degrees>Ph.D</degrees>\n          <xref rid="A3" ref-type="aff">3</xref>\n          <xref rid="A4" ref-type="aff">4</xref>\n          <xref rid="A10" ref-type="aff">10</xref>\n          <xref rid="CR1" ref-type="corresp">#</xref>\n        </contrib>\n      </contrib-group>\n      <aff id="A1"><label>1</label>Lewis-Sigler Institute for Integrative Genomics and Department of Chemistry, Princeton University, Princeton, NJ</aff>\n      <aff id="A2"><label>2</label>Department of Medicine, Division of Hematology/Oncology, The University of California, Los Angeles, Los Angeles, CA</aff>\n      <aff id="A3"><label>3</label>Department of Internal Medicine, University of Iowa Hospitals and Clinics, Iowa City, IA</aff>\n      <aff id="A4"><label>4</label>Fraternal Order of Eagles Diabetes Research Center, Roy J. and Lucille A. Carver College of Medicine, University of Iowa, Iowa City, IA</aff>\n      <aff id="A5"><label>5</label>Department of Orthopaedic Surgery, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA</aff>\n      <aff id="A6"><label>6</label>McKay Orthopaedic Research Lab, University of Pennsylvania, Philadelphia, PA</aff>\n      <aff id="A7"><label>7</label>Department of Cell &amp; Tissue Biology, University of California, San Francisco, San Francisco, CA</aff>\n      <aff id="A8"><label>8</label>Fisk-Vanderbilt Center of Excellence, Peabody College, Vanderbilt University, Nashville, TN</aff>\n      <aff id="A9"><label>9</label>Department of Biological Sciences, Winston-Salem State University, Winston-Salem, NC</aff>\n      <aff id="A10"><label>10</label>Geminii, Inc., Iowa City, IA</aff>\n      <author-notes>\n        <corresp id="CR1"><label>#</label>Co-corresponding authors: Calvin S. Carter, PhD, Geminii, Inc. Iowa City, IA 52246 and Haysetta Shuler, PhD, Winston-Salem State University, Winston-Salem NC 27110. <ext-link xlink:href="http://@calvinscarter" ext-link-type="uri">@calvinscarter</ext-link></corresp>\n        <fn id="FN1">\n          <label>*</label>\n          <p id="P1">Denotes Co-first authorship</p>\n        </fn>\n      </author-notes>\n      <pub-date pub-type="nihms-submitted">\n        <day>28</day>\n        <month>1</month>\n        <year>2024</year>\n      </pub-date>\n      <pub-date pub-type="ppub">\n        <month>12</month>\n        <year>2020</year>\n      </pub-date>\n      <pub-date pub-type="pmc-release">\n        <day>04</day>\n        <month>2</month>\n        <year>2024</year>\n      </pub-date>\n      <volume>38</volume>\n      <issue>12</issue>\n      <fpage>1477</fpage>\n      <lpage>1482</lpage>\n    </article-meta>\n  </front>\n</article>\n</pmc-articleset>']
Result A: The publisher of this article does not allow downloading of the full text in XML form so the full text is not accessible
Usage Comments A: Not all articles are available in full text form. This is a common occurence with articles that are not open access.

Query B: Get the full article text in xml form for the Pubmed Article corresponding to ID 1790863 if it exists
[https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC1790863]
Result B: A full (encoded) xml page containing the full text of the article and its metadata.
Usage Comments B: The full xml page will be too large to handle as a raw string. You MUST use file input/output and shell piping to handle the output of this.
    """

    return s


    

@schema_tool
async def ncbi_api_call(ncbi_query_url : str = Field(description = "A url that uses the NCBI web apis to answer genomic questions")) -> str:
    """Use the NCBI Web API to work on tasks that are aided by use of the NCBI databases"""
    query_response = call_api(ncbi_query_url)
    return query_response