{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "import os\n",
    "import pandas as pd\n",
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('/Users/gkreder/Downloads/pubmedqa/data/ori_pqaa.json', 'r') as f:\n",
    "    data = json.load(f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Epiretinal Membrane',\n",
       " 'Blood Sedimentation',\n",
       " 'Automation, Laboratory',\n",
       " 'Portasystemic Shunt, Surgical',\n",
       " 'Purinergic P1 Receptor Agonists',\n",
       " 'Health Status Indicators',\n",
       " 'Ear Auricle',\n",
       " 'Proctoscopes',\n",
       " 'Public Sector',\n",
       " 'Trans-Splicing',\n",
       " 'Lingual Nerve Injuries',\n",
       " 'Cardiac Output',\n",
       " 'Interleukin-10 Receptor beta Subunit',\n",
       " 'Voltage-Gated Sodium Channel beta-2 Subunit',\n",
       " 'Hemodialysis Units, Hospital',\n",
       " 'Withholding Treatment',\n",
       " 'Complement Inactivating Agents',\n",
       " 'Thogotovirus',\n",
       " 'Vacuum',\n",
       " 'Pasteurization',\n",
       " 'Medical Device Recalls',\n",
       " 'Communication Barriers',\n",
       " 'Cholecystectomy, Laparoscopic',\n",
       " 'Androsterone',\n",
       " 'Parabiosis',\n",
       " 'Hereditary Angioedema Types I and II',\n",
       " 'Subarachnoid Hemorrhage',\n",
       " 'Malignant Atrophic Papulosis',\n",
       " 'Heart Failure, Diastolic',\n",
       " 'Hydrochloric Acid',\n",
       " 'Plant Gums',\n",
       " 'Thyrotropin',\n",
       " 'Host-Pathogen Interactions',\n",
       " 'Cardiotocography',\n",
       " 'Biological Specimen Banks',\n",
       " 'Trichosporonosis',\n",
       " 'Blotting, Far-Western',\n",
       " 'Cyclooctanes',\n",
       " 'Ophthalmologists',\n",
       " 'Cause of Death',\n",
       " 'Colon',\n",
       " 'Genes, MHC Class I',\n",
       " 'Burkholderia cenocepacia',\n",
       " 'Seminiferous Epithelium',\n",
       " 'Peptic Ulcer Perforation',\n",
       " 'Leg',\n",
       " 'Alpha-Globulins',\n",
       " 'Bioterrorism',\n",
       " 'Caroli Disease',\n",
       " 'Fibric Acids',\n",
       " 'Dentin',\n",
       " 'Intraabdominal Infections',\n",
       " 'Propylthiouracil',\n",
       " 'Coccidioidomycosis',\n",
       " 'Hymenolepis',\n",
       " 'Hexamethonium Compounds',\n",
       " 'Glucose Transporter Type 2',\n",
       " 'Olmesartan Medoxomil',\n",
       " 'Reoviridae Infections',\n",
       " 'Potassium Cyanide',\n",
       " 'Glycyrrhiza uralensis',\n",
       " 'Delayed-Action Preparations',\n",
       " 'GTP-Binding Proteins',\n",
       " 'Gymnastics',\n",
       " 'Stochastic Processes',\n",
       " 'Ecchymosis',\n",
       " 'Behavior Control',\n",
       " 'Methyltyrosines',\n",
       " 'Magnesium Deficiency',\n",
       " 'Enalaprilat',\n",
       " 'Dose-Response Relationship, Drug',\n",
       " 'Convalescence',\n",
       " 'Jaundice',\n",
       " 'Nephroblastoma Overexpressed Protein',\n",
       " 'Plant Stems',\n",
       " 'Anterior Chamber',\n",
       " 'Salpingitis',\n",
       " 'Chemoradiotherapy, Adjuvant',\n",
       " 'N-Myc Proto-Oncogene Protein',\n",
       " 'Carpal Bones',\n",
       " \"5' Untranslated Regions\",\n",
       " 'Receptors, Glucocorticoid',\n",
       " 'CA3 Region, Hippocampal',\n",
       " 'Efficiency, Organizational',\n",
       " 'Auditory Pathways',\n",
       " 'Denture, Complete, Upper',\n",
       " 'Receptors, Prostaglandin E, EP1 Subtype',\n",
       " 'Microbial Viability',\n",
       " 'Receptors, Interleukin-8A',\n",
       " 'H(+)-K(+)-Exchanging ATPase',\n",
       " 'Diastema',\n",
       " 'Ampyrone',\n",
       " 'Loneliness',\n",
       " 'Pancreatic Diseases',\n",
       " 'Antibodies, Viral',\n",
       " 'Hexanes',\n",
       " 'Atrioventricular Node',\n",
       " 'Silicone Oils',\n",
       " 'Occupational Health Services',\n",
       " 'Macrophages, Peritoneal',\n",
       " 'Sex Distribution',\n",
       " 'Pneumothorax, Artificial',\n",
       " 'Numerical Analysis, Computer-Assisted',\n",
       " 'Spirituality',\n",
       " 'Phosphotransferases (Phosphomutases)',\n",
       " 'Shoulder Impingement Syndrome',\n",
       " 'Streptonigrin',\n",
       " 'School Dentistry',\n",
       " 'Liver Circulation',\n",
       " 'Pectoralis Muscles',\n",
       " 'Pain Perception',\n",
       " 'Carboxymethylcellulose Sodium',\n",
       " 'Reflex, Stretch',\n",
       " 'Carnitine O-Palmitoyltransferase',\n",
       " 'Neurotrophin 3',\n",
       " 'Cyanates',\n",
       " 'Hyponatremia',\n",
       " 'Prolyl-Hydroxylase Inhibitors',\n",
       " 'Lactulose',\n",
       " 'Binding Sites',\n",
       " 'Medicine, Chinese Traditional',\n",
       " 'Dynorphins',\n",
       " 'Biodegradation, Environmental',\n",
       " 'Malate Dehydrogenase (NADP+)',\n",
       " 'Campanulaceae',\n",
       " 'RNA, Archaeal',\n",
       " 'Prolactinoma',\n",
       " 'Nucleoside-Diphosphate Kinase',\n",
       " 'Peripheral Nervous System Diseases',\n",
       " 'Aggregatibacter segnis',\n",
       " 'Cathepsin Z',\n",
       " 'Uterine Cervicitis',\n",
       " 'Aminoacetonitrile',\n",
       " 'No-Show Patients',\n",
       " 'Autoreceptors',\n",
       " 'Tellurium',\n",
       " 'Cineradiography',\n",
       " 'Ultrasonography, Doppler',\n",
       " 'Atractyloside',\n",
       " 'Nurse Anesthetists',\n",
       " 'Embolism, Amniotic Fluid',\n",
       " 'Nicorandil',\n",
       " 'Genes, p53',\n",
       " 'APOBEC-1 Deaminase',\n",
       " 'Self Administration',\n",
       " 'Brachial Plexus Block',\n",
       " 'Lactose',\n",
       " 'Disease Transmission, Infectious',\n",
       " 'Latent Tuberculosis',\n",
       " 'Vitamin B 6',\n",
       " 'Ambulatory Care Information Systems',\n",
       " 'Castor Oil',\n",
       " 'Fluorescent Antibody Technique, Indirect',\n",
       " 'Hydroxytryptophol',\n",
       " 'Genes, Immunoglobulin',\n",
       " 'Poly G',\n",
       " 'Scopolamine Derivatives',\n",
       " 'Karyopherins',\n",
       " 'Mandibular Condyle',\n",
       " 'Cannabinoid Receptor Modulators',\n",
       " 'Organic Chemistry Phenomena',\n",
       " 'Fontan Procedure',\n",
       " 'Herpestidae',\n",
       " 'Self-Control',\n",
       " 'Floods',\n",
       " 'Organometallic Compounds',\n",
       " 'Iridoid Glycosides',\n",
       " 'Birth Certificates',\n",
       " 'Phlebitis',\n",
       " 'Pseudomonas aeruginosa',\n",
       " 'ATP Binding Cassette Transporter, Subfamily D, Member 1',\n",
       " 'Pioglitazone',\n",
       " 'Synteny',\n",
       " 'Housing, Animal',\n",
       " 'Compomers',\n",
       " 'Clausena',\n",
       " 'Endotoxins',\n",
       " 'Loranthaceae',\n",
       " 'Foods, Specialized',\n",
       " 'Soluble Guanylyl Cyclase',\n",
       " 'Respiration',\n",
       " 'Omentum',\n",
       " 'Trillium',\n",
       " 'Product Labeling',\n",
       " 'Mycoplasma penetrans',\n",
       " 'Sex Differentiation',\n",
       " 'Mucin-1',\n",
       " 'Embryo Transfer',\n",
       " 'Regression (Psychology)',\n",
       " 'Myeloid-Derived Suppressor Cells',\n",
       " 'Light Signal Transduction',\n",
       " 'beta-Adrenergic Receptor Kinases',\n",
       " 'Carbonic Anhydrase I',\n",
       " 'Adrenal Glands',\n",
       " 'Wrist Joint',\n",
       " 'Aqueous Humor',\n",
       " 'Telomere',\n",
       " 'Oncogene Proteins',\n",
       " 'Febuxostat',\n",
       " 'Trypanosoma brucei rhodesiense',\n",
       " 'Gingival Neoplasms',\n",
       " 'HLA-DR7 Antigen',\n",
       " 'Brevibacterium',\n",
       " 'Protein Refolding',\n",
       " 'Population Surveillance',\n",
       " 'Phosphocreatine',\n",
       " 'Vesicle-Associated Membrane Protein 1',\n",
       " 'Stomach Neoplasms',\n",
       " 'Pyridoxine',\n",
       " 'Catechol O-Methyltransferase Inhibitors',\n",
       " 'Aftercare',\n",
       " 'Interview, Psychological',\n",
       " 'Nipecotic Acids',\n",
       " 'Hospital Charges',\n",
       " 'Luteinizing Hormone',\n",
       " 'CD2 Antigens',\n",
       " 'Clobetasol',\n",
       " 'Rhabdomyosarcoma, Alveolar',\n",
       " 'Circovirus',\n",
       " 'Arteriovenous Fistula',\n",
       " 'Chemical and Drug Induced Liver Injury',\n",
       " 'Personnel Administration, Hospital',\n",
       " 'Phosphodiesterase I',\n",
       " 'Thromboxane-A Synthase',\n",
       " 'Axial Length, Eye',\n",
       " 'Bivalvia',\n",
       " 'Chelidonium',\n",
       " 'Intraoperative Care',\n",
       " 'Mushroom Bodies',\n",
       " 'Norpregnadienes',\n",
       " 'G-Quadruplexes',\n",
       " 'Fraser Syndrome',\n",
       " 'Protein Binding',\n",
       " 'Nebivolol',\n",
       " 'Axin Signaling Complex',\n",
       " 'Genes, gag',\n",
       " 'PPAR-beta',\n",
       " 'Mentors',\n",
       " 'Glossectomy',\n",
       " 'Angiocardiography',\n",
       " 'Fumigation',\n",
       " 'Receptors, Galanin',\n",
       " 'Rats, Inbred SHR',\n",
       " 'Colonic Diseases',\n",
       " 'Acetylgalactosamine',\n",
       " 'Canidae',\n",
       " 'Collectins',\n",
       " 'Trematode Infections',\n",
       " 'Platycodon',\n",
       " 'Ethics, Medical',\n",
       " 'Haemonchiasis',\n",
       " 'Vomeronasal Organ',\n",
       " 'Myanmar',\n",
       " 'Fossil Fuels',\n",
       " 'Microvessels',\n",
       " 'MAP Kinase Kinase 7',\n",
       " 'Clonorchis sinensis',\n",
       " 'Surgical Wound Infection',\n",
       " 'Censuses',\n",
       " 'Serenoa',\n",
       " 'Gingivoplasty',\n",
       " 'Neisseria lactamica',\n",
       " 'Albinism',\n",
       " 'Turnera',\n",
       " 'Ligase Chain Reaction',\n",
       " 'Police',\n",
       " 'Duodenoscopy',\n",
       " 'Psidium',\n",
       " 'Fanconi Anemia',\n",
       " 'Monkeypox',\n",
       " 'Bee Venoms',\n",
       " 'Phalaris',\n",
       " 'Indazoles',\n",
       " 'Chromobacterium',\n",
       " 'Sinus Thrombosis, Intracranial',\n",
       " 'Neurotensin',\n",
       " 'Keratin-4',\n",
       " 'Computer Literacy',\n",
       " 'Fentanyl',\n",
       " 'Dorsal Raphe Nucleus',\n",
       " 'Allantoin',\n",
       " 'Jagged-2 Protein',\n",
       " 'Betahistine',\n",
       " 'Monitoring, Ambulatory',\n",
       " 'Levivirus',\n",
       " 'Chorismic Acid',\n",
       " 'Tomography',\n",
       " 'Ambrosia',\n",
       " 'Adenocarcinoma, Follicular',\n",
       " 'Type C Phospholipases',\n",
       " 'Cholesterol Oxidase',\n",
       " 'Enoyl-(Acyl-Carrier-Protein) Reductase (NADH)',\n",
       " 'Artesunate',\n",
       " 'Pyometra',\n",
       " 'Pyruvate Dehydrogenase (Lipoamide)',\n",
       " 'COP-Coated Vesicles',\n",
       " 'Sulfatases',\n",
       " 'Matrilin Proteins',\n",
       " 'G2 Phase',\n",
       " 'Tetraspanin 28',\n",
       " 'Myeloid-Lymphoid Leukemia Protein',\n",
       " 'Lincomycin',\n",
       " 'Aortic Rupture',\n",
       " 'Gitelman Syndrome',\n",
       " 'Genes, Recessive',\n",
       " 'Heart-Lung Machine',\n",
       " 'Ubiquitination',\n",
       " 'Group Processes',\n",
       " 'Resins, Synthetic',\n",
       " 'Data Display',\n",
       " 'Mycoplasma Infections',\n",
       " 'Rabies Vaccines',\n",
       " 'Ablation Techniques',\n",
       " 'Aminooxyacetic Acid',\n",
       " 'Pulse Radiolysis',\n",
       " 'Metoprolol',\n",
       " 'Body Constitution',\n",
       " 'Homocysteine',\n",
       " 'Neoplasms, Hormone-Dependent',\n",
       " 'Enzootic Bovine Leukosis',\n",
       " 'Kidney Pelvis',\n",
       " 'Maxillofacial Development',\n",
       " 'Acidosis, Respiratory',\n",
       " 'Salicylamides',\n",
       " 'Animals, Newborn',\n",
       " 'Gravity Sensing',\n",
       " 'Polytene Chromosomes',\n",
       " 'Scattering, Small Angle',\n",
       " 'gamma-Aminobutyric Acid',\n",
       " 'Delayed Graft Function',\n",
       " 'Carcinoid Tumor',\n",
       " 'Sodium Benzoate',\n",
       " 'Pulmonary Artery',\n",
       " 'Proto-Oncogene Proteins c-pim-1',\n",
       " 'Gonadotropin-Releasing Hormone',\n",
       " 'Drug Monitoring',\n",
       " 'ets-Domain Protein Elk-4',\n",
       " 'Spinal Nerve Roots',\n",
       " 'Snow',\n",
       " 'Urinary Bladder Fistula',\n",
       " 'Receptors, Tachykinin',\n",
       " 'Blood Physiological Phenomena',\n",
       " 'Ribs',\n",
       " 'Exanthema',\n",
       " 'Thanatophoric Dysplasia',\n",
       " 'Heterogeneous-Nuclear Ribonucleoprotein Group C',\n",
       " 'Antimony Sodium Gluconate',\n",
       " 'Microsatellite Repeats',\n",
       " 'Flavobacteriaceae',\n",
       " 'MutS Homolog 3 Protein',\n",
       " 'Mitochondrial Trifunctional Protein',\n",
       " 'Secretory Pathway',\n",
       " 'Task Performance and Analysis',\n",
       " 'Medical Overuse',\n",
       " 'Fibrin',\n",
       " 'Insurance, Dental',\n",
       " 'Chlorine Compounds',\n",
       " 'Islands',\n",
       " 'Gold Radioisotopes',\n",
       " 'Informed Consent',\n",
       " 'Forced Expiratory Flow Rates',\n",
       " 'Hypopituitarism',\n",
       " 'Microbiota',\n",
       " 'Blood Protein Disorders',\n",
       " 'Isoniazid',\n",
       " 'Restaurants',\n",
       " 'Projective Techniques',\n",
       " 'Discrimination Learning',\n",
       " 'Relaxin',\n",
       " 'Contraception, Postcoital',\n",
       " 'Clopidogrel',\n",
       " 'Polychlorinated Biphenyls',\n",
       " 'Microtomy',\n",
       " 'Neck Dissection',\n",
       " 'Dental Implantation, Endosseous',\n",
       " 'Ileus',\n",
       " 'Cell Tracking',\n",
       " 'Interleukin-1 Receptor-Associated Kinases',\n",
       " 'Organotin Compounds',\n",
       " 'Vesicle-Associated Membrane Protein 3',\n",
       " 'Ethics, Pharmacy',\n",
       " 'beta-Glucans',\n",
       " 'tau Proteins',\n",
       " 'E-Box Elements',\n",
       " 'Radiographic Image Enhancement',\n",
       " 'Glucosephosphate Dehydrogenase Deficiency',\n",
       " 'Pseudarthrosis',\n",
       " 'Veins',\n",
       " 'Destrin',\n",
       " 'Neurogranin',\n",
       " 'Mouth Neoplasms',\n",
       " 'Sweden',\n",
       " 'Adenoma, Bile Duct',\n",
       " 'Calsequestrin',\n",
       " 'DNA Polymerase II',\n",
       " 'Aluminum Hydroxide',\n",
       " 'Bed Occupancy',\n",
       " 'Cytochrome P450 Family 21',\n",
       " 'Potassium Compounds',\n",
       " 'Recoverin',\n",
       " 'Sports Equipment',\n",
       " 'Phosphatidylinositol Diacylglycerol-Lyase',\n",
       " 'Venous Valves',\n",
       " 'Flap Endonucleases',\n",
       " 'Cell Differentiation',\n",
       " 'Malates',\n",
       " 'Crown-Rump Length',\n",
       " 'Germ-Free Life',\n",
       " 'Gene Expression Regulation, Plant',\n",
       " 'Tenecteplase',\n",
       " 'Conus Snail',\n",
       " 'Ralstonia solanacearum',\n",
       " 'Hepcidins',\n",
       " 'Fructokinases',\n",
       " 'Ostomy',\n",
       " 'Models, Theoretical',\n",
       " 'Esophagoscopes',\n",
       " 'Optic Disk',\n",
       " 'Vaccinium',\n",
       " 'Piracetam',\n",
       " 'Electrophoresis, Microchip',\n",
       " 'Osteoma',\n",
       " 'Glutarates',\n",
       " 'Physical Endurance',\n",
       " 'Nedocromil',\n",
       " 'Post-Synaptic Density',\n",
       " 'Feedback',\n",
       " 'Visual Fields',\n",
       " 'Xenobiotics',\n",
       " 'Arthritis, Juvenile',\n",
       " 'Helicobacter Infections',\n",
       " 'Microchip Analytical Procedures',\n",
       " 'Kinesthesis',\n",
       " 'Portoenterostomy, Hepatic',\n",
       " 'Hemoglobinometry',\n",
       " 'Porphyrias',\n",
       " 'Aneuploidy',\n",
       " 'Insulin-Like Growth Factor Binding Protein 4',\n",
       " 'Attachment Sites, Microbiological',\n",
       " 'Dioctyl Sulfosuccinic Acid',\n",
       " 'Atrial Premature Complexes',\n",
       " 'Government Programs',\n",
       " 'Trichlorfon',\n",
       " 'Coal',\n",
       " 'CD13 Antigens',\n",
       " 'Myristic Acids',\n",
       " 'Neurotic Disorders',\n",
       " 'Arctium',\n",
       " 'Oximetry',\n",
       " 'Cysteine',\n",
       " 'Sphingosine',\n",
       " 'Loganiaceae',\n",
       " 'Carbonic Anhydrase IX',\n",
       " 'Nucleic Acid Hybridization',\n",
       " 'Microalgae',\n",
       " 'Central Nervous System Sensitization',\n",
       " 'Culicidae',\n",
       " 'Phosphorus, Dietary',\n",
       " 'Pseudomyxoma Peritonei',\n",
       " 'Spinal Cord Injuries',\n",
       " 'Carotid Artery, Common',\n",
       " 'Primary Nursing',\n",
       " 'Walk Test',\n",
       " 'Lebanon',\n",
       " 'Nigella sativa',\n",
       " \"3',5'-Cyclic-GMP Phosphodiesterases\",\n",
       " 'Personnel Downsizing',\n",
       " 'Plasmacytoma',\n",
       " '1-Phosphatidylinositol 4-Kinase',\n",
       " 'Receptors, Interleukin-18',\n",
       " 'Latency Period (Psychology)',\n",
       " 'Enzyme Activators',\n",
       " 'Double Outlet Right Ventricle',\n",
       " 'Jugular Veins',\n",
       " 'Dietary Proteins',\n",
       " 'Carmine',\n",
       " 'Purkinje Cells',\n",
       " 'Knee',\n",
       " 'Food-Drug Interactions',\n",
       " 'Arrestins',\n",
       " 'Metabolic Equivalent',\n",
       " 'Carbohydrate Dehydrogenases',\n",
       " 'Colic',\n",
       " 'OX40 Ligand',\n",
       " 'Aortic Valve Insufficiency',\n",
       " 'Paternal Exposure',\n",
       " 'Victoria',\n",
       " 'Nutrition Surveys',\n",
       " 'Spermine',\n",
       " 'Inhalant Abuse',\n",
       " 'Kashin-Beck Disease',\n",
       " 'Hair Cells, Auditory, Inner',\n",
       " 'Lithospermum',\n",
       " 'Masculinity',\n",
       " 'Dental Pulp Test',\n",
       " 'Morus',\n",
       " 'Iodophors',\n",
       " 'Equol',\n",
       " 'Hydroxyethylrutoside',\n",
       " 'Mosaic Viruses',\n",
       " 'Construction Materials',\n",
       " 'Calcium Sulfate',\n",
       " 'Ethics, Professional',\n",
       " 'Psychology',\n",
       " 'Heart Rate',\n",
       " 'Phacoemulsification',\n",
       " 'Pilonidal Sinus',\n",
       " 'Bepridil',\n",
       " 'Spiramycin',\n",
       " 'RNA Recognition Motif Proteins',\n",
       " 'Chondrosarcoma',\n",
       " 'Octanols',\n",
       " 'Salmonella typhi',\n",
       " 'Polydnaviridae',\n",
       " 'Acetyl Coenzyme A',\n",
       " 'Whole Genome Sequencing',\n",
       " 'Gravity Suits',\n",
       " 'Intracranial Hemorrhage, Hypertensive',\n",
       " 'Lethargy',\n",
       " 'Receptors, Cell Surface',\n",
       " 'Weightlessness',\n",
       " 'Ketotifen',\n",
       " 'Education, Dental, Graduate',\n",
       " 'Opisthorchiasis',\n",
       " 'Chief Cells, Gastric',\n",
       " 'Loteprednol Etabonate',\n",
       " 'Nasopharyngitis',\n",
       " 'Gluconacetobacter xylinus',\n",
       " 'Psychoneuroimmunology',\n",
       " 'Fibrocystic Breast Disease',\n",
       " 'Thelazioidea',\n",
       " 'Social Environment',\n",
       " 'Atrial Fibrillation',\n",
       " 'Cystamine',\n",
       " 'Receptors, Urokinase Plasminogen Activator',\n",
       " 'Homeless Persons',\n",
       " 'United States Government Agencies',\n",
       " 'Cervical Vertebrae',\n",
       " 'Diterpenes, Clerodane',\n",
       " 'Death Domain Receptor Signaling Adaptor Proteins',\n",
       " 'Myopathies, Structural, Congenital',\n",
       " 'Spiritual Therapies',\n",
       " 'Community-Acquired Infections',\n",
       " 'Mass Casualty Incidents',\n",
       " 'Fenofibrate',\n",
       " 'Benzydamine',\n",
       " 'Endometrium',\n",
       " 'Baroreflex',\n",
       " 'Manipulation, Spinal',\n",
       " 'Nevirapine',\n",
       " 'Panthera',\n",
       " 'Salsola',\n",
       " 'Immunoglobulins',\n",
       " 'COUP Transcription Factor II',\n",
       " 'Thrombectomy',\n",
       " 'Residential Treatment',\n",
       " 'Pharmacological Phenomena',\n",
       " 'Buserelin',\n",
       " 'Raloxifene Hydrochloride',\n",
       " 'Research Design',\n",
       " 'United Kingdom',\n",
       " 'Romania',\n",
       " 'Ossicular Prosthesis',\n",
       " 'Glucuronosyltransferase',\n",
       " 'Ertapenem',\n",
       " 'Immunity, Heterologous',\n",
       " 'Plyometric Exercise',\n",
       " 'Gram-Positive Bacteria',\n",
       " 'Tropical Climate',\n",
       " 'Smallpox Vaccine',\n",
       " 'Middle Cerebral Artery',\n",
       " 'Electrodes',\n",
       " 'Terminology as Topic',\n",
       " 'Dopamine D2 Receptor Antagonists',\n",
       " 'Muscidae',\n",
       " 'Azaserine',\n",
       " 'Sterol Esterase',\n",
       " 'Mediastinoscopy',\n",
       " 'Male Urogenital Diseases',\n",
       " 'Uterine Perforation',\n",
       " 'Prevotella ruminicola',\n",
       " 'Immunologic Factors',\n",
       " 'Autoimmunity',\n",
       " 'Folklore',\n",
       " 'Tunica Intima',\n",
       " 'YY1 Transcription Factor',\n",
       " 'Cranial Nerve Neoplasms',\n",
       " 'Leukodystrophy, Globoid Cell',\n",
       " 'Teratoma',\n",
       " 'Oryza',\n",
       " 'Steroid 21-Hydroxylase',\n",
       " 'Integration Host Factors',\n",
       " 'Darier Disease',\n",
       " 'Bovine papillomavirus 1',\n",
       " 'Disease Progression',\n",
       " 'Thioredoxin-Disulfide Reductase',\n",
       " 'Bone Marrow Examination',\n",
       " 'Binge Drinking',\n",
       " 'Reproductive Techniques',\n",
       " 'Glycerol-3-Phosphate Dehydrogenase (NAD+)',\n",
       " 'Sodium Selenite',\n",
       " 'Cyproheptadine',\n",
       " 'Bacteriophage T4',\n",
       " 'Aminobiphenyl Compounds',\n",
       " 'Tattooing',\n",
       " 'Electronystagmography',\n",
       " 'Abortifacient Agents, Steroidal',\n",
       " 'Restriction Mapping',\n",
       " 'Diverticulosis, Colonic',\n",
       " 'Taurolithocholic Acid',\n",
       " 'Abomasum',\n",
       " 'Factor V Deficiency',\n",
       " 'Retinal Diseases',\n",
       " 'Esophageal Sphincter, Lower',\n",
       " 'Escherichia coli K12',\n",
       " 'Fiber Optic Technology',\n",
       " 'Antidiuretic Hormone Receptor Antagonists',\n",
       " 'Haemophilus parasuis',\n",
       " 'eIF-2 Kinase',\n",
       " 'HLA-A Antigens',\n",
       " 'Intracellular Space',\n",
       " 'Phenylthiohydantoin',\n",
       " 'Finasteride',\n",
       " 'Mitochondrial Proton-Translocating ATPases',\n",
       " 'Nerve Regeneration',\n",
       " 'Fertility Agents, Male',\n",
       " 'Coleoptera',\n",
       " 'Vasovasostomy',\n",
       " 'Biotinidase Deficiency',\n",
       " 'Physics',\n",
       " 'Bulimia Nervosa',\n",
       " 'Tongue Habits',\n",
       " 'Clerodendrum',\n",
       " 'Mardivirus',\n",
       " 'Prometaphase',\n",
       " 'Tilt-Table Test',\n",
       " 'Nosema',\n",
       " 'Capillary Leak Syndrome',\n",
       " 'Cystadenocarcinoma, Serous',\n",
       " 'Explosive Agents',\n",
       " 'Education, Professional',\n",
       " 'Lythrum',\n",
       " 'Human bocavirus',\n",
       " 'Piperaceae',\n",
       " 'Genu Valgum',\n",
       " 'Georgia',\n",
       " 'Chyle',\n",
       " 'Pyruvate Dehydrogenase Complex',\n",
       " 'Grief',\n",
       " 'Deoxyribonucleases',\n",
       " 'Follicular Fluid',\n",
       " 'Anodontia',\n",
       " 'Leukemia Virus, Gibbon Ape',\n",
       " 'Thoracic Wall',\n",
       " 'Pulmonary Surfactant-Associated Protein A',\n",
       " 'Uroplakin III',\n",
       " 'Receptors, Interleukin-15',\n",
       " 'Penicillic Acid',\n",
       " 'Reactive Nitrogen Species',\n",
       " 'Sudan',\n",
       " 'Amaryllidaceae Alkaloids',\n",
       " 'Gallamine Triethiodide',\n",
       " 'Cuprizone',\n",
       " 'Agriculture',\n",
       " 'Progesterone Reductase',\n",
       " 'Beer',\n",
       " 'Tooth Cervix',\n",
       " 'Moloney murine leukemia virus',\n",
       " 'Acupuncture Analgesia',\n",
       " 'Lyme Disease',\n",
       " 'HLA Antigens',\n",
       " 'Radioimmunoprecipitation Assay',\n",
       " 'Ventricular Outflow Obstruction',\n",
       " 'S100 Calcium Binding Protein A7',\n",
       " 'Acro-Osteolysis',\n",
       " 'Physician-Nurse Relations',\n",
       " 'Platelet Aggregation',\n",
       " 'Antigens, Human Platelet',\n",
       " 'Computer Peripherals',\n",
       " 'Anticholesteremic Agents',\n",
       " 'Occludin',\n",
       " 'Radiosurgery',\n",
       " 'Pathology',\n",
       " 'H-2 Antigens',\n",
       " 'Neisseriaceae',\n",
       " 'Chemical Industry',\n",
       " 'Nerve Fibers, Unmyelinated',\n",
       " 'Anaplasma phagocytophilum',\n",
       " 'Gallium Isotopes',\n",
       " 'Plum Pox Virus',\n",
       " 'Carcinoma, Giant Cell',\n",
       " 'Hydrogen',\n",
       " 'Diphtheria-Tetanus Vaccine',\n",
       " 'Stiff-Person Syndrome',\n",
       " 'Arthritis, Experimental',\n",
       " 'Fibrous Dysplasia, Polyostotic',\n",
       " 'Sun Protection Factor',\n",
       " 'Multivesicular Bodies',\n",
       " 'Astigmatism',\n",
       " 'North Carolina',\n",
       " 'Neuroticism',\n",
       " 'Ichthyosis, X-Linked',\n",
       " 'Houseflies',\n",
       " 'Benzyl Alcohol',\n",
       " 'Thiadiazoles',\n",
       " 'Mucor',\n",
       " 'src Homology Domains',\n",
       " 'Fagus',\n",
       " 'Enterochromaffin-like Cells',\n",
       " 'Autoradiography',\n",
       " 'Trifluoroacetic Acid',\n",
       " 'Teratogenesis',\n",
       " 'Bloodletting',\n",
       " 'Somatostatin-Secreting Cells',\n",
       " 'Needles',\n",
       " 'E2F7 Transcription Factor',\n",
       " 'Reproduction, Asexual',\n",
       " 'Post-Concussion Syndrome',\n",
       " 'Meningitis',\n",
       " 'S100A12 Protein',\n",
       " 'Life Tables',\n",
       " 'Chancroid',\n",
       " 'Laminoplasty',\n",
       " 'Embryonic Stem Cells',\n",
       " 'Calicivirus, Feline',\n",
       " 'Freeze Drying',\n",
       " 'Propane',\n",
       " 'Myoclonic Epilepsies, Progressive',\n",
       " 'Radiography, Dental, Digital',\n",
       " 'Angina, Stable',\n",
       " 'Finger Joint',\n",
       " 'ortho-Aminobenzoates',\n",
       " 'Severe Acute Respiratory Syndrome',\n",
       " 'Gatifloxacin',\n",
       " 'Veterans Health',\n",
       " 'Lycoris',\n",
       " 'Gingiva',\n",
       " 'Hepatitis B, Chronic',\n",
       " 'Myofascial Pain Syndromes',\n",
       " 'Multiple Endocrine Neoplasia Type 2a',\n",
       " 'ATP Binding Cassette Transporter, Subfamily B, Member 3',\n",
       " 'Gastric Mucosa',\n",
       " 'Integrin alpha2',\n",
       " 'Idoxuridine',\n",
       " 'Vasopressins',\n",
       " 'Xenorhabdus',\n",
       " 'Pyrimidinones',\n",
       " 'Leprosy, Tuberculoid',\n",
       " 'Bacteriochlorophyll A',\n",
       " 'Analgesics, Non-Narcotic',\n",
       " 'Oropharynx',\n",
       " 'Time Factors',\n",
       " 'Hemorrhagic Disease Virus, Rabbit',\n",
       " 'Hallux',\n",
       " 'Administration, Inhalation',\n",
       " 'Radiography',\n",
       " 'Toxoplasmosis, Animal',\n",
       " 'Lactococcus lactis',\n",
       " 'Keratin-19',\n",
       " 'Niacin',\n",
       " 'Blood Component Transfusion',\n",
       " 'Receptors, Adrenergic, alpha-2',\n",
       " 'Membrane Potentials',\n",
       " 'Polybrominated Biphenyls',\n",
       " 'Exome',\n",
       " 'Vulvar Neoplasms',\n",
       " 'Bevacizumab',\n",
       " 'Pectinidae',\n",
       " 'Ventriculoperitoneal Shunt',\n",
       " 'Metric System',\n",
       " 'Marfan Syndrome',\n",
       " 'Prunus persica',\n",
       " 'Apolipoprotein A-V',\n",
       " 'Radiotherapy, Computer-Assisted',\n",
       " 'Babesiosis',\n",
       " 'Haloperidol',\n",
       " 'Bufanolides',\n",
       " 'Phodopus',\n",
       " 'Proteaceae',\n",
       " 'Pediococcus',\n",
       " 'Chloride Channels',\n",
       " 'Videodisc Recording',\n",
       " 'Cyclin-Dependent Kinase Inhibitor p16',\n",
       " 'Gestational Sac',\n",
       " 'Mass Media',\n",
       " 'rab3 GTP-Binding Proteins',\n",
       " 'Radiodermatitis',\n",
       " 'RGS Proteins',\n",
       " 'Pinealoma',\n",
       " 'Transgender Persons',\n",
       " 'Capsicum',\n",
       " 'Hemagglutination Tests',\n",
       " 'Heart Arrest',\n",
       " 'Follicular Phase',\n",
       " 'Speech Perception',\n",
       " 'Admitting Department, Hospital',\n",
       " 'Lumican',\n",
       " 'Liver Abscess, Pyogenic',\n",
       " \"Hutchinson's Melanotic Freckle\",\n",
       " 'Literacy',\n",
       " 'External Fixators',\n",
       " 'Bruch Membrane',\n",
       " 'Zinc Acetate',\n",
       " 'Ocimum basilicum',\n",
       " 'Interleukin-8',\n",
       " 'Receptors, Antigen, B-Cell',\n",
       " 'Australia',\n",
       " 'Kidney Tubular Necrosis, Acute',\n",
       " '17-Ketosteroids',\n",
       " 'Vitallium',\n",
       " 'Christianity',\n",
       " 'Choledochal Cyst',\n",
       " 'Prolactin',\n",
       " 'Bile Duct Neoplasms',\n",
       " 'Watchful Waiting',\n",
       " 'Hot Flashes',\n",
       " 'Oncogene Protein p65(gag-jun)',\n",
       " 'Methanobacteriaceae',\n",
       " 'Matrix Metalloproteinase 16',\n",
       " 'Cell-Free System',\n",
       " 'Virion',\n",
       " 'Genes, tat',\n",
       " 'Cranial Fossa, Posterior',\n",
       " 'Phosphodiesterase 3 Inhibitors',\n",
       " 'Cognition Disorders',\n",
       " 'RNA, Plant',\n",
       " 'Cyproterone Acetate',\n",
       " 'Adenosine Diphosphate',\n",
       " 'Oxalobacter formigenes',\n",
       " 'Cardiac Complexes, Premature',\n",
       " 'Coagulants',\n",
       " 'Dose Fractionation, Radiation',\n",
       " 'Persistent Hyperplastic Primary Vitreous',\n",
       " 'Encephalitis Viruses, Tick-Borne',\n",
       " 'Premenstrual Syndrome',\n",
       " 'Short Rib-Polydactyly Syndrome',\n",
       " 'Hyperargininemia',\n",
       " 'Aldehydes',\n",
       " 'Phosphatidylglycerols',\n",
       " 'Astroviridae Infections',\n",
       " 'Alveolar Process',\n",
       " 'Receptors, Tumor Necrosis Factor, Member 6b',\n",
       " 'Eye',\n",
       " 'Serial Extraction',\n",
       " 'Electric Injuries',\n",
       " 'Apansporoblastina',\n",
       " 'Confined Spaces',\n",
       " 'Peripheral Vascular Diseases',\n",
       " 'Occupational Health Nursing',\n",
       " 'Body Height',\n",
       " 'Legionellosis',\n",
       " 'Chlorpromazine',\n",
       " 'Myocardial Infarction',\n",
       " 'Pre-B-Cell Leukemia Transcription Factor 1',\n",
       " 'Anemia, Refractory, with Excess of Blasts',\n",
       " 'Athletes',\n",
       " 'Phosphorus Isotopes',\n",
       " 'Accidental Falls',\n",
       " 'Autophagy-Related Protein-1 Homolog',\n",
       " 'Immunophilins',\n",
       " 'Orbivirus',\n",
       " 'Sibling Relations',\n",
       " 'Wound Infection',\n",
       " 'Splanchnic Circulation',\n",
       " 'Amino Acids, Diamino',\n",
       " 'Radar',\n",
       " 'Disorders of Sex Development',\n",
       " 'Galanin-Like Peptide',\n",
       " 'Densitometry',\n",
       " 'Probenecid',\n",
       " 'Phosphothreonine',\n",
       " 'Methyl-CpG-Binding Protein 2',\n",
       " 'Magnetic Resonance Angiography',\n",
       " 'Cinnamates',\n",
       " 'Environmental Microbiology',\n",
       " 'Dental Care for Disabled',\n",
       " 'Cysteinyldopa',\n",
       " 'Transcription Factor TFIIIA',\n",
       " 'Glutaryl-CoA Dehydrogenase',\n",
       " 'Costameres',\n",
       " 'Paraganglioma',\n",
       " 'Transverse Sinuses',\n",
       " 'Gene Expression Regulation, Leukemic',\n",
       " 'CD59 Antigens',\n",
       " 'Choline Dehydrogenase',\n",
       " 'Stomach Ulcer',\n",
       " 'Lac Repressors',\n",
       " 'Optical Tweezers',\n",
       " 'SRS-A',\n",
       " 'Fosinopril',\n",
       " 'Prolyl Hydroxylases',\n",
       " 'Nipples',\n",
       " 'Phosphogluconate Dehydrogenase',\n",
       " 'Spatial Navigation',\n",
       " 'Genitalia',\n",
       " 'Managed Care Programs',\n",
       " 'Shoulder Joint',\n",
       " 'Empyema',\n",
       " 'Cardia',\n",
       " 'Thallium Radioisotopes',\n",
       " 'Cord Blood Stem Cell Transplantation',\n",
       " 'Echocardiography, Doppler, Color',\n",
       " 'Wortmannin',\n",
       " 'Receptors, Interleukin-4',\n",
       " 'Ethyl Chloride',\n",
       " 'North Sea',\n",
       " 'GB virus B',\n",
       " 'Waxes',\n",
       " 'Lipopolysaccharide Receptors',\n",
       " 'Oogonia',\n",
       " 'Endophthalmitis',\n",
       " 'Intracranial Embolism and Thrombosis',\n",
       " 'Adenosine A3 Receptor Agonists',\n",
       " 'Antibody Specificity',\n",
       " 'Calotropis',\n",
       " 'Receptors, Glucagon',\n",
       " 'Bone Morphogenetic Protein 7',\n",
       " 'Hemerythrin',\n",
       " 'Isotonic Contraction',\n",
       " 'Surgery, Veterinary',\n",
       " 'Cocarcinogenesis',\n",
       " 'Secretory Rate',\n",
       " 'Peri-Implantitis',\n",
       " 'Emergency Medical Dispatch',\n",
       " 'Amylin Receptor Agonists',\n",
       " 'Jaw Neoplasms',\n",
       " 'Neuroglia',\n",
       " 'Motor Neurons, Gamma',\n",
       " 'Glyburide',\n",
       " 'Acetyl-CoA C-Acetyltransferase',\n",
       " 'Progeria',\n",
       " 'Community Medicine',\n",
       " 'Time and Motion Studies',\n",
       " 'Biopterin',\n",
       " 'Arginine',\n",
       " 'Pylorus',\n",
       " 'Minichromosome Maintenance Proteins',\n",
       " 'Periapical Tissue',\n",
       " 'Lateral Ligament, Ankle',\n",
       " 'Nitroso Compounds',\n",
       " 'Congresses as Topic',\n",
       " 'Epispadias',\n",
       " 'Weapons of Mass Destruction',\n",
       " 'Betamethasone Valerate',\n",
       " 'Protein Kinase C beta',\n",
       " 'Alcoholics',\n",
       " 'Sulfobromophthalein',\n",
       " 'Rotavirus',\n",
       " 'Etorphine',\n",
       " 'Receptor Protein-Tyrosine Kinases',\n",
       " 'Receptors, Autocrine Motility Factor',\n",
       " 'RNA Polymerase I',\n",
       " 'RNA Interference',\n",
       " 'Staphylococcus Phages',\n",
       " 'Cell-Free Nucleic Acids',\n",
       " 'Hypoglycemia',\n",
       " 'Plasmodium vivax',\n",
       " 'Osteoblastoma',\n",
       " 'Aneurysm, Infected',\n",
       " 'Salivary Gland Calculi',\n",
       " 'Physician Executives',\n",
       " 'Pyoderma Gangrenosum',\n",
       " 'Mood Disorders',\n",
       " 'Analgesics, Opioid',\n",
       " 'Amino Acid Metabolism, Inborn Errors',\n",
       " 'Receptors, Kisspeptin-1',\n",
       " 'Immunoglobulin Subunits',\n",
       " 'Carcinoma, Lewis Lung',\n",
       " 'NF-E2 Transcription Factor, p45 Subunit',\n",
       " 'HEK293 Cells',\n",
       " 'Identity Crisis',\n",
       " 'Antibodies, Phospho-Specific',\n",
       " 'Spectrometry, X-Ray Emission',\n",
       " 'Antifoaming Agents',\n",
       " 'Respiratory Therapy Department, Hospital',\n",
       " 'Optical Storage Devices',\n",
       " 'Gangliosidosis, GM1',\n",
       " 'Adolescent Behavior',\n",
       " 'Factor XIIIa',\n",
       " 'Laboratories, Hospital',\n",
       " 'Endothelium, Lymphatic',\n",
       " 'Anthracenes',\n",
       " 'Amnesia, Transient Global',\n",
       " 'Penicillium',\n",
       " 'Shellfish',\n",
       " 'Light-Curing of Dental Adhesives',\n",
       " 'Dentate Gyrus',\n",
       " 'Aminopterin',\n",
       " 'Hemorrhoids',\n",
       " 'Anuria',\n",
       " 'Hybrid Cells',\n",
       " 'Rhodophyta',\n",
       " 'RNA, Transfer, Amino Acid-Specific',\n",
       " 'Pyruvic Acid',\n",
       " 'Dreissena',\n",
       " 'Middle East',\n",
       " 'Urinary Bladder Neck Obstruction',\n",
       " 'Lomustine',\n",
       " 'Speech Therapy',\n",
       " 'Natriuretic Peptides',\n",
       " ...}"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "set([item for sublist in [v['MESHES'] for k, v in data.items() if v['MESHES']] for item in sublist])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'18549508': {'QUESTION': 'Does systems analysis of quantitative shRNA-library screens identify regulators of cell adhesion?',\n",
       "  'CONTEXTS': ['High throughput screens with RNA interference technology enable loss-of-function analyses of gene activities in mammalian cells. While the construction of genome-scale shRNA libraries has been successful, results of large-scale screening of those libraries can be difficult to analyze because of the relatively high noise levels and the fact that not all shRNAs in a library are equally effective in silencing gene expression.',\n",
       "   'We have screened a library consisting of 43,828 shRNAs directed against 8,500 human genes for functions that are necessary in cell detachment induced by a constitutively activated c-Abl tyrosine kinase. To deal with the issues of noise and uncertainty of knockdown efficiencies, we employed an analytical strategy that combines quantitative data analysis with biological knowledge, i.e. Gene Ontology and pathway information, to increase the power of the RNAi screening technique. Using this strategy we found 16 candidate genes to be involved in Abl-induced disruption of cell adhesion, and verified that the knockdown of IL6ST is associated with enhanced cell attachment.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our results suggest that the power of genome-wide quantitative shRNA screens can be significantly increased when analyzed using a systems biology-based approach to identify functional gene networks.',\n",
       "  'MESHES': ['Cell Adhesion',\n",
       "   'Cell Line',\n",
       "   'Cell Separation',\n",
       "   'Cytokine Receptor gp130',\n",
       "   'Databases, Genetic',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Library',\n",
       "   'Humans',\n",
       "   'Proto-Oncogene Proteins c-abl',\n",
       "   'RNA Interference',\n",
       "   'RNA, Double-Stranded',\n",
       "   'Reproducibility of Results',\n",
       "   'Semantics',\n",
       "   'Signal Transduction',\n",
       "   'Systems Analysis',\n",
       "   'Systems Biology',\n",
       "   'Transcription, Genetic'],\n",
       "  'final_decision': 'yes'},\n",
       " '25960134': {'QUESTION': 'Does phenotype-dependent alteration of pathways and networks reveal a pure synergistic mechanism for compounds treating mouse cerebral ischemia?',\n",
       "  'CONTEXTS': ['Our previous studies have showed that ursodeoxycholic acid (UA) and jasminoidin (JA) effectively reduce cerebral infarct volume in mice. In this study we explored the pure synergistic mechanism of these compounds in treatment of mouse cerebral ischemia, which was defined as synergistic actions specific for phenotype variations after excluding interference from ineffective compounds.',\n",
       "   'Mice with focal cerebral ischemia were treated with UA, JA or a combination JA and UA (JU). Concha margaritifera (CM) was taken as ineffective compound. Cerebral infarct volume of the mice was determined, and the hippocampi were taken for microarray analysis. Particular signaling pathways and biological functions were enriched based on differentially expressed genes, and corresponding networks were constructed through Ingenuity Pathway Analysis.',\n",
       "   'In phenotype analysis, UA, JA, and JU significantly reduced the ischemic infarct volume with JU being superior to UA or JA alone, while CM was ineffective. As a result, 4 pathways enriched in CM were excluded. Core pathways in the phenotype-positive groups (UA or JA) were involved in neuronal homeostasis and neuropathology. JU-contributing pathways included all UA-contributing and the majority (71.7%) of JA-contributing pathways, and 10 new core pathways whose effects included inflammatory immunity, apoptosis and nervous system development. The functions of JU group included all functions of JA group, the majority (93.1%) of UA-contributing functions, and 3 new core functions, which focused on physiological system development and function.'],\n",
       "  'LABELS': ['OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The pure synergism between UA and JA underlies 10 new core pathways and 3 new core functions, which are involved in inflammation, immune responses, apoptosis and nervous system development.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Brain Ischemia',\n",
       "   'Data Mining',\n",
       "   'Databases, Genetic',\n",
       "   'Disease Models, Animal',\n",
       "   'Drug Combinations',\n",
       "   'Drug Synergism',\n",
       "   'Drugs, Chinese Herbal',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genomics',\n",
       "   'Male',\n",
       "   'Medicine, Chinese Traditional',\n",
       "   'Mice',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'Phenotype',\n",
       "   'Protein Interaction Maps',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology',\n",
       "   'Systems Integration'],\n",
       "  'final_decision': 'yes'},\n",
       " '20583741': {'QUESTION': 'Does biosimulation of inflammation and healing in surgically injured vocal fold?',\n",
       "  'CONTEXTS': ['The pathogenesis of vocal fold scarring is complex and remains to be deciphered. The current study is part of research endeavors aimed at applying systems biology approaches to address the complex biological processes involved in the pathogenesis of vocal fold scarring and other lesions affecting the larynx.',\n",
       "   'We developed a computational agent-based model (ABM) to quantitatively characterize multiple cellular and molecular interactions involved in inflammation and healing in vocal fold mucosa after surgical trauma. The ABM was calibrated with empirical data on inflammatory mediators (eg, tumor necrosis factor) and extracellular matrix components (eg, hyaluronan) from published studies on surgical vocal fold injury in the rat population.',\n",
       "   'The simulation results reproduced and predicted trajectories seen in the empirical data from the animals. Moreover, the ABM studies suggested that hyaluronan fragments might be the clinical surrogate of tissue damage, a key variable that in these simulations both is enhanced by and further induces inflammation.'],\n",
       "  'LABELS': ['OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'A relatively simple ABM such as the one reported in this study can provide new understanding of laryngeal wound healing and generate working hypotheses for further wet-lab studies.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Cicatrix',\n",
       "   'Extracellular Matrix',\n",
       "   'Male',\n",
       "   'Models, Biological',\n",
       "   'Mucous Membrane',\n",
       "   'Rats',\n",
       "   'Rats, Sprague-Dawley',\n",
       "   'Systems Biology',\n",
       "   'Vocal Cords',\n",
       "   'Wound Healing'],\n",
       "  'final_decision': 'yes'},\n",
       " '20976234': {'QUESTION': 'Are proteins involved in platelet signaling differentially regulated in acute coronary syndrome : a proteomic study?',\n",
       "  'CONTEXTS': ['Platelets play a fundamental role in pathological events underlying acute coronary syndrome (ACS). Because platelets do not have a nucleus, proteomics constitutes an optimal approach to follow platelet molecular events associated with the onset of the acute episode.',\n",
       "   'We performed the first high-resolution two-dimensional gel electrophoresis-based proteome analysis of circulating platelets from patients with non-ST segment elevation ACS (NSTE-ACS). Proteins were identified by mass spectrometry and validations were by western blotting. Forty protein features (corresponding to 22 unique genes) were found to be differentially regulated between NSTE-ACS patients and matched controls with chronic ischemic cardiopathy. The number of differences decreased at day 5 (28) and 6 months after the acute event (5). Interestingly, a systems biology approach demonstrated that 16 of the 22 differentially regulated proteins identified are interconnected as part of a common network related to cell assembly and organization and cell morphology, processes very related to platelet activation. Indeed, 14 of those proteins are either signaling or cytoskeletal, and nine of them are known to participate in platelet activation by αIIbβ3 and/or GPVI receptors. Several of the proteins identified participate in platelet activation through post-translational modifications, as shown here for ILK, Src and Talin. Interestingly, the platelet-secreted glycoprotein SPARC was down-regulated in NSTE-ACS patients compared to stable controls, which is consistent with a secretion process from activated platelets.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The present study provides novel information on platelet proteome changes associated with platelet activation in NSTE-ACS, highlighting the presence of proteins involved in platelet signaling. This investigation paves the way for future studies in the search for novel platelet-related biomarkers and drug targets in ACS.',\n",
       "  'MESHES': ['Acute Coronary Syndrome',\n",
       "   'Aged',\n",
       "   'Blood Proteins',\n",
       "   'Blotting, Western',\n",
       "   'Electrophoresis, Gel, Two-Dimensional',\n",
       "   'Female',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Mass Spectrometry',\n",
       "   'Middle Aged',\n",
       "   'Platelet Activation',\n",
       "   'Proteomics',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '27642133': {'QUESTION': 'Are gene expression profiles different in venous and capillary blood : Implications for vaccine studies?',\n",
       "  'CONTEXTS': ['Detailed analysis of the immunological pathways leading to robust vaccine responses has become possible with the application of systems biology, including transcriptomic analysis. Venous blood is usually obtained for such studies but others have obtained capillary blood (e.g. finger-prick). Capillary samples are practically advantageous, especially in children.',\n",
       "   'The aim of this study was to compare gene expression profiles in venous and capillary blood before, 12h and 24h after vaccination with 23-valent pneumococcal polysaccharide or trivalent inactivated seasonal influenza vaccines.',\n",
       "   'Gene expression at baseline was markedly different between venous and capillary samples, with 4940 genes differentially expressed, and followed a different pattern of changes after vaccination. At baseline, multiple pathways were upregulated in venous compared to capillary blood, including transforming growth factor-beta receptor signalling and toll-like receptor cascades. After vaccination with the influenza vaccine, there was enrichment for T and NK cell related signatures in capillary blood, and monocyte signatures in venous blood. By contrast, after vaccination with the pneumococcal vaccination, there was enrichment of dendritic cells, monocytes and interferon related signatures in capillary blood, whilst at 24h there was enrichment for T and NK cell related signatures in venous blood.'],\n",
       "  'LABELS': ['BACKGROUND', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'These data show differences between venous and capillary gene expression both at baseline, and post vaccination, which may impact on the conclusions regarding immunological mechanisms drawn from studies using these different sampling methodologies.',\n",
       "  'MESHES': ['Adolescent',\n",
       "   'Adult',\n",
       "   'Blood',\n",
       "   'Capillaries',\n",
       "   'Genomics',\n",
       "   'Hemagglutination Inhibition Tests',\n",
       "   'Humans',\n",
       "   'Influenza Vaccines',\n",
       "   'Interferons',\n",
       "   'Middle Aged',\n",
       "   'Pneumococcal Vaccines',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology',\n",
       "   'Transcriptome',\n",
       "   'Vaccines',\n",
       "   'Veins',\n",
       "   'Young Adult'],\n",
       "  'final_decision': 'yes'},\n",
       " '22502986': {'QUESTION': 'Are common genetic variants and gene-expression changes associated with bipolar disorder over-represented in brain signaling pathway genes?',\n",
       "  'CONTEXTS': ['Despite high heritability, the genetic variants influencing bipolar disorder (BD) susceptibility remain largely unknown. Low statistical power to detect the small effect-size alleles believed to underlie much of the genetic risk and possible heterogeneity between cohorts are an increasing concern. Integrative biology approaches might offer advantages over genetic analysis alone by combining different genomic datasets at the higher level of biological processes rather than the level of specific genetic variants or genes. We employed this strategy to identify biological processes involved in BD etiopathology.',\n",
       "   'Three genome-wide association studies and a brain gene-expression study were combined with the Human Protein Reference Database protein-protein interaction network data. We used bioinformatic analysis to search for biological networks with evidence of association on the basis of enrichment among both genetic and differential-expression associations with BD.',\n",
       "   'We identified association with gene networks involved in transmission of nerve impulse, Wnt, and Notch signaling. Three features stand out among these genes: 1) they localized to the human postsynaptic density, which is crucial for neuronal function; 2) their mouse knockouts present altered behavioral phenotypes; and 3) some are known targets of the pharmacological treatments for BD.'],\n",
       "  'LABELS': ['BACKGROUND', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Genetic and gene-expression associations of BD cluster in discrete regions of the protein-protein interaction network. We found replicated evidence for association for networks involving several interlinked signaling pathways. These genes are promising candidates to generate animal models and pharmacological interventions. Our results demonstrate the potential advantage of integrative biology analyses of BD datasets.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Bipolar Disorder',\n",
       "   'Brain',\n",
       "   'Gene Expression',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genome-Wide Association Study',\n",
       "   'Humans',\n",
       "   'Mice',\n",
       "   'Post-Synaptic Density',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '20616744': {'QUESTION': 'Does advanced approach to characterize the human intestinal microbiota by computational meta-analysis?',\n",
       "  'CONTEXTS': ['We describe advanced approaches for the computational meta-analysis of a collection of independent studies, including over 1000 phylogenetic array datasets, as a means to characterize the variability of human intestinal microbiota.',\n",
       "   'The human intestinal microbiota is a complex microbial community, consisting of several thousands of phylotypes, is specific for each individual, and impacts health and disease. We have developed a phylogenetic microarray, the Human Intestinal Tract Chip, to address the microbial diversity of the intestinal microbiota and used this tool to generate large datasets. It is of significant interest to use these datasets to be able to provide relations between microbial taxa, describe the extent and type of variability of the microbiota in the human gut, and establish relations between microbial taxa and their interaction with the host, intestinal location, or genotype.',\n",
       "   'We present an advanced computational meta-analysis approach for studying human intestinal microbiota, outline the advantages and disadvantages of such a meta-analysis, and reflect it to analogous approaches in other fields. Finally, we illustrate the potential of this meta-analysis by identifying salient signatures of site-specific microbial communities, describe impact of genotype, and provide first examples of relevant relations between microbial taxa.'],\n",
       "  'LABELS': ['OBJECTIVE', 'BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We are in the process of designing and applying appropriate methods for carrying out a full meta-analysis of the present data. Beyond that, the next large challenges in future meta-analyses lie in the integration of data from several heterogeneous measurement methods such as next generation sequencing techniques, metaproteomics, or metabolomics.',\n",
       "  'MESHES': ['Bacteria',\n",
       "   'Cluster Analysis',\n",
       "   'Feces',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation, Bacterial',\n",
       "   'Humans',\n",
       "   'Intestines',\n",
       "   'Metagenome',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'Phylogeny',\n",
       "   'Principal Component Analysis',\n",
       "   'Ribotyping',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '27557330': {'QUESTION': 'Do a genomics-based systems approach towards drug repositioning for rheumatoid arthritis?',\n",
       "  'CONTEXTS': ['Rheumatoid arthritis (RA) is a chronic autoimmune disease characterized by inflammation and destruction of synovial joints. RA affects up to 1 % of the population worldwide. Currently, there are no drugs that can cure RA or achieve sustained remission. The unknown cause of the disease represents a significant challenge in the drug development. In this study, we address this challenge by proposing an alternative drug discovery approach that integrates and reasons over genetic interrelationships between RA and other genetic diseases as well as a large amount of higher-level drug treatment data. We first constructed a genetic disease network using disease genetics data from Genome-Wide Association Studies (GWAS). We developed a network-based ranking algorithm to prioritize diseases genetically-related to RA (RA-related diseases). We then developed a drug prioritization algorithm to reposition drugs from RA-related diseases to treat RA.',\n",
       "   'Our algorithm found 74 of the 80 FDA-approved RA drugs and ranked them highly (recall: 0.925, median ranking: 8.93 %), demonstrating the validity of our strategy. When compared to a study that used GWAS data to directly connect RA-associated genes to drug targets (\"direct genetics-based\" approach), our algorithm (\"indirect genetics-based\") achieved a comparable overall performance, but complementary precision and recall in retrospective validation (precision: 0.22, recall: 0.36; F1: 0.27 vs. precision: 0.74, recall: 0.16; F1: 0.28). Our approach performed significantly better in novel predictions when evaluated using 165 not-yet-FDA-approved RA drugs (precision: 0.46, recall: 0.50; F1: 0.47 vs. precision: 0.40, recall: 0.006; F1: 0.01).'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'In summary, although the fundamental pathophysiological mechanisms remain uncharacterized, our proposed computation-based drug discovery approach to analyzing genetic and treatment interrelationships among thousands of diseases and drugs can facilitate the discovery of innovative drugs for treating RA.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Antirheumatic Agents',\n",
       "   'Arthritis, Rheumatoid',\n",
       "   'Drug Discovery',\n",
       "   'Genetic Predisposition to Disease',\n",
       "   'Genome, Human',\n",
       "   'Genome-Wide Association Study',\n",
       "   'Genomics',\n",
       "   'Humans',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '23539213': {'QUESTION': 'Does a systems biology framework identify molecular underpinnings of coronary heart disease?',\n",
       "  'CONTEXTS': ['Genetic approaches have identified numerous loci associated with coronary heart disease (CHD). The molecular mechanisms underlying CHD gene-disease associations, however, remain unclear. We hypothesized that genetic variants with both strong and subtle effects drive gene subnetworks that in turn affect CHD.',\n",
       "   'We surveyed CHD-associated molecular interactions by constructing coexpression networks using whole blood gene expression profiles from 188 CHD cases and 188 age- and sex-matched controls. Twenty-four coexpression modules were identified, including 1 case-specific and 1 control-specific differential module (DM). The DMs were enriched for genes involved in B-cell activation, immune response, and ion transport. By integrating the DMs with gene expression-associated single-nucleotide polymorphisms and with results of genome-wide association studies of CHD and its risk factors, the control-specific DM was implicated as CHD causal based on its significant enrichment for both CHD and lipid expression-associated single-nucleotide polymorphisms. This causal DM was further integrated with tissue-specific Bayesian networks and protein-protein interaction networks to identify regulatory key driver genes. Multitissue key drivers (SPIB and TNFRSF13C) and tissue-specific key drivers (eg, EBF1) were identified.'],\n",
       "  'LABELS': ['OBJECTIVE', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our network-driven integrative analysis not only identified CHD-related genes, but also defined network structure that sheds light on the molecular interactions of genes associated with CHD risk.',\n",
       "  'MESHES': ['Age Distribution',\n",
       "   'Aged',\n",
       "   'Bayes Theorem',\n",
       "   'Case-Control Studies',\n",
       "   'Coronary Artery Disease',\n",
       "   'Female',\n",
       "   'Gene Expression Regulation',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genetic Predisposition to Disease',\n",
       "   'Genetic Variation',\n",
       "   'Genome-Wide Association Study',\n",
       "   'Humans',\n",
       "   'Incidence',\n",
       "   'Male',\n",
       "   'Middle Aged',\n",
       "   'Polymorphism, Single Nucleotide',\n",
       "   'Reference Values',\n",
       "   'Risk Assessment',\n",
       "   'Sex Distribution',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '24970389': {'QUESTION': 'Do boolean ErbB network reconstructions and perturbation simulations reveal individual drug response in different breast cancer cell lines?',\n",
       "  'CONTEXTS': ['Despite promising progress in targeted breast cancer therapy, drug resistance remains challenging. The monoclonal antibody drugs trastuzumab and pertuzumab as well as the small molecule inhibitor erlotinib were designed to prevent ErbB-2 and ErbB-1 receptor induced deregulated protein signalling, contributing to tumour progression. The oncogenic potential of ErbB receptors unfolds in case of overexpression or mutations. Dimerisation with other receptors allows to bypass pathway blockades. Our intention is to reconstruct the ErbB network to reveal resistance mechanisms. We used longitudinal proteomic data of ErbB receptors and downstream targets in the ErbB-2 amplified breast cancer cell lines BT474, SKBR3 and HCC1954 treated with erlotinib, trastuzumab or pertuzumab, alone or combined, up to 60 minutes and 30 hours, respectively. In a Boolean modelling approach, signalling networks were reconstructed based on these data in a cell line and time course specific manner, including prior literature knowledge. Finally, we simulated network response to inhibitor combinations to detect signalling nodes reflecting growth inhibition.',\n",
       "   'The networks pointed to cell line specific activation patterns of the MAPK and PI3K pathway. In BT474, the PI3K signal route was favoured, while in SKBR3, novel edges highlighted MAPK signalling. In HCC1954, the inferred edges stimulated both pathways. For example, we uncovered feedback loops amplifying PI3K signalling, in line with the known trastuzumab resistance of this cell line. In the perturbation simulations on the short-term networks, we analysed ERK1/2, AKT and p70S6K. The results indicated a pathway specific drug response, driven by the type of growth factor stimulus. HCC1954 revealed an edgetic type of PIK3CA-mutation, contributing to trastuzumab inefficacy. Drug impact on the AKT and ERK1/2 signalling axes is mirrored by effects on RB and RPS6, relating to phenotypic events like cell growth or proliferation. Therefore, we additionally analysed RB and RPS6 in the long-term networks.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We derived protein interaction models for three breast cancer cell lines. Changes compared to the common reference network hint towards individual characteristics and potential drug resistance mechanisms. Simulation of perturbations were consistent with the experimental data, confirming our combined reverse and forward engineering approach as valuable for drug discovery and personalised medicine.',\n",
       "  'MESHES': ['Antineoplastic Agents',\n",
       "   'Antineoplastic Combined Chemotherapy Protocols',\n",
       "   'Breast Neoplasms',\n",
       "   'Cell Line, Tumor',\n",
       "   'ErbB Receptors',\n",
       "   'Erlotinib Hydrochloride',\n",
       "   'Humans',\n",
       "   'MAP Kinase Signaling System',\n",
       "   'Mitogen-Activated Protein Kinase 1',\n",
       "   'Mitogen-Activated Protein Kinase 3',\n",
       "   'Phosphatidylinositol 3-Kinases',\n",
       "   'Proto-Oncogene Proteins c-akt',\n",
       "   'Quinazolines',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology',\n",
       "   'Time Factors'],\n",
       "  'final_decision': 'yes'},\n",
       " '22640921': {'QUESTION': 'Does a systems biology approach reveal common metastatic pathways in osteosarcoma?',\n",
       "  'CONTEXTS': ['Osteosarcoma (OS) is the most common malignant bone tumor in children and adolescents. The survival rate of patients with metastatic disease remains very dismal. Nevertheless, metastasis is a complex process and a single-level analysis is not likely to identify its key biological determinants. In this study, we used a systems biology approach to identify common metastatic pathways that are jointly supported by both mRNA and protein expression data in two distinct human metastatic OS models.',\n",
       "   'mRNA expression microarray and N-linked glycoproteomic analyses were performed on two commonly used isogenic pairs of human metastatic OS cell lines, namely HOS/143B and SaOS-2/LM7. Pathway analysis of the differentially regulated genes and glycoproteins separately revealed pathways associated to metastasis including cell cycle regulation, immune response, and epithelial-to-mesenchymal-transition. However, no common significant pathway was found at both genomic and proteomic levels between the two metastatic models, suggesting a very different biological nature of the cell lines. To address this issue, we used a topological significance analysis based on a \"shortest-path\" algorithm to identify topological nodes, which uncovered additional biological information with respect to the genomic and glycoproteomic profiles but remained hidden from the direct analyses. Pathway analysis of the significant topological nodes revealed a striking concordance between the models and identified significant common pathways, including \"Cytoskeleton remodeling/TGF/WNT\", \"Cytoskeleton remodeling/Cytoskeleton remodeling\", and \"Cell adhesion/Chemokines and adhesion\". Of these, the \"Cytoskeleton remodeling/TGF/WNT\" was the top ranked common pathway from the topological analysis of the genomic and proteomic profiles in the two metastatic models. The up-regulation of proteins in the \"Cytoskeleton remodeling/TGF/WNT\" pathway in the SaOS-2/LM7 and HOS/143B models was further validated using an orthogonal Reverse Phase Protein Array platform.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'In this study, we used a systems biology approach by integrating genomic and proteomic data to identify key and common metastatic mechanisms in OS. The use of the topological analysis revealed hidden biological pathways that are known to play critical roles in metastasis. Wnt signaling has been previously implicated in OS and other tumors, and inhibitors of Wnt signaling pathways are available for clinical testing. Further characterization of this common pathway and other topological pathways identified from this study may lead to a novel therapeutic strategy for the treatment of metastatic OS.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Cell Line, Tumor',\n",
       "   'Cytoskeleton',\n",
       "   'Gene Expression Regulation, Neoplastic',\n",
       "   'Glycogen',\n",
       "   'Glycoproteins',\n",
       "   'Humans',\n",
       "   'Lung Neoplasms',\n",
       "   'Mice',\n",
       "   'Neoplasm Metastasis',\n",
       "   'Osteosarcoma',\n",
       "   'Proteomics',\n",
       "   'RNA, Messenger',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology',\n",
       "   'Transcriptome',\n",
       "   'Tumor Necrosis Factor-alpha',\n",
       "   'Wheat Germ Agglutinins',\n",
       "   'Wnt Proteins'],\n",
       "  'final_decision': 'yes'},\n",
       " '22631053': {'QUESTION': 'Do trade-offs between drug toxicity and benefit in the multi-antibiotic resistance system underlie optimal growth of E. coli?',\n",
       "  'CONTEXTS': ['Efflux is a widespread mechanism of reversible drug resistance in bacteria that can be triggered by environmental stressors, including many classes of drugs. While such chemicals when used alone are typically toxic to the cell, they can also induce the efflux of a broad range of agents and may therefore prove beneficial to cells in the presence of multiple stressors. The cellular response to a combination of such chemical stressors may be governed by a trade-off between the fitness costs due to drug toxicity and benefits mediated by inducible systems. Unfortunately, disentangling the cost-benefit interplay using measurements of bacterial growth in response to the competing effects of the drugs is not possible without the support of a theoretical framework.',\n",
       "   'Here, we use the well-studied multiple antibiotic resistance (MAR) system in E. coli to experimentally characterize the trade-off between drug toxicity (\"cost\") and drug-induced resistance (\"benefit\") mediated by efflux pumps. Specifically, we show that the combined effects of a MAR-inducing drug and an antibiotic are governed by a superposition of cost and benefit functions that govern these trade-offs. We find that this superposition holds for all drug concentrations, and it therefore allows us to describe the full dose-response diagram for a drug pair using simpler cost and benefit functions. Moreover, this framework predicts the existence of optimal growth at a non-trivial concentration of inducer. We demonstrate that optimal growth does not coincide with maximum induction of the mar promoter, but instead results from the interplay between drug toxicity and mar induction. Finally, we derived and experimentally validated a general phase diagram highlighting the role of these opposing effects in shaping the interaction between two drugs.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our analysis provides a quantitative description of the MAR system and highlights the trade-off between inducible resistance and the toxicity of the inducing agent in a multi-component environment. The results provide a predictive framework for the combined effects of drug toxicity and induction of the MAR system that are usually masked by bulk measurements of bacterial growth. The framework may also be useful for identifying optimal growth conditions in more general systems where combinations of environmental cues contribute to both transient resistance and toxicity.',\n",
       "  'MESHES': ['Anti-Bacterial Agents',\n",
       "   'Dose-Response Relationship, Drug',\n",
       "   'Drug Interactions',\n",
       "   'Drug Resistance, Bacterial',\n",
       "   'Drug Resistance, Multiple',\n",
       "   'Escherichia coli',\n",
       "   'Microbial Sensitivity Tests',\n",
       "   'Salicylates',\n",
       "   'Stress, Physiological',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '18062810': {'QUESTION': 'Does threshold-dominated regulation hide genetic variation in gene expression networks?',\n",
       "  'CONTEXTS': ['In dynamical models with feedback and sigmoidal response functions, some or all variables have thresholds around which they regulate themselves or other variables. A mathematical analysis has shown that when the dose-response functions approach binary or on/off responses, any variable with an equilibrium value close to one of its thresholds is very robust to parameter perturbations of a homeostatic state. We denote this threshold robustness. To check the empirical relevance of this phenomenon with response function steepnesses ranging from a near on/off response down to Michaelis-Menten conditions, we have performed a simulation study to investigate the degree of threshold robustness in models for a three-gene system with one downstream gene, using several logical input gates, but excluding models with positive feedback to avoid multistationarity. Varying parameter values representing functional genetic variation, we have analysed the coefficient of variation (CV) of the gene product concentrations in the stable state for the regulating genes in absolute terms and compared to the CV for the unregulating downstream gene. The sigmoidal or binary dose-response functions in these models can be considered as phenomenological models of the aggregated effects on protein or mRNA expression rates of all cellular reactions involved in gene expression.',\n",
       "   'For all the models, threshold robustness increases with increasing response steepness. The CVs of the regulating genes are significantly smaller than for the unregulating gene, in particular for steep responses. The effect becomes less prominent as steepnesses approach Michaelis-Menten conditions. If the parameter perturbation shifts the equilibrium value too far away from threshold, the gene product is no longer an effective regulator and robustness is lost. Threshold robustness arises when a variable is an active regulator around its threshold, and this function is maintained by the feedback loop that the regulator necessarily takes part in and also is regulated by. In the present study all feedback loops are negative, and our results suggest that threshold robustness is maintained by negative feedback which necessarily exists in the homeostatic state.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Threshold robustness of a variable can be seen as its ability to maintain an active regulation around its threshold in a homeostatic state despite external perturbations. The feedback loop that the system necessarily possesses in this state, ensures that the robust variable is itself regulated and kept close to its threshold. Our results suggest that threshold regulation is a generic phenomenon in feedback-regulated networks with sigmoidal response functions, at least when there is no positive feedback. Threshold robustness in gene regulatory networks illustrates that hidden genetic variation can be explained by systemic properties of the genotype-phenotype map.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Computer Simulation',\n",
       "   'Data Interpretation, Statistical',\n",
       "   'Feedback, Physiological',\n",
       "   'Gene Expression',\n",
       "   'Gene Expression Regulation',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genetic Variation',\n",
       "   'Humans',\n",
       "   'Logistic Models',\n",
       "   'Models, Genetic',\n",
       "   'Nonlinear Dynamics',\n",
       "   'Reference Values',\n",
       "   'Stochastic Processes',\n",
       "   'Systems Biology',\n",
       "   'Transcription Factors'],\n",
       "  'final_decision': 'yes'},\n",
       " '24376773': {'QUESTION': \"Does integrative network analysis unveil convergent molecular pathways in Parkinson 's disease and diabetes?\",\n",
       "  'CONTEXTS': [\"Shared dysregulated pathways may contribute to Parkinson's disease and type 2 diabetes, chronic diseases that afflict millions of people worldwide. Despite the evidence provided by epidemiological and gene profiling studies, the molecular and functional networks implicated in both diseases, have not been fully explored. In this study, we used an integrated network approach to investigate the extent to which Parkinson's disease and type 2 diabetes are linked at the molecular level.\",\n",
       "   \"Using a random walk algorithm within the human functional linkage network we identified a molecular cluster of 478 neighboring genes closely associated with confirmed Parkinson's disease and type 2 diabetes genes. Biological and functional analysis identified the protein serine-threonine kinase activity, MAPK cascade, activation of the immune response, and insulin receptor and lipid signaling as convergent pathways. Integration of results from microarrays studies identified a blood signature comprising seven genes whose expression is dysregulated in Parkinson's disease and type 2 diabetes. Among this group of genes, is the amyloid precursor protein (APP), previously associated with neurodegeneration and insulin regulation. Quantification of RNA from whole blood of 192 samples from two independent clinical trials, the Harvard Biomarker Study (HBS) and the Prognostic Biomarker Study (PROBE), revealed that expression of APP is significantly upregulated in Parkinson's disease patients compared to healthy controls. Assessment of biomarker performance revealed that expression of APP could distinguish Parkinson's disease from healthy individuals with a diagnostic accuracy of 80% in both cohorts of patients.\"],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': \"These results provide the first evidence that Parkinson's disease and diabetes are strongly linked at the molecular level and that shared molecular networks provide an additional source for identifying highly sensitive biomarkers. Further, these results suggest for the first time that increased expression of APP in blood may modulate the neurodegenerative phenotype in type 2 diabetes patients.\",\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Biomarkers',\n",
       "   'Diabetes Mellitus, Type 2',\n",
       "   'Female',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Middle Aged',\n",
       "   'Parkinson Disease',\n",
       "   'Prognosis',\n",
       "   'Stochastic Processes',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '21711513': {'QUESTION': 'Does systems analysis of multiple regulator perturbations allow discovery of virulence factors in Salmonella?',\n",
       "  'CONTEXTS': ['Systemic bacterial infections are highly regulated and complex processes that are orchestrated by numerous virulence factors. Genes that are coordinately controlled by the set of regulators required for systemic infection are potentially required for pathogenicity.',\n",
       "   'In this study we present a systems biology approach in which sample-matched multi-omic measurements of fourteen virulence-essential regulator mutants were coupled with computational network analysis to efficiently identify Salmonella virulence factors. Immunoblot experiments verified network-predicted virulence factors and a subset was determined to be secreted into the host cytoplasm, suggesting that they are virulence factors directly interacting with host cellular components. Two of these, SrfN and PagK2, were required for full mouse virulence and were shown to be translocated independent of either of the type III secretion systems in Salmonella or the type III injectisome-related flagellar mechanism.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Integrating multi-omic datasets from Salmonella mutants lacking virulence regulators not only identified novel virulence factors but also defined a new class of translocated effectors involved in pathogenesis. The success of this strategy at discovery of known and novel virulence factors suggests that the approach may have applicability for other bacterial pathogens.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Animals',\n",
       "   'Bacterial Proteins',\n",
       "   'Cytoplasm',\n",
       "   'Female',\n",
       "   'Gene Deletion',\n",
       "   'Gene Expression Profiling',\n",
       "   'Mice',\n",
       "   'Protein Transport',\n",
       "   'Proteomics',\n",
       "   'Reproducibility of Results',\n",
       "   'Salmonella',\n",
       "   'Sequence Homology, Amino Acid',\n",
       "   'Systems Biology',\n",
       "   'Virulence Factors'],\n",
       "  'final_decision': 'yes'},\n",
       " '17651484': {'QUESTION': 'Does nonlinear regulation enhance the phenotypic expression of trans-acting genetic polymorphisms?',\n",
       "  'CONTEXTS': ['Genetic variation explains a considerable part of observed phenotypic variation in gene expression networks. This variation has been shown to be located both locally (cis) and distally (trans) to the genes being measured. Here we explore to which degree the phenotypic manifestation of local and distant polymorphisms is a dynamic feature of regulatory design.',\n",
       "   'By combining mathematical models of gene expression networks with genetic maps and linkage analysis we find that very different network structures and regulatory motifs give similar cis/trans linkage patterns. However, when the shape of the cis-regulatory input functions is more nonlinear or threshold-like, we observe for all networks a dramatic increase in the phenotypic expression of distant compared to local polymorphisms under otherwise equal conditions.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our findings indicate that genetic variation affecting the form of cis-regulatory input functions may reshape the genotype-phenotype map by changing the relative importance of cis and trans variation. Our approach combining nonlinear dynamic models with statistical genetics opens up for a systematic investigation of how functional genetic variation is translated into phenotypic variation under various systemic conditions.',\n",
       "  'MESHES': ['Gene Expression Regulation',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genetic Linkage',\n",
       "   'Genotype',\n",
       "   'Models, Genetic',\n",
       "   'Phenotype',\n",
       "   'Polymorphism, Genetic',\n",
       "   'Quantitative Trait Loci',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '18400081': {'QUESTION': 'Does genomic context analysis in Archaea suggest previously unrecognized links between DNA replication and translation?',\n",
       "  'CONTEXTS': ['Comparative analysis of genomes is valuable to explore evolution of genomes, deduce gene functions, or predict functional linking between proteins. Here, we have systematically analyzed the genomic environment of all known DNA replication genes in 27 archaeal genomes to infer new connections for DNA replication proteins from conserved genomic associations.',\n",
       "   'Two distinct sets of DNA replication genes frequently co-localize in archaeal genomes: the first includes the genes for PCNA, the small subunit of the DNA primase (PriS), and Gins15; the second comprises the genes for MCM and Gins23. Other genomic associations of genes encoding proteins involved in informational processes that may be functionally relevant at the cellular level have also been noted; in particular, the association between the genes for PCNA, transcription factor S, and NudF. Surprisingly, a conserved cluster of genes coding for proteins involved in translation or ribosome biogenesis (S27E, L44E, aIF-2 alpha, Nop10) is almost systematically contiguous to the group of genes coding for PCNA, PriS, and Gins15. The functional relevance of this cluster encoding proteins conserved in Archaea and Eukarya is strongly supported by statistical analysis. Interestingly, the gene encoding the S27E protein, also known as metallopanstimulin 1 (MPS-1) in human, is overexpressed in multiple cancer cell lines.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our genome context analysis suggests specific functional interactions for proteins involved in DNA replication between each other or with proteins involved in DNA repair or transcription. Furthermore, it suggests a previously unrecognized regulatory network coupling DNA replication and translation in Archaea that may also exist in Eukarya.',\n",
       "  'MESHES': ['Archaea',\n",
       "   'DNA Repair',\n",
       "   'DNA Replication',\n",
       "   'Genome, Archaeal',\n",
       "   'Protein Binding',\n",
       "   'Protein Biosynthesis',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '23086299': {'QUESTION': 'Does systems biology analysis unravel the complementary action of combined rosuvastatin and ezetimibe therapy?',\n",
       "  'CONTEXTS': ['Combination-drug therapy takes advantage of the complementary action of their individual components, thereby potentiating its therapeutic effect. Potential disadvantages include side effects that are not foreseen on basis of the data available from drug monotherapy. Here, we used a systems biology approach to understand both the efficacy and the side effects of a cholesterol-lowering drug-combination therapy on the basis of the biological pathways and molecular processes affected by each drug alone or in combination.',\n",
       "   'ApoE*3Leiden transgenic mice, a mouse model with human-like cholesterol-lowering drug responses, were treated with rosuvastatin and ezetimibe, alone and in combination. Analyses included functional responses, viz. effects on cardiovascular risk factors, inflammation, and atherosclerosis, and measurement of global gene expression, and identification of enriched biological pathways and molecular processes. Combination therapy reduced plasma cholesterol, plasma inflammation markers, and atherosclerosis stronger than the single drugs did. Systems biology analysis at the level of biological processes shows that the therapeutic benefit of combined therapy is largely the result of additivity of the complementary mechanisms of action of the two single drugs. Importantly, combination therapy also exerted a significant effect on 16 additional and mostly NF-κB-linked signaling processes, 11 of which tended to be regulated in a similar direction with monotherapy.'],\n",
       "  'LABELS': ['OBJECTIVE', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'This study shows that gene expression analysis together with bioinformatics pathway analysis has the potential to help predict and identify drug combination-specific complementary and side effects.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Anticholesteremic Agents',\n",
       "   'Apolipoprotein E3',\n",
       "   'Atherosclerosis',\n",
       "   'Azetidines',\n",
       "   'Drug Therapy, Combination',\n",
       "   'Ezetimibe',\n",
       "   'Female',\n",
       "   'Fluorobenzenes',\n",
       "   'Mice',\n",
       "   'Mice, Transgenic',\n",
       "   'Pyrimidines',\n",
       "   'Risk Factors',\n",
       "   'Rosuvastatin Calcium',\n",
       "   'Sulfonamides',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '23068602': {'QUESTION': 'Does gene regulatory network analysis support inflammation as a key neurodegeneration process in prion disease?',\n",
       "  'CONTEXTS': ['The activation of immune cells in the brain is believed to be one of the earliest events in prion disease development, where misfolded PrionSc protein deposits are thought to act as irritants leading to a series of events that culminate in neuronal cell dysfunction and death. The role of these events in prion disease though is still a matter of debate. To elucidate the mechanisms leading from abnormal protein deposition to neuronal injury, we have performed a detailed network analysis of genes differentially expressed in several mouse prion models.',\n",
       "   'We found a master regulatory core of genes related to immune response controlling other genes involved in prion protein replication and accumulation, and neuronal cell death. This regulatory core determines the existence of two stable states that are consistent with the transcriptome analysis comparing prion infected versus uninfected mouse brain. An in silico perturbation analysis demonstrates that core genes are individually capable of triggering the transition and that the network remains locked once the diseased state is reached.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We hypothesize that this locking may be the cause of the sustained immune response observed in prion disease. Our analysis supports the hypothesis that sustained brain inflammation is the main pathogenic process leading to neuronal dysfunction and loss, which, in turn, leads to clinical symptoms in prion disease.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Cell Death',\n",
       "   'Disease Progression',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Inflammation',\n",
       "   'Mice',\n",
       "   'Neurons',\n",
       "   'Prion Diseases',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '21073692': {'QUESTION': 'Do in vitro gene regulatory networks predict in vivo function of liver?',\n",
       "  'CONTEXTS': ['Evolution of toxicity testing is predicated upon using in vitro cell based systems to rapidly screen and predict how a chemical might cause toxicity to an organ in vivo. However, the degree to which we can extend in vitro results to in vivo activity and possible mechanisms of action remains to be fully addressed.',\n",
       "   'Here we use the nitroaromatic 2,4,6-trinitrotoluene (TNT) as a model chemical to compare and determine how we might extrapolate from in vitro data to in vivo effects. We found 341 transcripts differentially expressed in common among in vitro and in vivo assays in response to TNT. The major functional term corresponding to these transcripts was cell cycle. Similarly modulated common pathways were identified between in vitro and in vivo. Furthermore, we uncovered the conserved common transcriptional gene regulatory networks between in vitro and in vivo cellular liver systems that responded to TNT exposure, which mainly contain 2 subnetwork modules: PTTG1 and PIR centered networks. Interestingly, all 7 genes in the PTTG1 module were involved in cell cycle and downregulated by TNT both in vitro and in vivo.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The results of our investigation of TNT effects on gene expression in liver suggest that gene regulatory networks obtained from an in vitro system can predict in vivo function and mechanisms. Inhibiting PTTG1 and its targeted cell cycle related genes could be key mechanism for TNT induced liver toxicity.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Cell Cycle',\n",
       "   'Cell Death',\n",
       "   'Cell Proliferation',\n",
       "   'Female',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Hepatocytes',\n",
       "   'Inactivation, Metabolic',\n",
       "   'Lipid Metabolism',\n",
       "   'Liver',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'Rats',\n",
       "   'Rats, Sprague-Dawley',\n",
       "   'Reproducibility of Results',\n",
       "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology',\n",
       "   'Time Factors',\n",
       "   'Toxicity Tests',\n",
       "   'Trinitrotoluene'],\n",
       "  'final_decision': 'yes'},\n",
       " '22851653': {'QUESTION': 'Is reperfusion-triggered stress protein response in the myocardium blocked by post-conditioning . Systems biology pathway analysis highlights the key role of the canonical aryl-hydrocarbon receptor pathway?',\n",
       "  'CONTEXTS': ['Ischaemic post-conditioning (IPost-Co) exerts cardioprotection by diminishing ischaemia/reperfusion injury. Yet, the mechanisms involved in such protection remain largely unknown. We have investigated the effects of IPost-Co in cardiac cells and in heart performance using molecular, proteomic and functional approaches.',\n",
       "   'Pigs underwent 1.5 h mid-left anterior descending balloon occlusion and then were sacrificed without reperfusion (ischaemia; n= 7), subjected to 2.5 h of cardiac reperfusion and sacrificed (n= 5); or subjected to IPost-Co before reperfusion and sacrificed 0.5 h (n= 4) and 2.5 h (n= 5) afterwards. A sham-operated group was included (n= 4). Ischaemic and non-ischaemic myocardium was obtained for molecular/histological analysis. Proteomic analysis was performed by two-dimensional electrophoresis followed by matrix-assisted laser desorption/ionization-time-of-flight identification. Potential protein networks involved were identified by bioinformatics and Ingenuity Pathway Analysis (IPA). Cardiac function was assessed by echocardiography. IPost-Co diminished (up to 2.5 h) reperfusion-induced apoptosis of both the intrinsic and extrinsic pathways whereas it did not affect reperfusion-induced Akt/mammalian target of rapamycin (mTOR)/P70S6K activation. Proteomic studies showed that IPost-Co reverted 43% of cardiac cytoplasmic protein changes observed during ischaemia and ischaemia + reperfusion. Systems biology assessment revealed significant changes in the aryl-hydrocarbon receptor (AhR) pathway (cell damage related). Bioinformatic data were confirmed since the expression of HSP90, AhR, ANRT, and β-tubulin (involved in AhR-signalling transduction) were accordingly modified after IPost-Co. IPost-Co rescued 52% of the left ventricle-at-risk compared with reperfusion alone and resulted in a ≈30% relative improvement in left ventricular ejection fraction (P <0.05).'],\n",
       "  'LABELS': ['OBJECTIVE', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'IPost-Co improves cardiac function post-myocardial infarction and reduces reperfusion-induced cell damage by down-regulation of the AhR-signalling transduction pathway ultimately leading to infarct size reduction.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Apoptosis',\n",
       "   'Aryl Hydrocarbon Receptor Nuclear Translocator',\n",
       "   'Caspase 8',\n",
       "   'Coronary Vessels',\n",
       "   'Deoxyuracil Nucleotides',\n",
       "   'Echocardiography',\n",
       "   'Heat-Shock Proteins',\n",
       "   'Ischemic Postconditioning',\n",
       "   'Myocardial Contraction',\n",
       "   'Myocardial Reperfusion Injury',\n",
       "   'Phosphatidylinositol 3-Kinases',\n",
       "   'Proteome',\n",
       "   'Proteomics',\n",
       "   'Proto-Oncogene Proteins c-bcl-2',\n",
       "   'Random Allocation',\n",
       "   'Receptors, Aryl Hydrocarbon',\n",
       "   'Signal Transduction',\n",
       "   'Sus scrofa',\n",
       "   'Systems Biology',\n",
       "   'TOR Serine-Threonine Kinases'],\n",
       "  'final_decision': 'yes'},\n",
       " '22186876': {'QUESTION': 'Is the TCA cycle required for selection or survival of multidrug-resistant Salmonella?',\n",
       "  'CONTEXTS': ['The initial aim of this study was to use a systems biology approach to analyse a ciprofloxacin-selected multidrug-resistant (MDR) Salmonella enterica serotype Typhimurium, L664.',\n",
       "   'The whole genome sequence and transcriptome of L664 were analysed. Site-directed mutagenesis to recreate each mutation was carried out, followed by phenotypic characterization and mutation frequency analysis. As a mutation in the TCA cycle was detected we tested the controversial hypothesis regarding the bacterial response to bactericidal antibiotics, put forward by Kohanski et al. (Cell 2007; 130: 797-810 and Mol Cell 2010; 37: 311-20), that exposure of bacteria to agents such as ciprofloxacin produces reactive oxygen species (ROS), which transiently increase the mutation rate giving rise to MDR bacteria.',\n",
       "   'L664 contained a mutation in ramR that conferred MDR. A mutation in tctA affected the TCA cycle and conferred the inability to grow on minimal agar. The virulence of L664 was not attenuated. Ciprofloxacin exposure produced ROS in L664 and SL1344 (tctA::aph), but it was reduced and occurred later. There were no significant differences in the rates of killing or mutations per generation to antibiotic resistance between the strains.'],\n",
       "  'LABELS': ['OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Whilst we confirm production of ROS in response to ciprofloxacin, we have no data to support the hypothesis that this leads to selection of MDR strains. Our results indicate that the mutations in tctA and glgA were random as they did not pre-exist in the parental strain, and that the mutation in tctA did not provide a survival advantage or disadvantage in the presence of antibiotic.',\n",
       "  'MESHES': ['Anti-Bacterial Agents',\n",
       "   'Ciprofloxacin',\n",
       "   'Citric Acid Cycle',\n",
       "   'Drug Resistance, Multiple, Bacterial',\n",
       "   'Humans',\n",
       "   'Microbial Viability',\n",
       "   'Mutagenesis, Site-Directed',\n",
       "   'Mutation',\n",
       "   'Reactive Oxygen Species',\n",
       "   'Salmonella typhimurium',\n",
       "   'Selection, Genetic',\n",
       "   'Systems Biology',\n",
       "   'Transcriptome'],\n",
       "  'final_decision': 'no'},\n",
       " '20651058': {'QUESTION': 'Does system-level analysis of neuroblastoma tumor-initiating cells implicate AURKB as a novel drug target for neuroblastoma?',\n",
       "  'CONTEXTS': ['Neuroblastoma (NB) is an aggressive tumor of the developing peripheral nervous system that remains difficult to cure in the advanced stages. The poor prognosis for high-risk NB patients is associated with common disease recurrences that fail to respond to available therapies. NB tumor-initiating cells (TICs), isolated from metastases and primary tumors, may escape treatment and contribute to tumor relapse. New therapies that target the TICs may therefore prevent or treat tumor recurrences.',\n",
       "   'We undertook a system-level characterization of NB TICs to identify potential drug targets against recurrent NB. We used next-generation RNA sequencing and/or human exon arrays to profile the transcriptomes of 11 NB TIC lines from six NB patients, revealing genes that are highly expressed in the TICs compared with normal neural crest-like cells and unrelated cancer tissues. We used gel-free two-dimensional liquid chromatography coupled to shotgun tandem mass spectrometry to confirm the presence of proteins corresponding to the most abundant TIC-enriched transcripts, thereby providing validation to the gene expression result.',\n",
       "   'Our study revealed that genes in the BRCA1 signaling pathway are frequently misexpressed in NB TICs and implicated Aurora B kinase as a potential drug target for NB therapy. Treatment with a selective AURKB inhibitor was cytotoxic to NB TICs but not to the normal neural crest-like cells.'],\n",
       "  'LABELS': ['OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'This work provides the first high-resolution system-level analysis of the transcriptomes of 11 primary human NB TICs and identifies a set of candidate NB TIC-enriched transcripts for further development as therapeutic targets.',\n",
       "  'MESHES': ['Antineoplastic Agents',\n",
       "   'Aurora Kinase B',\n",
       "   'Aurora Kinases',\n",
       "   'Chromatography, Liquid',\n",
       "   'Drug Evaluation, Preclinical',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation, Neoplastic',\n",
       "   'Gene Knockdown Techniques',\n",
       "   'Humans',\n",
       "   'Microarray Analysis',\n",
       "   'Molecular Targeted Therapy',\n",
       "   'Neoplastic Stem Cells',\n",
       "   'Neuroblastoma',\n",
       "   'Protein Kinase Inhibitors',\n",
       "   'Protein-Serine-Threonine Kinases',\n",
       "   'RNA, Small Interfering',\n",
       "   'Systems Biology',\n",
       "   'Tandem Mass Spectrometry',\n",
       "   'Therapies, Investigational',\n",
       "   'Validation Studies as Topic'],\n",
       "  'final_decision': 'yes'},\n",
       " '26208504': {'QUESTION': 'Do integrated systems approach identifies risk regulatory pathways and key regulators in coronary artery disease?',\n",
       "  'CONTEXTS': ['Coronary artery disease (CAD) is the most common type of heart disease. However, the molecular mechanisms of CAD remain elusive. Regulatory pathways are known to play crucial roles in many pathogenic processes. Thus, inferring risk regulatory pathways is an important step toward elucidating the mechanisms underlying CAD. With advances in high-throughput data, we developed an integrated systems approach to identify CAD risk regulatory pathways and key regulators. Firstly, a CAD-related core subnetwork was identified from a curated transcription factor (TF) and microRNA (miRNA) regulatory network based on a random walk algorithm. Secondly, candidate risk regulatory pathways were extracted from the subnetwork by applying a breadth-first search (BFS) algorithm. Then, risk regulatory pathways were prioritized based on multiple CAD-associated data sources. Finally, we also proposed a new measure to prioritize upstream regulators. We inferred that phosphatase and tensin homolog (PTEN) may be a key regulator in the dysregulation of risk regulatory pathways. This study takes a closer step than the identification of disease subnetworks or modules. From the risk regulatory pathways, we could understand the flow of regulatory information in the initiation and progression of the disease. Our approach helps to uncover its potential etiology.'],\n",
       "  'LABELS': ['UNLABELLED'],\n",
       "  'LONG_ANSWER': 'We developed an integrated systems approach to identify risk regulatory pathways. We proposed a new measure to prioritize the key regulators in CAD. PTEN may be a key regulator in dysregulation of the risk regulatory pathways.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Cluster Analysis',\n",
       "   'Computational Biology',\n",
       "   'Coronary Artery Disease',\n",
       "   'Datasets as Topic',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Humans',\n",
       "   'MicroRNAs',\n",
       "   'PTEN Phosphohydrolase',\n",
       "   'Protein Interaction Mapping',\n",
       "   'Protein Interaction Maps',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology',\n",
       "   'Transcription Factors'],\n",
       "  'final_decision': 'yes'},\n",
       " '23822816': {'QUESTION': 'Does large-scale integrative network-based analysis identify common pathways disrupted by copy number alterations across cancers?',\n",
       "  'CONTEXTS': ['Many large-scale studies analyzed high-throughput genomic data to identify altered pathways essential to the development and progression of specific types of cancer. However, no previous study has been extended to provide a comprehensive analysis of pathways disrupted by copy number alterations across different human cancers. Towards this goal, we propose a network-based method to integrate copy number alteration data with human protein-protein interaction networks and pathway databases to identify pathways that are commonly disrupted in many different types of cancer.',\n",
       "   'We applied our approach to a data set of 2,172 cancer patients across 16 different types of cancers, and discovered a set of commonly disrupted pathways, which are likely essential for tumor formation in majority of the cancers. We also identified pathways that are only disrupted in specific cancer types, providing molecular markers for different human cancers. Analysis with independent microarray gene expression datasets confirms that the commonly disrupted pathways can be used to identify patient subgroups with significantly different survival outcomes. We also provide a network view of disrupted pathways to explain how copy number alterations affect pathways that regulate cell growth, cycle, and differentiation for tumorigenesis.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'In this work, we demonstrated that the network-based integrative analysis can help to identify pathways disrupted by copy number alterations across 16 types of human cancers, which are not readily identifiable by conventional overrepresentation-based and other pathway-based methods. All the results and source code are available at http://compbio.cs.umn.edu/NetPathID/.',\n",
       "  'MESHES': ['DNA Copy Number Variations',\n",
       "   'Databases, Genetic',\n",
       "   'Genomics',\n",
       "   'Humans',\n",
       "   'Neoplasms',\n",
       "   'Protein Interaction Maps',\n",
       "   'Signal Transduction',\n",
       "   'Survival Analysis',\n",
       "   'Systems Biology',\n",
       "   'Transforming Growth Factor beta'],\n",
       "  'final_decision': 'yes'},\n",
       " '21533132': {'QUESTION': 'Does systems-scale analysis reveal pathways involved in cellular response to methamphetamine?',\n",
       "  'CONTEXTS': ['Methamphetamine (METH), an abused illicit drug, disrupts many cellular processes, including energy metabolism, spermatogenesis, and maintenance of oxidative status. However, many components of the molecular underpinnings of METH toxicity have yet to be established. Network analyses of integrated proteomic, transcriptomic and metabolomic data are particularly well suited for identifying cellular responses to toxins, such as METH, which might otherwise be obscured by the numerous and dynamic changes that are induced.',\n",
       "   'We used network analyses of proteomic and transcriptomic data to evaluate pathways in Drosophila melanogaster that are affected by acute METH toxicity. METH exposure caused changes in the expression of genes involved with energy metabolism, suggesting a Warburg-like effect (aerobic glycolysis), which is normally associated with cancerous cells. Therefore, we tested the hypothesis that carbohydrate metabolism plays an important role in METH toxicity. In agreement with our hypothesis, we observed that increased dietary sugars partially alleviated the toxic effects of METH. Our systems analysis also showed that METH impacted genes and proteins known to be associated with muscular homeostasis/contraction, maintenance of oxidative status, oxidative phosphorylation, spermatogenesis, iron and calcium homeostasis. Our results also provide numerous candidate genes for the METH-induced dysfunction of spermatogenesis, which have not been previously characterized at the molecular level.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our results support our overall hypothesis that METH causes a toxic syndrome that is characterized by the altered carbohydrate metabolism, dysregulation of calcium and iron homeostasis, increased oxidative stress, and disruption of mitochondrial functions.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Chromatography, High Pressure Liquid',\n",
       "   'Dietary Carbohydrates',\n",
       "   'Drosophila melanogaster',\n",
       "   'Electron Transport',\n",
       "   'Energy Metabolism',\n",
       "   'Gene Expression Profiling',\n",
       "   'Homeostasis',\n",
       "   'Male',\n",
       "   'Metabolomics',\n",
       "   'Methamphetamine',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'Oxidative Stress',\n",
       "   'Proteomics',\n",
       "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
       "   'Systems Biology',\n",
       "   'Tandem Mass Spectrometry',\n",
       "   'Trehalose'],\n",
       "  'final_decision': 'yes'},\n",
       " '21545728': {'QUESTION': 'Are transcriptional robustness and protein interactions associated in yeast?',\n",
       "  'CONTEXTS': ['Robustness to insults, both external and internal, is a characteristic feature of life. One level of biological organization for which noise and robustness have been extensively studied is gene expression. Cells have a variety of mechanisms for buffering noise in gene expression, but it is not completely clear what rules govern whether or not a given gene uses such tools to maintain appropriate expression.',\n",
       "   \"Here, we show a general association between the degree to which yeast cells have evolved mechanisms to buffer changes in gene expression and whether they possess protein-protein interactions. We argue that this effect bears an affinity to epistasis, because yeast appears to have evolved regulatory mechanisms such that distant changes in gene copy number for a protein-protein interaction partner gene can alter a gene's expression. This association is not unexpected given recent work linking epistasis and the deleterious effects of changes in gene dosage (i.e., the dosage balance hypothesis). Using gene expression data from artificial aneuploid strains of bakers' yeast, we found that genes coding for proteins that physically interact with other proteins show less expression variation in response to aneuploidy than do other genes. This effect is even more pronounced for genes whose products interact with proteins encoded on aneuploid chromosomes. We further found that genes targeted by transcription factors encoded on aneuploid chromosomes were more likely to change in expression after aneuploidy.\"],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We suggest that these observations can be best understood as resulting from the higher fitness cost of misexpression in epistatic genes and a commensurate greater regulatory control of them.',\n",
       "  'MESHES': ['Aneuploidy',\n",
       "   'Epistasis, Genetic',\n",
       "   'Fungal Proteins',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation, Fungal',\n",
       "   'Genes, Fungal',\n",
       "   'Models, Biological',\n",
       "   'Protein Interaction Mapping',\n",
       "   'RNA, Messenger',\n",
       "   'Regression Analysis',\n",
       "   'Saccharomyces cerevisiae',\n",
       "   'Systems Biology',\n",
       "   'Transcription, Genetic'],\n",
       "  'final_decision': 'yes'},\n",
       " '19371405': {'QUESTION': 'Does transcript length bias in RNA-seq data confound systems biology?',\n",
       "  'CONTEXTS': ['Several recent studies have demonstrated the effectiveness of deep sequencing for transcriptome analysis (RNA-seq) in mammals. As RNA-seq becomes more affordable, whole genome transcriptional profiling is likely to become the platform of choice for species with good genomic sequences. As yet, a rigorous analysis methodology has not been developed and we are still in the stages of exploring the features of the data.',\n",
       "   'We investigated the effect of transcript length bias in RNA-seq data using three different published data sets. For standard analyses using aggregated tag counts for each gene, the ability to call differentially expressed genes between samples is strongly associated with the length of the transcript.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Transcript length bias for calling differentially expressed genes is a general feature of current protocols for RNA-seq technology. This has implications for the ranking of differentially expressed genes, and in particular may introduce bias in gene set testing for pathway analysis and other multi-gene systems biology analyses.',\n",
       "  'MESHES': ['Base Sequence',\n",
       "   'Bias',\n",
       "   'Gene Expression Profiling',\n",
       "   'Humans',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'RNA, Messenger',\n",
       "   'Systems Biology',\n",
       "   'Transcription, Genetic'],\n",
       "  'final_decision': 'yes'},\n",
       " '25881257': {'QUESTION': 'Does bicluster Sampled Coherence Metric ( BSCM ) provide an accurate environmental context for phenotype predictions?',\n",
       "  'CONTEXTS': ['Biclustering is a popular method for identifying under which experimental conditions biological signatures are co-expressed. However, the general biclustering problem is NP-hard, offering room to focus algorithms on specific biological tasks. We hypothesize that conditional co-regulation of genes is a key factor in determining cell phenotype and that accurately segregating conditions in biclusters will improve such predictions. Thus, we developed a bicluster sampled coherence metric (BSCM) for determining which conditions and signals should be included in a bicluster.',\n",
       "   'Our BSCM calculates condition and cluster size specific p-values, and we incorporated these into the popular integrated biclustering algorithm cMonkey. We demonstrate that incorporation of our new algorithm significantly improves bicluster co-regulation scores (p-value = 0.009) and GO annotation scores (p-value = 0.004). Additionally, we used a bicluster based signal to predict whether a given experimental condition will result in yeast peroxisome induction. Using the new algorithm, the classifier accuracy improves from 41.9% to 76.1% correct.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We demonstrate that the proposed BSCM helps determine which signals ought to be co-clustered, resulting in more accurately assigned bicluster membership. Furthermore, we show that BSCM can be extended to more accurately detect under which experimental conditions the genes are co-clustered. Features derived from this more accurate analysis of conditional regulation results in a dramatic improvement in the ability to predict a cellular phenotype in yeast. The latest cMonkey is available for download at https://github.com/baliga-lab/cmonkey2. The experimental data and source code featured in this paper is available http://AitchisonLab.com/BSCM. BSCM has been incorporated in the official cMonkey release.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Cluster Analysis',\n",
       "   'Gene Expression Regulation',\n",
       "   'Phenotype',\n",
       "   'Pneumonia, Mycoplasma',\n",
       "   'Saccharomyces cerevisiae',\n",
       "   'Software',\n",
       "   'Systems Biology',\n",
       "   'Transcriptome'],\n",
       "  'final_decision': 'yes'},\n",
       " '23282319': {'QUESTION': 'Do module-based subnetwork alignments reveal novel transcriptional regulators in malaria parasite Plasmodium falciparum?',\n",
       "  'CONTEXTS': ['Malaria causes over one million deaths annually, posing an enormous health and economic burden in endemic regions. The completion of genome sequencing of the causative agents, a group of parasites in the genus Plasmodium, revealed potential drug and vaccine candidates. However, genomics-driven target discovery has been significantly hampered by our limited knowledge of the cellular networks associated with parasite development and pathogenesis. In this paper, we propose an approach based on aligning neighborhood PPI subnetworks across species to identify network components in the malaria parasite P. falciparum.',\n",
       "   'Instead of only relying on sequence similarities to detect functional orthologs, our approach measures the conservation between the neighborhood subnetworks in protein-protein interaction (PPI) networks in two species, P. falciparum and E. coli. 1,082 P. falciparum proteins were predicted as functional orthologs of known transcriptional regulators in the E. coli network, including general transcriptional regulators, parasite-specific transcriptional regulators in the ApiAP2 protein family, and other potential regulatory proteins. They are implicated in a variety of cellular processes involving chromatin remodeling, genome integrity, secretion, invasion, protein processing, and metabolism.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'In this proof-of-concept study, we demonstrate that a subnetwork alignment approach can reveal previously uncharacterized members of the subnetworks, which opens new opportunities to identify potential therapeutic targets and provide new insights into parasite biology, pathogenesis and virulence. This approach can be extended to other systems, especially those with poor genome annotation and a paucity of knowledge about cellular networks.',\n",
       "  'MESHES': ['Chromatin Assembly and Disassembly',\n",
       "   'Chromosome Mapping',\n",
       "   'Escherichia coli',\n",
       "   'Gene Expression Regulation',\n",
       "   'Genome, Protozoan',\n",
       "   'Genomics',\n",
       "   'Plasmodium falciparum',\n",
       "   'Protein Interaction Maps',\n",
       "   'Protozoan Proteins',\n",
       "   'Regulatory Elements, Transcriptional',\n",
       "   'Sequence Alignment',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '26586157': {'QUESTION': 'Does differential variability and correlation of gene expression identify key genes involved in neuronal differentiation?',\n",
       "  'CONTEXTS': ['Understanding the dynamics of stem cell differentiation processes at the molecular level is a central challenge in developmental biology and regenerative medicine. Although the dynamic behaviors of differentiation regulators have been partially characterized, the architecture regulating the underlying molecular systems remains unclear.',\n",
       "   'System-level analysis of transcriptional data was performed to characterize the dynamics of molecular networks in neural differentiation of stem cells. Expression of a network module of genes tightly co-expressed in mouse embryonic stem (ES) cells fluctuated greatly among cell populations before differentiation, but became stable following neural differentiation. During the neural differentiation process, genes exhibiting both differential variance and differential correlation between undifferentiated and differentiating states were related to developmental functions such as body axis development, neuronal movement, and transcriptional regulation. Furthermore, these genes were genetically associated with neuronal differentiation, providing support for the idea they are not only differentiation markers but could also play important roles in neural differentiation. Comparisons with transcriptional data from human induced pluripotent stem (iPS) cells revealed that the system of genes dynamically regulated during neural differentiation is conserved between mouse and human.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The results of this study provide a systematic analytical framework for identifying key genes involved in neural differentiation by detecting their dynamical behaviors, as well as a basis for understanding the dynamic molecular mechanisms underlying the processes of neural differentiation.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Cell Differentiation',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Humans',\n",
       "   'Induced Pluripotent Stem Cells',\n",
       "   'Mice',\n",
       "   'Models, Genetic',\n",
       "   'Mouse Embryonic Stem Cells',\n",
       "   'Neurons',\n",
       "   'Stem Cells',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '18620602': {'QUESTION': 'Do a systems approach to model natural variation in reactive properties of bacterial ribosomes?',\n",
       "  'CONTEXTS': ['Natural variation in protein output from translation in bacteria and archaea may be an organism-specific property of the ribosome. This paper adopts a systems approach to model the protein output as a measure of specific ribosome reactive properties in a ribosome-mediated translation apparatus. We use the steady-state assumption to define a transition state complex for the ribosome, coupled with mRNA, tRNA, amino acids and reaction factors, as a subsystem that allows a focus on the completed translational output as a measure of specific properties of the ribosome.',\n",
       "   'In analogy to the steady-state reaction of an enzyme complex, we propose a steady-state translation complex for mRNA from any gene, and derive a maximum specific translation activity, Ta(max), as a property of the ribosomal reaction complex. Ta(max) has units of a-protein output per time per a-specific mRNA. A related property of the ribosome, T a(max), has units of a-protein per time per total RNA with the relationshipT a(max) = rho(a)Ta(max), where rho(a) represents the fraction of total RNA committed to translation outpout of P(a) from gene a message. Ta(max) as a ribosome property is analogous to k(cat) for a purified enzyme, and Ta(max) is analogous to enzyme specific activity in a crude extract.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Analogy to an enzyme reaction complex led us to a ribosome reaction model for measuring specific translation activity of a bacterial ribosome. We propose to use this model to design experimental tests of our hypothesis that specific translation activity is a ribosomal property that is subject to natural variation and natural selection much like Vmax and Km for any specific enzyme.',\n",
       "  'MESHES': ['Adaptation, Biological',\n",
       "   'Bacteria',\n",
       "   'Catalysis',\n",
       "   'Kinetics',\n",
       "   'Models, Genetic',\n",
       "   'Protein Biosynthesis',\n",
       "   'RNA, Bacterial',\n",
       "   'RNA, Messenger',\n",
       "   'RNA, Transfer, Amino Acid-Specific',\n",
       "   'Ribosomal Proteins',\n",
       "   'Ribosomes',\n",
       "   'Species Specificity',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '24632444': {'QUESTION': 'Does network analysis reveal roles of inflammatory factors in different phenotypes of kidney transplant patients?',\n",
       "  'CONTEXTS': ['Systems-level characterization of inflammation in kidney transplantation remains incomplete. By stratifying kidney transplant patients based on phenotypes, the present study sought to identify the role of inflammatory proteins in disease progress and assess potential biomarkers for allograft monitoring.',\n",
       "   'Kidney transplant patients with different allograft status were enrolled in the study: stable renal function (ST), impaired renal function (UNST), acute rejection (AR), and chronic rejection (CR). We stratified the patients into 3 phenotype levels according to their symptoms and pathogenesis. Serum protein concentrations were measured by a quantitative protein array. All differentially expressed proteins were analyzed by protein-protein interaction networks (PPINs) to highlight protein interactions in patients with the above dysfunction levels. We identified level-related proteins and evaluated the classification efficiency of these biomarkers based on leave-one-out validation. The candidate proteins related to phenotype transforming were annotated by Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analysis.',\n",
       "   'Based on the hypothesis that common proteins and their up- or down-regulation promote disease progress, we obtained 12 common proteins and 11 level-specific proteins from the phenotype-related PPINs. The common proteins were annotated for KEGG enrichment: (1) cytokine-cytokine receptor interaction; (2) hematopoietic cell lineage; (3) Jak-STAT signaling pathway; (4) allograft rejection; and (5) T cell receptor signaling pathway. The level-specific proteins could be potential biomarkers with diagnostic value. The classification potency of the 11 level-specific proteins (IL-1R-1, IL-16, TIMP-1, G-CSF, MIG, IL-11, BLC, TNF-β, Eotaxin-2, I-309 and IL-6 sR) was better than the performance using all 40 proteins.'],\n",
       "  'LABELS': ['BACKGROUND', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The study demonstrated the potential value of PPINs-based approach to understanding inflammation-derived mechanisms and developing diagnostic biomarkers. Independent evaluations are required to further estimate the clinical relevance of the new diagnostic biomarkers.',\n",
       "  'MESHES': ['Adult',\n",
       "   'Antibodies',\n",
       "   'Biomarkers',\n",
       "   'Cell Lineage',\n",
       "   'Cell Proliferation',\n",
       "   'Cytokines',\n",
       "   'False Positive Reactions',\n",
       "   'Female',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation',\n",
       "   'Graft Rejection',\n",
       "   'HLA Antigens',\n",
       "   'Humans',\n",
       "   'Inflammation',\n",
       "   'Kidney Failure, Chronic',\n",
       "   'Kidney Transplantation',\n",
       "   'Male',\n",
       "   'Middle Aged',\n",
       "   'Phenotype',\n",
       "   'Protein Interaction Mapping',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '19604360': {'QUESTION': 'Do interactome and Gene Ontology provide congruent yet subtly different views of a eukaryotic cell?',\n",
       "  'CONTEXTS': ['The characterization of the global functional structure of a cell is a major goal in bioinformatics and systems biology. Gene Ontology (GO) and the protein-protein interaction network offer alternative views of that structure.',\n",
       "   'This study presents a comparison of the global structures of the Gene Ontology and the interactome of Saccharomyces cerevisiae. Sensitive, unsupervised methods of clustering applied to a large fraction of the proteome led to establish a GO-interactome correlation value of +0.47 for a general dataset that contains both high and low-confidence interactions and +0.58 for a smaller, high-confidence dataset.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The structures of the yeast cell deduced from GO and interactome are substantially congruent. However, some significant differences were also detected, which may contribute to a better understanding of cell function and also to a refinement of the current ontologies.',\n",
       "  'MESHES': ['Databases, Protein',\n",
       "   'Protein Interaction Mapping',\n",
       "   'Proteome',\n",
       "   'Ribonucleoproteins',\n",
       "   'Saccharomyces cerevisiae',\n",
       "   'Saccharomyces cerevisiae Proteins',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '22738223': {'QUESTION': 'Does a quantitative model of the initiation of DNA replication in Saccharomyces cerevisiae predict the effects of system perturbations?',\n",
       "  'CONTEXTS': [\"Eukaryotic cell proliferation involves DNA replication, a tightly regulated process mediated by a multitude of protein factors. In budding yeast, the initiation of replication is facilitated by the heterohexameric origin recognition complex (ORC). ORC binds to specific origins of replication and then serves as a scaffold for the recruitment of other factors such as Cdt1, Cdc6, the Mcm2-7 complex, Cdc45 and the Dbf4-Cdc7 kinase complex. While many of the mechanisms controlling these associations are well documented, mathematical models are needed to explore the network's dynamic behaviour. We have developed an ordinary differential equation-based model of the protein-protein interaction network describing replication initiation.\",\n",
       "   'The model was validated against quantified levels of protein factors over a range of cell cycle timepoints. Using chromatin extracts from synchronized Saccharomyces cerevisiae cell cultures, we were able to monitor the in vivo fluctuations of several of the aforementioned proteins, with additional data obtained from the literature. The model behaviour conforms to perturbation trials previously reported in the literature, and accurately predicts the results of our own knockdown experiments. Furthermore, we successfully incorporated our replication initiation model into an established model of the entire yeast cell cycle, thus providing a comprehensive description of these processes.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'This study establishes a robust model of the processes driving DNA replication initiation. The model was validated against observed cell concentrations of the driving factors, and characterizes the interactions between factors implicated in eukaryotic DNA replication. Finally, this model can serve as a guide in efforts to generate a comprehensive model of the mammalian cell cycle in order to explore cancer-related phenotypes.',\n",
       "  'MESHES': ['Calibration',\n",
       "   'Cell Cycle',\n",
       "   'DNA Replication',\n",
       "   'DNA, Fungal',\n",
       "   'Models, Biological',\n",
       "   'Saccharomyces cerevisiae',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '26415494': {'QUESTION': 'Does clustering reveal limits of parameter identifiability in multi-parameter models of biochemical dynamics?',\n",
       "  'CONTEXTS': ['Compared to engineering or physics problems, dynamical models in quantitative biology typically depend on a relatively large number of parameters. Progress in developing mathematics to manipulate such multi-parameter models and so enable their efficient interplay with experiments has been slow. Existing solutions are significantly limited by model size.',\n",
       "   'In order to simplify analysis of multi-parameter models a method for clustering of model parameters is proposed. It is based on a derived statistically meaningful measure of similarity between groups of parameters. The measure quantifies to what extend changes in values of some parameters can be compensated by changes in values of other parameters. The proposed methodology provides a natural mathematical language to precisely communicate and visualise effects resulting from compensatory changes in values of parameters. As a results, a relevant insight into identifiability analysis and experimental planning can be obtained. Analysis of NF-κB and MAPK pathway models shows that highly compensative parameters constitute clusters consistent with the network topology. The method applied to examine an exceptionally rich set of published experiments on the NF-κB dynamics reveals that the experiments jointly ensure identifiability of only 60% of model parameters. The method indicates which further experiments should be performed in order to increase the number of identifiable parameters.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We currently lack methods that simplify broadly understood analysis of multi-parameter models. The introduced tools depict mutually compensative effects between parameters to provide insight regarding role of individual parameters, identifiability and experimental design. The method can also find applications in related methodological areas of model simplification and parameters estimation.',\n",
       "  'MESHES': ['Cluster Analysis',\n",
       "   'Mitogen-Activated Protein Kinases',\n",
       "   'Models, Biological',\n",
       "   'NF-kappa B',\n",
       "   'Research Design',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '23033980': {'QUESTION': 'Do individuals with increased inflammatory response to ozone demonstrate muted signaling of immune cell trafficking pathways?',\n",
       "  'CONTEXTS': ['Exposure to ozone activates innate immune function and causes neutrophilic (PMN) airway inflammation that in some individuals is robustly elevated. The interplay between immuno-inflammatory function and genomic signaling in those with heightened inflammatory responsiveness to ozone is not well understood.',\n",
       "   'Determine baseline predictors and post exposure discriminators for the immuno-inflammatory response to ozone in inflammatory responsive adult volunteers.',\n",
       "   'Sputum induction was performed on 27 individuals before and after a two hour chamber exposure to 0.4\\u2009ppm ozone. Subjects were classified as inflammatory responders or non-responders to ozone based on their PMN response. Innate immune function, inflammatory cell and cytokine modulation and transcriptional signaling pathways were measured in sputum.',\n",
       "   'Post exposure, responders showed activated innate immune function (CD16: 31,004 MFI vs 8988 MFI; CD11b: 44,986 MFI vs 24,770 MFI; CD80: 2236 MFI vs 1506 MFI; IL-8: 37,603\\u2009pg/ml vs 2828\\u2009pg/ml; and IL-1β: 1380 pg/ml vs 318 pg/ml) with muted signaling of immune cell trafficking pathways. In contrast, non-responders displayed decreased innate immune activity (CD16, CD80; phagocytosis: 2 particles/PMN vs 4 particles/PMN) post exposure that was accompanied by a heightened signaling of immune cell trafficking pathways.'],\n",
       "  'LABELS': ['BACKGROUND', 'OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Inflammatory responsive and non responsive individuals to ozone show an inverse relationship between immune cell trafficking and immuno-inflammatory functional responses to ozone. These distinct genomic signatures may further our understanding about ozone-induced morbidity in individuals with different levels of inflammatory responsiveness.',\n",
       "  'MESHES': ['Adult',\n",
       "   'Air Pollutants',\n",
       "   'Asthma',\n",
       "   'Biomarkers',\n",
       "   'Cell Movement',\n",
       "   'Cytokines',\n",
       "   'Female',\n",
       "   'Gene Expression Profiling',\n",
       "   'Genetic Predisposition to Disease',\n",
       "   'Humans',\n",
       "   'Hypersensitivity, Immediate',\n",
       "   'Immunity, Innate',\n",
       "   'Inflammation',\n",
       "   'Inflammation Mediators',\n",
       "   'Inhalation Exposure',\n",
       "   'Lung',\n",
       "   'Macrophages',\n",
       "   'Male',\n",
       "   'Neutrophils',\n",
       "   'Ozone',\n",
       "   'Phagocytosis',\n",
       "   'Phenotype',\n",
       "   'Respiratory Burst',\n",
       "   'Signal Transduction',\n",
       "   'Sputum',\n",
       "   'Systems Biology',\n",
       "   'Time Factors',\n",
       "   'Transcription, Genetic',\n",
       "   'Young Adult'],\n",
       "  'final_decision': 'yes'},\n",
       " '26685285': {'QUESTION': 'Does a Systems Biology Approach reveal Converging Molecular Mechanisms that Link Different POPs to Common Metabolic Diseases?',\n",
       "  'CONTEXTS': ['A number of epidemiological studies have identified statistical associations between persistent organic pollutants (POPs) and metabolic diseases, but testable hypotheses regarding underlying molecular mechanisms to explain these linkages have not been published.',\n",
       "   'We assessed the underlying mechanisms of POPs that have been associated with metabolic diseases; three well-known POPs [2,3,7,8-tetrachlorodibenzodioxin (TCDD), 2,2´,4,4´,5,5´-hexachlorobiphenyl (PCB 153), and 4,4´-dichlorodiphenyldichloroethylene (p,p´-DDE)] were studied. We used advanced database search tools to delineate testable hypotheses and to guide laboratory-based research studies into underlying mechanisms by which this POP mixture could produce or exacerbate metabolic diseases.',\n",
       "   'For our searches, we used proprietary systems biology software (MetaCore™/MetaDrug™) to conduct advanced search queries for the underlying interactions database, followed by directional network construction to identify common mechanisms for these POPs within two or fewer interaction steps downstream of their primary targets. These common downstream pathways belong to various cytokine and chemokine families with experimentally well-documented causal associations with type 2 diabetes.'],\n",
       "  'LABELS': ['BACKGROUND', 'OBJECTIVE', 'METHODS'],\n",
       "  'LONG_ANSWER': 'Our systems biology approach allowed identification of converging pathways leading to activation of common downstream targets. To our knowledge, this is the first study to propose an integrated global set of step-by-step molecular mechanisms for a combination of three common POPs using a systems biology approach, which may link POP exposure to diseases. Experimental evaluation of the proposed pathways may lead to development of predictive biomarkers of the effects of POPs, which could translate into disease prevention and effective clinical treatment strategies.',\n",
       "  'MESHES': ['Biomarkers',\n",
       "   'Dichlorodiphenyl Dichloroethylene',\n",
       "   'Environmental Exposure',\n",
       "   'Environmental Pollutants',\n",
       "   'Humans',\n",
       "   'Metabolic Diseases',\n",
       "   'Organic Chemicals',\n",
       "   'Polychlorinated Biphenyls',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '20009967': {'QUESTION': 'Does systems biology guided by Chinese medicine reveal new markers for sub-typing rheumatoid arthritis patients?',\n",
       "  'CONTEXTS': ['Complex chronic diseases such as rheumatoid arthritis have become a major challenge in medicine and for the pharmaceutical industry. New impulses for drug development are needed.',\n",
       "   ': A systems biology approach is explored to find subtypes of rheumatoid arthritis patients enabling a development towards more personalized medicine.',\n",
       "   'Blood samples of 33 rheumatoid arthritis (RA) patients and 16 healthy volunteers were collected. The RA patients were diagnosed according to Chinese medicine (CM) theory and divided into 2 groups, the RA Heat and RA Cold group. CD4 T-cells were used for a total gene expression analysis. Metabolite profiles were measured in plasma using gas chromatography/mass spectrometry. Multivariate statistics was employed to find potential biomarkers for the RA Heat and RA Cold phenotype. A comprehensive biologic interpretation of the results is discussed.',\n",
       "   ': The genomics and metabolomics analysis showed statistically relevant different gene expression and metabolite profiles between healthy controls and RA patients as well as between the RA Heat and RA Cold group. Differences were found in the regulation of apoptosis. In the RA Heat group caspase 8 activated apoptosis seems to be stimulated while in the RA Cold group apoptosis seems to be suppressed through the Nrf2 pathway.'],\n",
       "  'LABELS': ['BACKGROUND', 'OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'RA patients could be divided in 2 groups according to CM theory. Molecular differences between the RA Cold and RA Heat groups were found which suggest differences in apoptotic activity. Subgrouping of patients according to CM diagnosis has the potential to provide opportunities for better treatment outcomes by targeting Western or CM treatment to specific groups of patients.',\n",
       "  'MESHES': ['Adult',\n",
       "   'Apoptosis',\n",
       "   'Arthralgia',\n",
       "   'Arthritis, Rheumatoid',\n",
       "   'Biomarkers',\n",
       "   'CD4-Positive T-Lymphocytes',\n",
       "   'Case-Control Studies',\n",
       "   'Caspase 8',\n",
       "   'Cold Temperature',\n",
       "   'Female',\n",
       "   'Hot Temperature',\n",
       "   'Humans',\n",
       "   'Medicine, Chinese Traditional',\n",
       "   'Middle Aged',\n",
       "   'NF-E2-Related Factor 2',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '22236809': {'QUESTION': 'Does biomarker robustness reveal the PDGF network as driving disease outcome in ovarian cancer patients in multiple studies?',\n",
       "  'CONTEXTS': ['Ovarian cancer causes more deaths than any other gynecological cancer. Identifying the molecular mechanisms that drive disease progress in ovarian cancer is a critical step in providing therapeutics, improving diagnostics, and affiliating clinical behavior with disease etiology. Identification of molecular interactions that stratify prognosis is key in facilitating a clinical-molecular perspective.',\n",
       "   'The Cancer Genome Atlas has recently made available the molecular characteristics of more than 500 patients. We used the TCGA multi-analysis study, and two additional datasets and a set of computational algorithms that we developed. The computational algorithms are based on methods that identify network alterations and quantify network behavior through gene expression.We identify a network biomarker that significantly stratifies survival rates in ovarian cancer patients. Interestingly, expression levels of single or sets of genes do not explain the prognostic stratification. The discovered biomarker is composed of the network around the PDGF pathway. The biomarker enables prognosis stratification.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The work presented here demonstrates, through the power of gene-expression networks, the criticality of the PDGF network in driving disease course. In uncovering the specific interactions within the network, that drive the phenotype, we catalyze targeted treatment, facilitate prognosis and offer a novel perspective into hidden disease heterogeneity.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Biomarkers',\n",
       "   'Computational Biology',\n",
       "   'DNA Copy Number Variations',\n",
       "   'DNA Methylation',\n",
       "   'Databases, Genetic',\n",
       "   'Female',\n",
       "   'Gene Expression Regulation, Neoplastic',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Humans',\n",
       "   'Ovarian Neoplasms',\n",
       "   'Platelet-Derived Growth Factor',\n",
       "   'Signal Transduction',\n",
       "   'Survival Analysis',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '24330479': {'QUESTION': 'Are multiple types of data required to identify the mechanisms influencing the spatial expansion of melanoma cell colonies?',\n",
       "  'CONTEXTS': ['The expansion of cell colonies is driven by a delicate balance of several mechanisms including cell motility, cell-to-cell adhesion and cell proliferation. New approaches that can be used to independently identify and quantify the role of each mechanism will help us understand how each mechanism contributes to the expansion process. Standard mathematical modelling approaches to describe such cell colony expansion typically neglect cell-to-cell adhesion, despite the fact that cell-to-cell adhesion is thought to play an important role.',\n",
       "   'We use a combined experimental and mathematical modelling approach to determine the cell diffusivity, D, cell-to-cell adhesion strength, q, and cell proliferation rate, λ, in an expanding colony of MM127 melanoma cells. Using a circular barrier assay, we extract several types of experimental data and use a mathematical model to independently estimate D, q and λ. In our first set of experiments, we suppress cell proliferation and analyse three different types of data to estimate D and q. We find that standard types of data, such as the area enclosed by the leading edge of the expanding colony and more detailed cell density profiles throughout the expanding colony, does not provide sufficient information to uniquely identify D and q. We find that additional data relating to the degree of cell-to-cell clustering is required to provide independent estimates of q, and in turn D. In our second set of experiments, where proliferation is not suppressed, we use data describing temporal changes in cell density to determine the cell proliferation rate. In summary, we find that our experiments are best described using the range D=161-243μm2 hour-1, q=0.3-0.5 (low to moderate strength) and λ=0.0305-0.0398 hour-1, and with these parameters we can accurately predict the temporal variations in the spatial extent and cell density profile throughout the expanding melanoma cell colony.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our systematic approach to identify the cell diffusivity, cell-to-cell adhesion strength and cell proliferation rate highlights the importance of integrating multiple types of data to accurately quantify the factors influencing the spatial expansion of melanoma cell colonies.',\n",
       "  'MESHES': ['Cell Adhesion',\n",
       "   'Cell Count',\n",
       "   'Cell Line, Tumor',\n",
       "   'Cell Movement',\n",
       "   'Cell Proliferation',\n",
       "   'Cell Size',\n",
       "   'Humans',\n",
       "   'Kinetics',\n",
       "   'Melanoma',\n",
       "   'Mitomycin',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '24330501': {'QUESTION': 'Does carbon catabolite repression correlate with the maintenance of near invariant molecular crowding in proliferating E. coli cells?',\n",
       "  'CONTEXTS': ['Carbon catabolite repression (CCR) is critical for optimal bacterial growth, and in bacterial (and yeast) cells it leads to their selective consumption of a single substrate from a complex environment. However, the root cause(s) for the development of this regulatory mechanism is unknown. Previously, a flux balance model (FBAwMC) of Escherichia coli metabolism that takes into account the crowded intracellular milieu of the bacterial cell correctly predicted selective glucose uptake in a medium containing five different carbon sources, suggesting that CCR may be an adaptive mechanism that ensures optimal bacterial metabolic network activity for growth.',\n",
       "   'Here, we show that slowly growing E. coli cells do not display CCR in a mixed substrate culture and gradual activation of CCR correlates with an increasing rate of E. coli cell growth and proliferation. In contrast, CCR mutant cells do not achieve fast growth in mixed substrate culture, and display differences in their cell volume and density compared to wild-type cells. Analyses of transcriptome data from wt E. coli cells indicate the expected regulation of substrate uptake and metabolic pathway utilization upon growth rate change. We also find that forced transient increase of intracellular crowding or transient perturbation of CCR delay cell growth, the latter leading to associated cell density-and volume alterations.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'CCR is activated at an increased bacterial cell growth rate when it is required for optimal cell growth while intracellular macromolecular density is maintained within a narrow physiological range. In addition to CCR, there are likely to be other regulatory mechanisms of cell metabolism that have evolved to ensure optimal cell growth in the context of the fundamental biophysical constraint imposed by intracellular molecular crowding.',\n",
       "  'MESHES': ['Biological Transport',\n",
       "   'Biomass',\n",
       "   'Carbon',\n",
       "   'Cell Count',\n",
       "   'Cell Culture Techniques',\n",
       "   'Cell Proliferation',\n",
       "   'Escherichia coli',\n",
       "   'Gene Expression Profiling',\n",
       "   'Models, Biological',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '17183729': {'QUESTION': 'Does a systems biology strategy reveal biological pathways and plasma biomarker candidates for potentially toxic statin-induced changes in muscle?',\n",
       "  'CONTEXTS': ['Aggressive lipid lowering with high doses of statins increases the risk of statin-induced myopathy. However, the cellular mechanisms leading to muscle damage are not known and sensitive biomarkers are needed to identify patients at risk of developing statin-induced serious side effects.',\n",
       "   'We performed bioinformatics analysis of whole genome expression profiling of muscle specimens and UPLC/MS based lipidomics analyses of plasma samples obtained in an earlier randomized trial from patients either on high dose simvastatin (80 mg), atorvastatin (40 mg), or placebo.',\n",
       "   'High dose simvastatin treatment resulted in 111 differentially expressed genes (1.5-fold change and p-value<0.05), while expression of only one and five genes was altered in the placebo and atorvastatin groups, respectively. The Gene Set Enrichment Analysis identified several affected pathways (23 gene lists with False Discovery Rate q-value<0.1) in muscle following high dose simvastatin, including eicosanoid synthesis and Phospholipase C pathways. Using lipidomic analysis we identified previously uncharacterized drug-specific changes in the plasma lipid profile despite similar statin-induced changes in plasma LDL-cholesterol. We also found that the plasma lipidomic changes following simvastatin treatment correlate with the muscle expression of the arachidonate 5-lipoxygenase-activating protein.'],\n",
       "  'LABELS': ['BACKGROUND', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'High dose simvastatin affects multiple metabolic and signaling pathways in skeletal muscle, including the pro-inflammatory pathways. Thus, our results demonstrate that clinically used high statin dosages may lead to unexpected metabolic effects in non-hepatic tissues. The lipidomic profiles may serve as highly sensitive biomarkers of statin-induced metabolic alterations in muscle and may thus allow us to identify patients who should be treated with a lower dose to prevent a possible toxicity.',\n",
       "  'MESHES': ['Atorvastatin',\n",
       "   'Biomarkers',\n",
       "   'Computational Biology',\n",
       "   'Gene Expression',\n",
       "   'Gene Expression Profiling',\n",
       "   'Heptanoic Acids',\n",
       "   'Humans',\n",
       "   'Hydroxymethylglutaryl-CoA Reductase Inhibitors',\n",
       "   'Lipids',\n",
       "   'Muscle, Skeletal',\n",
       "   'Muscular Diseases',\n",
       "   'Pyrroles',\n",
       "   'Simvastatin',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '25573024': {'QUESTION': 'Does systems biology with high-throughput sequencing reveal genetic mechanisms underlying the metabolic syndrome in the Lyon hypertensive rat?',\n",
       "  'CONTEXTS': ['The metabolic syndrome (MetS) is a collection of co-occurring complex disorders including obesity, hypertension, dyslipidemia, and insulin resistance. The Lyon hypertensive and Lyon normotensive rats are models of MetS sensitivity and resistance, respectively. To identify genetic determinants and mechanisms underlying MetS, an F2 intercross between Lyon hypertensive and Lyon normotensive was comprehensively studied.',\n",
       "   'Multidimensional data were obtained including genotypes of 1536 single-nucleotide polymorphisms, 23 physiological traits, and >150 billion nucleotides of RNA-seq reads from the livers of F2 intercross offspring and parental rats. Phenotypic and expression quantitative trait loci (eQTL) were mapped. Application of systems biology methods identified 17 candidate MetS genes. Several putative causal cis-eQTL were identified corresponding with phenotypic QTL loci. We found an eQTL hotspot on rat chromosome 17 that is causally associated with multiple MetS-related traits and found RGD1562963, a gene regulated in cis by this eQTL hotspot, as the most likely eQTL driver gene directly affected by genetic variation between Lyon hypertensive and Lyon normotensive rats.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our study sheds light on the intricate pathogenesis of MetS and demonstrates that systems biology with high-throughput sequencing is a powerful method to study the pathogenesis of complex genetic diseases.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Chromosomes, Mammalian',\n",
       "   'Hypertension',\n",
       "   'Metabolic Syndrome',\n",
       "   'Polymorphism, Single Nucleotide',\n",
       "   'Quantitative Trait Loci',\n",
       "   'Rats',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '23357979': {'QUESTION': 'Does a 12-gene set predict survival benefits from adjuvant chemotherapy in non-small cell lung cancer patients?',\n",
       "  'CONTEXTS': ['Prospectively identifying who will benefit from adjuvant chemotherapy (ACT) would improve clinical decisions for non-small cell lung cancer (NSCLC) patients. In this study, we aim to develop and validate a functional gene set that predicts the clinical benefits of ACT in NSCLC.',\n",
       "   'An 18-hub-gene prognosis signature was developed through a systems biology approach, and its prognostic value was evaluated in six independent cohorts. The 18-hub-gene set was then integrated with genome-wide functional (RNAi) data and genetic aberration data to derive a 12-gene predictive signature for ACT benefits in NSCLC.',\n",
       "   'Using a cohort of 442 stage I to III NSCLC patients who underwent surgical resection, we identified an 18-hub-gene set that robustly predicted the prognosis of patients with adenocarcinoma in all validation datasets across four microarray platforms. The hub genes, identified through a purely data-driven approach, have significant biological implications in tumor pathogenesis, including NKX2-1, Aurora Kinase A, PRC1, CDKN3, MBIP, and RRM2. The 12-gene predictive signature was successfully validated in two independent datasets (n = 90 and 176). The predicted benefit group showed significant improvement in survival after ACT (UT Lung SPORE data: HR = 0.34, P = 0.017; JBR.10 clinical trial data: HR = 0.36, P = 0.038), whereas the predicted nonbenefit group showed no survival benefit for 2 datasets (HR = 0.80, P = 0.70; HR = 0.91, P = 0.82).'],\n",
       "  'LABELS': ['OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'This is the first study to integrate genetic aberration, genome-wide RNAi data, and mRNA expression data to identify a functional gene set that predicts which resectable patients with non-small cell lung cancer will have a survival benefit with ACT.',\n",
       "  'MESHES': ['Adult',\n",
       "   'Aged',\n",
       "   'Carcinoma, Non-Small-Cell Lung',\n",
       "   'Chemotherapy, Adjuvant',\n",
       "   'Clinical Trials as Topic',\n",
       "   'Female',\n",
       "   'Gene Expression Regulation, Neoplastic',\n",
       "   'Genome, Human',\n",
       "   'Humans',\n",
       "   'Kaplan-Meier Estimate',\n",
       "   'Lung Neoplasms',\n",
       "   'Male',\n",
       "   'Middle Aged',\n",
       "   'Neoplasm Proteins',\n",
       "   'Neoplasm Staging',\n",
       "   'Prognosis',\n",
       "   'RNA Interference',\n",
       "   'Systems Biology',\n",
       "   'Treatment Outcome'],\n",
       "  'final_decision': 'yes'},\n",
       " '19171037': {'QUESTION': 'Does bifurcation analysis inform Bayesian inference in the Hes1 feedback loop?',\n",
       "  'CONTEXTS': ['Ordinary differential equations (ODEs) are an important tool for describing the dynamics of biological systems. However, for ODE models to be useful, their parameters must first be calibrated. Parameter estimation, that is, finding parameter values given experimental data, is an inference problem that can be treated systematically through a Bayesian framework.A Markov chain Monte Carlo approach can then be used to sample from the appropriate posterior probability distributions, provided that suitable prior distributions can be found for the unknown parameter values. Choosing these priors is therefore a vital first step in the inference process. We study here a negative feedback loop in gene regulation where an ODE incorporating a time delay has been proposed as a realistic model and where experimental data is available. Our aim is to show that a priori mathematical analysis can be exploited in the choice of priors.',\n",
       "   'By focussing on the onset of oscillatory behaviour through a Hopf Bifurcation, we derive a range of analytical expressions and constraints that link the model parameters to the observed dynamics of the system. Computational tests on both simulated and experimental data emphasise the usefulness of this analysis.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Mathematical analysis not only gives insights into the possible dynamical behaviour of gene expression models, but can also be used to inform the choice of priors when parameters are inferred from experimental data in a Bayesian setting.',\n",
       "  'MESHES': ['Basic Helix-Loop-Helix Transcription Factors',\n",
       "   'Bayes Theorem',\n",
       "   'Feedback, Physiological',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Models, Biological',\n",
       "   'RNA, Messenger',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '19171064': {'QUESTION': 'Do coordinations between gene modules control the operation of plant amino acid metabolic networks?',\n",
       "  'CONTEXTS': ['Being sessile organisms, plants should adjust their metabolism to dynamic changes in their environment. Such adjustments need particular coordination in branched metabolic networks in which a given metabolite can be converted into multiple other metabolites via different enzymatic chains. In the present report, we developed a novel \"Gene Coordination\" bioinformatics approach and use it to elucidate adjustable transcriptional interactions of two branched amino acid metabolic networks in plants in response to environmental stresses, using publicly available microarray results.',\n",
       "   'Using our \"Gene Coordination\" approach, we have identified in Arabidopsis plants two oppositely regulated groups of \"highly coordinated\" genes within the branched Asp-family network of Arabidopsis plants, which metabolizes the amino acids Lys, Met, Thr, Ile and Gly, as well as a single group of \"highly coordinated\" genes within the branched aromatic amino acid metabolic network, which metabolizes the amino acids Trp, Phe and Tyr. These genes possess highly coordinated adjustable negative and positive expression responses to various stress cues, which apparently regulate adjustable metabolic shifts between competing branches of these networks. We also provide evidence implying that these highly coordinated genes are central to impose intra- and inter-network interactions between the Asp-family and aromatic amino acid metabolic networks as well as differential system interactions with other growth promoting and stress-associated genome-wide genes.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our novel Gene Coordination elucidates that branched amino acid metabolic networks in plants are regulated by specific groups of highly coordinated genes that possess adjustable intra-network, inter-network and genome-wide transcriptional interactions. We also hypothesize that such transcriptional interactions enable regulatory metabolic adjustments needed for adaptation to the stresses.',\n",
       "  'MESHES': ['Amino Acids',\n",
       "   'Amino Acids, Aromatic',\n",
       "   'Arabidopsis',\n",
       "   'Aspartic Acid',\n",
       "   'Computational Biology',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genome, Plant',\n",
       "   'Metabolic Networks and Pathways',\n",
       "   'Models, Biological',\n",
       "   'Models, Genetic',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'Stress, Physiological',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '22014244': {'QUESTION': 'Do complex receptor-ligand dynamics control the response of the VEGF system to protease injury?',\n",
       "  'CONTEXTS': ['Vascular homeostasis and response to injury are dependent on the coordinated activity of growth factors such as vascular endothelial growth factor-A (VEGF). VEGF signaling is mediated by VEGF receptors 1 (VEGFR1) and 2 (VEGFR2). VEGF also binds to extracellular matrix (ECM) and neuropilin (NP), a cell surface glycoprotein that enhances VEGF binding to VEGFR2 while inhibiting VEGF-VEGFR1 interactions. Proteases such as neutrophil elastase release VEGF bound to ECM; however, this results in proteolytic processing of VEGF to a smaller species termed VEGF fragment (VEGFf). We hypothesized that the generation and presence of VEGFf would have significant effects on the binding distribution of VEGF.',\n",
       "   'We show that VEGFf, unlike VEGF, does not bind ECM, fibronectin, or NP-1. Using computational simulations, we find that excess VEGFf can lead to increased binding of VEGF to VEGFR2 through VEGFf binding to VEGFR1 and subsequent liberation of NP-1. We show experimentally that VEGF-induced migration has a biphasic response to conversion of VEGF to VEGFf. Simulations suggest that a simple change in VEGFR1 or VEGFR2 complexes are unlikely to be responsible and that a more complex integration of signals is more likely involved.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'These findings suggest that proteolytic damage at sites of tissue injury and inflammation has the potential to modulate the VEGF system through a complex process and highlight the need for quantitative analysis to reveal mechanisms of growth factor control.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Binding Sites',\n",
       "   'Cattle',\n",
       "   'Cell Movement',\n",
       "   'Cells, Cultured',\n",
       "   'Computer Simulation',\n",
       "   'Extracellular Matrix',\n",
       "   'Fibronectins',\n",
       "   'Homeostasis',\n",
       "   'Humans',\n",
       "   'Ligands',\n",
       "   'Models, Biological',\n",
       "   'Neuropilins',\n",
       "   'Pancreatic Elastase',\n",
       "   'Protein Interaction Maps',\n",
       "   'Receptors, Vascular Endothelial Growth Factor',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology',\n",
       "   'Vascular Endothelial Growth Factor A'],\n",
       "  'final_decision': 'yes'},\n",
       " '19531207': {'QUESTION': 'Do serial evolutionary networks of within-patient HIV-1 sequences reveal patterns of evolution of X4 strains?',\n",
       "  'CONTEXTS': ['The HIV virus is known for its ability to exploit numerous genetic and evolutionary mechanisms to ensure its proliferation, among them, high replication, mutation and recombination rates. Sliding MinPD, a recently introduced computational method 1, was used to investigate the patterns of evolution of serially-sampled HIV-1 sequence data from eight patients with a special focus on the emergence of X4 strains. Unlike other phylogenetic methods, Sliding MinPD combines distance-based inference with a nonparametric bootstrap procedure and automated recombination detection to reconstruct the evolutionary history of longitudinal sequence data. We present serial evolutionary networks as a longitudinal representation of the mutational pathways of a viral population in a within-host environment. The longitudinal representation of the evolutionary networks was complemented with charts of clinical markers to facilitate correlation analysis between pertinent clinical information and the evolutionary relationships.',\n",
       "   'Analysis based on the predicted networks suggests the following:: significantly stronger recombination signals (p = 0.003) for the inferred ancestors of the X4 strains, recombination events between different lineages and recombination events between putative reservoir virus and those from a later population, an early star-like topology observed for four of the patients who died of AIDS. A significantly higher number of recombinants were predicted at sampling points that corresponded to peaks in the viral load levels (p = 0.0042).'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our results indicate that serial evolutionary networks of HIV sequences enable systematic statistical analysis of the implicit relations embedded in the topology of the structure and can greatly facilitate identification of patterns of evolution that can lead to specific hypotheses and new insights. The conclusions of applying our method to empirical HIV data support the conventional wisdom of the new generation HIV treatments, that in order to keep the virus in check, viral loads need to be suppressed to almost undetectable levels.',\n",
       "  'MESHES': ['Base Sequence',\n",
       "   'Databases, Genetic',\n",
       "   'Evolution, Molecular',\n",
       "   'HIV-1',\n",
       "   'Humans',\n",
       "   'Mutation',\n",
       "   'Phylogeny',\n",
       "   'Recombination, Genetic',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '23782925': {'QUESTION': 'Does tissue factor induce human coronary artery smooth muscle cell motility through Wnt-signalling?',\n",
       "  'CONTEXTS': ['Tissue factor (TF) is the most relevant physiological trigger of thrombosis contributing to the presentation of clinical ischemic events after plaque rupture. However, the role of human vascular smooth muscle cell (HVSMC) TF in vascular remodeling, restenosis and atherosclerosis is less known. We have hypothesized that TF contributes to atherosclerotic lesion formation, triggering smooth muscle cell migration through a specific yet unknown signaling pathway.',\n",
       "   'The aim of this study has been to investigate the signal transduction mechanism by which TF may contribute to the transition of resident static contractile HVSMC into a migrating cell that promotes atherosclerotic plaque progression.',\n",
       "   'We have used a system biology discovery approach with gene-engineered HVSMCs to identify genes/proteins involved in the TF-triggered effects in HVSMC obtained from the coronary arteries of human adult hearts.',\n",
       "   'Analysis of wild-type HVSMC (TF(+) ) and TF(-) silenced HVSMC (TF(-) ) showed that TF is involved in the regulation of Wnt signaling and in the expression of downstream proteins that affect the atherosclerotic process.'],\n",
       "  'LABELS': ['BACKGROUND', 'OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': \"The 'in silico' analysis pointed to specific Wnt-pathway proteins that have been validated in cell culture and also have been found expressed in human advanced atherosclerotic plaques but not in early lesions. TF signals through Wnt to regulate coronary smooth muscle cell migration and vascular remodeling.\",\n",
       "  'MESHES': ['Cell Movement',\n",
       "   'Cells, Cultured',\n",
       "   'Coronary Vessels',\n",
       "   'Humans',\n",
       "   'Muscle, Smooth, Vascular',\n",
       "   'Real-Time Polymerase Chain Reaction',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology',\n",
       "   'Thromboplastin',\n",
       "   'Wnt Proteins'],\n",
       "  'final_decision': 'yes'},\n",
       " '18842138': {'QUESTION': \"Do variations in the transcriptome of Alzheimer 's disease reveal molecular networks involved in cardiovascular diseases?\",\n",
       "  'CONTEXTS': [\"Because of its polygenic nature, Alzheimer's disease is believed to be caused not by defects in single genes, but rather by variations in a large number of genes and their complex interactions. A systems biology approach, such as the generation of a network of co-expressed genes and the identification of functional modules and cis-regulatory elements, to extract insights and knowledge from microarray data will lead to a better understanding of complex diseases such as Alzheimer's disease. In this study, we perform a series of analyses using co-expression networks, cis-regulatory elements, and functions of co-expressed gene modules to analyze single-cell gene expression data from normal and Alzheimer's disease-affected subjects.\",\n",
       "   \"We identified six co-expressed gene modules, each of which represented a biological process perturbed in Alzheimer's disease. Alzheimer's disease-related genes, such as APOE, A2M, PON2 and MAP4, and cardiovascular disease-associated genes, including COMT, CBS and WNK1, all congregated in a single module. Some of the disease-related genes were hub genes while many of them were directly connected to one or more hub genes. Further investigation of this disease-associated module revealed cis-regulatory elements that match to the binding sites of transcription factors involved in Alzheimer's disease and cardiovascular disease.\"],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': \"Our results show the extensive links between Alzheimer's disease and cardiovascular disease at the co-expression and co-regulation levels, providing further evidence for the hypothesis that cardiovascular disease and Alzheimer's disease are linked. Our results support the notion that diseases in which the same set of biochemical pathways are affected may tend to co-occur with each other.\",\n",
       "  'MESHES': ['Aged, 80 and over',\n",
       "   'Alzheimer Disease',\n",
       "   'Brain-Derived Neurotrophic Factor',\n",
       "   'Cardiovascular Diseases',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genetic Variation',\n",
       "   'Humans',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '19032789': {'QUESTION': 'Does genetic and systems level analysis of Drosophila sticky/citron kinase and dFmr1 mutants reveal common regulation of genetic networks?',\n",
       "  'CONTEXTS': ['In Drosophila, the genes sticky and dFmr1 have both been shown to regulate cytoskeletal dynamics and chromatin structure. These genes also genetically interact with Argonaute family microRNA regulators. Furthermore, in mammalian systems, both genes have been implicated in neuronal development. Given these genetic and functional similarities, we tested Drosophila sticky and dFmr1 for a genetic interaction and measured whole genome expression in both mutants to assess similarities in gene regulation.',\n",
       "   'We found that sticky mutations can dominantly suppress a dFmr1 gain-of-function phenotype in the developing eye, while phenotypes produced by RNAi knock-down of sticky were enhanced by dFmr1 RNAi and a dFmr1 loss-of-function mutation. We also identified a large number of transcripts that were misexpressed in both mutants suggesting that sticky and dFmr1 gene products similarly regulate gene expression. By integrating gene expression data with a protein-protein interaction network, we found that mutations in sticky and dFmr1 resulted in misexpression of common gene networks, and consequently predicted additional specific phenotypes previously not known to be associated with either gene. Further phenotypic analyses validated these predictions.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'These findings establish a functional link between two previously unrelated genes. Microarray analysis indicates that sticky and dFmr1 are both required for regulation of many developmental genes in a variety of cell types. The diversity of transcripts regulated by these two genes suggests a clear cause of the pleiotropy that sticky and dFmr1 mutants display and provides many novel, testable hypotheses about the functions of these genes. As both of these genes are implicated in the development and function of the mammalian brain, these results have relevance to human health as well as to understanding more general biological processes.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Cell Polarity',\n",
       "   'Drosophila',\n",
       "   'Drosophila Proteins',\n",
       "   'Eye',\n",
       "   'Female',\n",
       "   'Fragile X Mental Retardation Protein',\n",
       "   'Gene Knockdown Techniques',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Intracellular Signaling Peptides and Proteins',\n",
       "   'Male',\n",
       "   'Mutation',\n",
       "   'Oocytes',\n",
       "   'Phenotype',\n",
       "   'Protein-Serine-Threonine Kinases',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '19032748': {'QUESTION': 'Does the integrated analysis of metabolic and protein interaction networks reveal novel molecular organizing principles?',\n",
       "  'CONTEXTS': ['The study of biological interaction networks is a central theme of systems biology. Here, we investigate the relationships between two distinct types of interaction networks: the metabolic pathway map and the protein-protein interaction network (PIN). It has long been established that successive enzymatic steps are often catalyzed by physically interacting proteins forming permanent or transient multi-enzymes complexes. Inspecting high-throughput PIN data, it was shown recently that, indeed, enzymes involved in successive reactions are generally more likely to interact than other protein pairs. In our study, we expanded this line of research to include comparisons of the underlying respective network topologies as well as to investigate whether the spatial organization of enzyme interactions correlates with metabolic efficiency.',\n",
       "   'Analyzing yeast data, we detected long-range correlations between shortest paths between proteins in both network types suggesting a mutual correspondence of both network architectures. We discovered that the organizing principles of physical interactions between metabolic enzymes differ from the general PIN of all proteins. While physical interactions between proteins are generally dissortative, enzyme interactions were observed to be assortative. Thus, enzymes frequently interact with other enzymes of similar rather than different degree. Enzymes carrying high flux loads are more likely to physically interact than enzymes with lower metabolic throughput. In particular, enzymes associated with catabolic pathways as well as enzymes involved in the biosynthesis of complex molecules were found to exhibit high degrees of physical clustering. Single proteins were identified that connect major components of the cellular metabolism and may thus be essential for the structural integrity of several biosynthetic systems.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our results reveal topological equivalences between the protein interaction network and the metabolic pathway network. Evolved protein interactions may contribute significantly towards increasing the efficiency of metabolic processes by permitting higher metabolic fluxes. Thus, our results shed further light on the unifying principles shaping the evolution of both the functional (metabolic) as well as the physical interaction network.',\n",
       "  'MESHES': ['Fungal Proteins',\n",
       "   'Metabolic Networks and Pathways',\n",
       "   'Protein Binding',\n",
       "   'Protein Interaction Mapping',\n",
       "   'Systems Biology',\n",
       "   'Yeasts'],\n",
       "  'final_decision': 'yes'},\n",
       " '20084294': {'QUESTION': 'Does p300 alter keratinocyte cell growth and differentiation through regulation of p21 ( Waf1/CIP1 )?',\n",
       "  'CONTEXTS': ['p300 functions as a transcriptional co-activator to regulate many cellular responses such as cell growth, transformation, development and differentiation. It has been shown to affect the transcriptional activity of p53 which regulates p21(Waf1/CIP1) expression, however, the role of p300 in differentiation remains unclear.',\n",
       "   'Knockdown of p300 protein with short hairpin RNA (shRNA) molecules delays human neonatal foreskin keratinocyte (HFKs) differentiation. Moreover, depletion of p300 increases the proliferative capacity of HFKs, extends the life span of cells and allows differentiated HFKs to re-enter the cell cycle. Studies indicate that depletion of p300 down-regulates the acetylation and expression of p53, and chromatin immunoprecipitation (ChIP) analysis shows that induction of p21(Waf1/CIP1) in early differentiation is a result of p300 dependent activation of p53 and that depletion of p21(Waf1/CIP1) results in the delay of differentiation and a phenotype similar to p300 depletion.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'p300 has a direct role in the control of cell growth and differentiation in primary epithelial cells, and p21(Waf1/CIP1) is an important mediator of these p300 functions.',\n",
       "  'MESHES': ['Cell Differentiation',\n",
       "   'Cell Division',\n",
       "   'Cyclin-Dependent Kinase Inhibitor p21',\n",
       "   'Humans',\n",
       "   'Systems Biology',\n",
       "   'p300-CBP Transcription Factors'],\n",
       "  'final_decision': 'yes'},\n",
       " '19389738': {'QUESTION': 'Does systematic analysis of synchronized oscillatory neuronal networks reveal an enrichment for coupled direct and indirect feedback motifs?',\n",
       "  'CONTEXTS': ['Synchronized bursting behavior is a remarkable phenomenon in neural dynamics. So, identification of the underlying functional structure is crucial to understand its regulatory mechanism at a system level. On the other hand, we noted that feedback loops (FBLs) are commonly used basic building blocks in engineering circuit design, especially for synchronization, and they have also been considered as important regulatory network motifs in systems biology. From these motivations, we have investigated the relationship between synchronized bursting behavior and feedback motifs in neural networks.',\n",
       "   'Through extensive simulations of synthetic spike oscillation models, we found that a particular structure of FBLs, coupled direct and indirect positive feedback loops (PFLs), can induce robust synchronized bursting behaviors. To further investigate this, we have developed a novel FBL identification method based on sampled time-series data and applied it to synchronized spiking records measured from cultured neural networks of rat by using multi-electrode array. As a result, we have identified coupled direct and indirect PFLs.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We therefore conclude that coupled direct and indirect PFLs might be an important design principle that causes the synchronized bursting behavior in neuronal networks although an extrapolation of this result to in vivo brain dynamics still remains an unanswered question.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Computer Simulation',\n",
       "   'Neural Networks (Computer)',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '25329661': {'QUESTION': 'Do systems biology studies of adult paragonimus lung flukes facilitate the identification of immunodominant parasite antigens?',\n",
       "  'CONTEXTS': ['Paragonimiasis is a food-borne trematode infection acquired by eating raw or undercooked crustaceans. It is a major public health problem in the far East, but it also occurs in South Asia, Africa, and in the Americas. Paragonimus worms cause chronic lung disease with cough, fever and hemoptysis that can be confused with tuberculosis or other non-parasitic diseases. Treatment is straightforward, but diagnosis is often delayed due to a lack of reliable parasitological or serodiagnostic tests. Hence, the purpose of this study was to use a systems biology approach to identify key parasite proteins that may be useful for development of improved diagnostic tests.',\n",
       "   'The transcriptome of adult Paragonimus kellicotti was sequenced with Illumina technology. Raw reads were pre-processed and assembled into 78,674 unique transcripts derived from 54,622 genetic loci, and 77,123 unique protein translations were predicted. A total of 2,555 predicted proteins (from 1,863 genetic loci) were verified by mass spectrometric analysis of total worm homogenate, including 63 proteins lacking homology to previously characterized sequences. Parasite proteins encoded by 321 transcripts (227 genetic loci) were reactive with antibodies from infected patients, as demonstrated by immunoaffinity purification and high-resolution liquid chromatography-mass spectrometry. Serodiagnostic candidates were prioritized based on several criteria, especially low conservation with proteins in other trematodes. Cysteine proteases, MFP6 proteins and myoglobins were abundant among the immunoreactive proteins, and these warrant further study as diagnostic candidates.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The transcriptome, proteome and immunolome of adult P. kellicotti represent a major advance in the study of Paragonimus species. These data provide a powerful foundation for translational research to develop improved diagnostic tests. Similar integrated approaches may be useful for identifying novel targets for drugs and vaccines in the future.',\n",
       "  'MESHES': ['Adult',\n",
       "   'Africa',\n",
       "   'Animals',\n",
       "   'Antigens, Helminth',\n",
       "   'Asia',\n",
       "   'Base Sequence',\n",
       "   'Female',\n",
       "   'Fever',\n",
       "   'Foodborne Diseases',\n",
       "   'Humans',\n",
       "   'Immunodominant Epitopes',\n",
       "   'Male',\n",
       "   'Paragonimiasis',\n",
       "   'Paragonimus',\n",
       "   'Proteome',\n",
       "   'Sequence Analysis, DNA',\n",
       "   'Systems Biology',\n",
       "   'Transcriptome',\n",
       "   'Tuberculosis'],\n",
       "  'final_decision': 'yes'},\n",
       " '17953764': {'QUESTION': 'Does a systems-biology analysis of isogenic megakaryocytic and granulocytic cultures identify new molecular components of megakaryocytic apoptosis?',\n",
       "  'CONTEXTS': ['The differentiation of hematopoietic stem cells into platelet-forming megakaryocytes is of fundamental importance to hemostasis. Constitutive apoptosis is an integral, yet poorly understood, facet of megakaryocytic (Mk) differentiation. Understanding Mk apoptosis could lead to advances in the treatment of Mk and platelet disorders.',\n",
       "   'We used a Gene-ontology-driven microarray-based transcriptional analysis coupled with protein-level and activity assays to identify genes and pathways involved in Mk apoptosis. Peripheral blood CD34+ hematopoietic progenitor cells were induced to either Mk differentiation or, as a negative control without observable apoptosis, granulocytic differentiation. Temporal gene-expression data were analyzed by a combination of intra- and inter-culture comparisons in order to identify Mk-associated genes. This novel approach was first applied to a curated set of general Mk-related genes in order to assess their dynamic transcriptional regulation. When applied to all apoptosis associated genes, it revealed a decrease in NF-kappaB signaling, which was explored using phosphorylation assays for IkappaBalpha and p65 (RELA). Up-regulation was noted among several pro-apoptotic genes not previously associated with Mk apoptosis such as components of the p53 regulon and TNF signaling. Protein-level analyses probed the involvement of the p53-regulated GADD45A, and the apoptosis signal-regulating kinase 1 (ASK1). Down-regulation of anti-apoptotic genes, including several of the Bcl-2 family, was also detected.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our comparative approach to analyzing dynamic large-scale transcriptional data, which was validated using a known set of Mk genes, robustly identified candidate Mk apoptosis genes. This led to novel insights into the molecular mechanisms regulating apoptosis in Mk cells.',\n",
       "  'MESHES': ['Antigens, CD34',\n",
       "   'Apoptosis',\n",
       "   'Apoptosis Regulatory Proteins',\n",
       "   'Cell Differentiation',\n",
       "   'Cells, Cultured',\n",
       "   'Cluster Analysis',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation',\n",
       "   'Granulocytes',\n",
       "   'Hematopoiesis',\n",
       "   'Humans',\n",
       "   'Megakaryocytes',\n",
       "   'Models, Biological',\n",
       "   'NF-kappa B',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'Signal Transduction',\n",
       "   'Stem Cells',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '26961692': {'QUESTION': 'Do multiplex methods provide effective integration of multi-omic data in genome-scale models?',\n",
       "  'CONTEXTS': ['Genomic, transcriptomic, and metabolic variations shape the complex adaptation landscape of bacteria to varying environmental conditions. Elucidating the genotype-phenotype relation paves the way for the prediction of such effects, but methods for characterizing the relationship between multiple environmental factors are still lacking. Here, we tackle the problem of extracting network-level information from collections of environmental conditions, by integrating the multiple omic levels at which the bacterial response is measured.',\n",
       "   'To this end, we model a large compendium of growth conditions as a multiplex network consisting of transcriptomic and fluxomic layers, and we propose a multi-omic network approach to infer similarity of growth conditions by integrating layers of the multiplex network. Each node of the network represents a single condition, while edges are similarities between conditions, as measured by phenotypic and transcriptomic properties on different layers of the network. We then fuse these layers into one network, therefore capturing a global network of conditions and the associated similarities across two omic levels. We apply this multi-omic fusion to an updated genome-scale reconstruction of Escherichia coli that includes underground metabolism and new gene-protein-reaction associations.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our method can be readily used to evaluate and cross-compare different collections of conditions among different species. Acquiring multi-omic information on the topology of the space of experimental conditions makes it possible to infer the position and to build condition-specific models of untested or incomplete profiles for which experimental data is not available. Our weighted network fusion method for genome-scale models is freely available at https://github.com/maxconway/SNFtool .',\n",
       "  'MESHES': ['Adaptation, Physiological',\n",
       "   'Bacterial Proteins',\n",
       "   'Environment',\n",
       "   'Escherichia coli',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genome, Bacterial',\n",
       "   'Genomics',\n",
       "   'Metabolic Networks and Pathways',\n",
       "   'Models, Biological',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '25935274': {'QUESTION': 'Are scavenger receptor class A member 5 ( SCARA5 ) and suprabasin ( SBSN ) hub genes of coexpression network modules associated with peripheral vein graft patency?',\n",
       "  'CONTEXTS': ['Approximately 30% of autogenous vein grafts develop luminal narrowing and fail because of intimal hyperplasia or negative remodeling. We previously found that vein graft cells from patients who later develop stenosis proliferate more in\\xa0vitro in response to growth factors than cells from patients who maintain patent grafts. To discover novel determinants of vein graft outcome, we have analyzed gene expression profiles of these cells using a systems biology approach to cluster the genes into modules by their coexpression patterns and to correlate the results with growth data from our prior study and with new studies of migration and matrix remodeling.',\n",
       "   'RNA from 4-hour serum- or platelet-derived growth factor (PDGF)-BB-stimulated human saphenous vein cells obtained from the outer vein wall (20 cell lines) was used for microarray analysis of gene expression, followed by weighted gene coexpression network analysis. Cell migration in microchemotaxis chambers in response to PDGF-BB and cell-mediated collagen gel contraction in response to serum were also determined. Gene function was determined using short-interfering RNA to inhibit gene expression before subjecting cells to growth or collagen gel contraction assays. These cells were derived from samples of the vein grafts obtained at surgery, and the long-term fate of these bypass grafts was known.',\n",
       "   'Neither migration nor cell-mediated collagen gel contraction showed a correlation with graft outcome. Although 1188 and 1340 genes were differentially expressed in response to treatment with serum and PDGF, respectively, no single gene was differentially expressed in cells isolated from patients whose grafts stenosed compared with those that remained patent. Network analysis revealed four unique groups of genes, which we term modules, associated with PDGF responses, and 20 unique modules associated with serum responses. The \"yellow\" and \"skyblue\" modules, from PDGF and serum analyses, respectively, correlated with later graft stenosis (P\\xa0= .005 and P\\xa0= .02, respectively). In response to PDGF, yellow was also associated with increased cell growth. For serum, skyblue was also associated with inhibition of collagen gel contraction. The hub genes for yellow and skyblue (ie, the gene most connected to other genes in the module), scavenger receptor class A member 5 (SCARA5) and suprabasin (SBSN), respectively, were tested for effects on proliferation and collagen contraction. Knockdown of SCARA5 increased proliferation by 29.9%\\xa0± 7.8% (P\\xa0< .01), whereas knockdown of SBSN had no effect. Knockdown of SBSN increased collagen gel contraction by 24.2%\\xa0± 8.6% (P\\xa0< .05), whereas knockdown of SCARA5 had no effect.'],\n",
       "  'LABELS': ['OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Using weighted gene coexpression network analysis of cultured vein graft cell gene expression, we have discovered two small gene modules, which comprise 42 genes, that are associated with vein graft failure. Further experiments are needed to delineate the venous cells that express these genes in\\xa0vivo and the roles these genes play in vein graft healing, starting with the module hub genes SCARA5 and SBSN, which have been shown to have modest effects on cell proliferation or collagen gel contraction.',\n",
       "  'MESHES': ['Antigens, Differentiation',\n",
       "   'Becaplermin',\n",
       "   'Cell Line',\n",
       "   'Cell Movement',\n",
       "   'Cell Proliferation',\n",
       "   'Cluster Analysis',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genetic Predisposition to Disease',\n",
       "   'Graft Occlusion, Vascular',\n",
       "   'Humans',\n",
       "   'Hyperplasia',\n",
       "   'Neointima',\n",
       "   'Neoplasm Proteins',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'Phenotype',\n",
       "   'Proto-Oncogene Proteins c-sis',\n",
       "   'RNA Interference',\n",
       "   'Risk Factors',\n",
       "   'Scavenger Receptors, Class A',\n",
       "   'Systems Biology',\n",
       "   'Transfection',\n",
       "   'Treatment Outcome',\n",
       "   'Vascular Grafting',\n",
       "   'Vascular Patency',\n",
       "   'Veins',\n",
       "   'Wound Healing'],\n",
       "  'final_decision': 'yes'},\n",
       " '25928148': {'QUESTION': 'Does a systems biology approach identify a regulatory network in parotid acinar cell terminal differentiation?',\n",
       "  'CONTEXTS': ['The transcription factor networks that drive parotid salivary gland progenitor cells to terminally differentiate, remain largely unknown and are vital to understanding the regeneration process.',\n",
       "   'A systems biology approach was taken to measure mRNA and microRNA expression in vivo across acinar cell terminal differentiation in the rat parotid salivary gland. Laser capture microdissection (LCM) was used to specifically isolate acinar cell RNA at times spanning the month-long period of parotid differentiation.',\n",
       "   'Clustering of microarray measurements suggests that expression occurs in four stages. mRNA expression patterns suggest a novel role for Pparg which is transiently increased during mid postnatal differentiation in concert with several target gene mRNAs. 79 microRNAs are significantly differentially expressed across time. Profiles of statistically significant changes of mRNA expression, combined with reciprocal correlations of microRNAs and their target mRNAs, suggest a putative network involving Klf4, a differentiation inhibiting transcription factor, which decreases as several targeting microRNAs increase late in differentiation. The network suggests a molecular switch (involving Prdm1, Sox11, Pax5, miR-200a, and miR-30a) progressively decreases repression of Xbp1 gene transcription, in concert with decreased translational repression by miR-214. The transcription factor Xbp1 mRNA is initially low, increases progressively, and may be maintained by a positive feedback loop with Atf6. Transfection studies show that Xbp1 activates the Mist1 promoter [corrected]. In addition, Xbp1 and Mist1 each activate the parotid secretory protein (Psp) gene, which encodes an abundant salivary protein, and is a marker of terminal differentiation.'],\n",
       "  'LABELS': ['OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'This study identifies novel expression patterns of Pparg, Klf4, and Sox11 during parotid acinar cell differentiation, as well as numerous differentially expressed microRNAs. Network analysis identifies a novel stemness arm, a genetic switch involving transcription factors and microRNAs, and transition to an Xbp1 driven differentiation network. This proposed network suggests key regulatory interactions in parotid gland terminal differentiation.',\n",
       "  'MESHES': ['Acinar Cells',\n",
       "   'Animals',\n",
       "   'Basic Helix-Loop-Helix Transcription Factors',\n",
       "   'Cell Differentiation',\n",
       "   'DNA-Binding Proteins',\n",
       "   'Female',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Parotid Gland',\n",
       "   'Pregnancy',\n",
       "   'RNA, Messenger',\n",
       "   'Rats',\n",
       "   'Regulatory Factor X Transcription Factors',\n",
       "   'Systems Biology',\n",
       "   'Transcription Factors',\n",
       "   'X-Box Binding Protein 1'],\n",
       "  'final_decision': 'yes'},\n",
       " '25928084': {'QUESTION': 'Does systems biology analysis reveal NFAT5 as a novel biomarker and master regulator of inflammatory breast cancer?',\n",
       "  'CONTEXTS': ['Inflammatory breast cancer (IBC) is the most rare and aggressive variant of breast cancer (BC); however, only a limited number of specific gene signatures with low generalization abilities are available and few reliable biomarkers are helpful to improve IBC classification into a molecularly distinct phenotype. We applied a network-based strategy to gain insight into master regulators (MRs) linked to IBC pathogenesis.',\n",
       "   'In-silico modeling and Algorithm for the Reconstruction of Accurate Cellular Networks (ARACNe) on IBC/non-IBC (nIBC) gene expression data (n\\u2009=\\u2009197) was employed to identify novel master regulators connected to the IBC phenotype. Pathway enrichment analysis was used to characterize predicted targets of candidate genes. The expression pattern of the most significant MRs was then evaluated by immunohistochemistry (IHC) in two independent cohorts of IBCs (n\\u2009=\\u200939) and nIBCs (n\\u2009=\\u200982) and normal breast tissues (n\\u2009=\\u200915) spotted on tissue microarrays. The staining pattern of non-neoplastic mammary epithelial cells was used as a normal control.',\n",
       "   'Using in-silico modeling of network-based strategy, we identified three top enriched MRs (NFAT5, CTNNB1 or β-catenin, and MGA) strongly linked to the IBC phenotype. By IHC assays, we found that IBC patients displayed a higher number of NFAT5-positive cases than nIBC (69.2% vs. 19.5%; p-value\\u2009=\\u20092.79 10(-7)). Accordingly, the majority of NFAT5-positive IBC samples revealed an aberrant nuclear expression in comparison with nIBC samples (70% vs. 12.5%; p-value\\u2009=\\u20090.000797). NFAT5 nuclear accumulation occurs regardless of WNT/β-catenin activated signaling in a substantial portion of IBCs, suggesting that NFAT5 pathway activation may have a relevant role in IBC pathogenesis. Accordingly, cytoplasmic NFAT5 and membranous β-catenin expression were preferentially linked to nIBC, accounting for the better prognosis of this phenotype.'],\n",
       "  'LABELS': ['BACKGROUND', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We provide evidence that NFAT-signaling pathway activation could help to identify aggressive forms of BC and potentially be a guide to assignment of phenotype-specific therapeutic agents. The NFAT5 transcription factor might be developed into routine clinical practice as a putative biomarker of IBC phenotype.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Biomarkers, Tumor',\n",
       "   'Cell Cycle',\n",
       "   'Cohort Studies',\n",
       "   'Computational Biology',\n",
       "   'Female',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation, Neoplastic',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Humans',\n",
       "   'Immunohistochemistry',\n",
       "   'Inflammatory Breast Neoplasms',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'Phenotype',\n",
       "   'Systems Biology',\n",
       "   'Transcription Factors',\n",
       "   'beta Catenin'],\n",
       "  'final_decision': 'yes'},\n",
       " '25937632': {'QUESTION': 'Does danHong injection dose-dependently vary amino acid metabolites and metabolic pathways in the treatment of rats with cerebral ischemia?',\n",
       "  'CONTEXTS': ['To determine how the relative amino acid contents and metabolic pathways regulate the pharmacological phenotypes in rats with cerebral ischemia after treatment with varying doses of DanHong injection (DHI).',\n",
       "   'Adult male rats underwent middle cerebral artery occlusion (MCAO), and were injected with DHI (DH-1: 1 mL/kg; DH-2: 2.5 mL/kg; DH-3: 5 mL/kg, and DH-4: 10 mL/kg, iv) daily for 3 d. The neurological deficit score, body weights and infarct volume were assessed. Serum levels of 20 free amino acids were determined using HPLC, and the values were transformed through the quantitative analysis of the amino acids in the serum metabolic spectrum. Multivariate statistical analysis methods (PCA and PLS-DA) and web-based metabolomics tools (MetPa and MetaboAnalyst) were used to analyze the biological data sets for the amino acids.',\n",
       "   'Administration of DHI dose-dependently decreased cerebral infarct volume, and ameliorated neurological deficits. A total of 5, 6, 7 and 7 non-overlapping metabolites were identified in the DH-1, DH-2, DH-3, and DH-4 groups, respectively. Eight metabolites were shared between the DHI groups and the vehicle group. In addition, the serum levels of glutamic acid, aspartic acid and serine increased with increasing DHI dose. A total of 3, 2, 2 and 5 non-overlapping metabolic pathways were identified in the DH-1, DH-2, DH-3 and DH-4 groups, respectively, and glycine, serine, threonine and histidine metabolism were identified as overlapping pathways among the 4 dose groups.'],\n",
       "  'LABELS': ['OBJECTIVE', 'METHODS', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Overlapping and non-overlapping amino acid metabolites and metabolic pathways are associated with the dose-dependent neuroprotective effect of DHI.',\n",
       "  'MESHES': ['Amino Acids',\n",
       "   'Animals',\n",
       "   'Biomarkers',\n",
       "   'Brain',\n",
       "   'Chromatography, High Pressure Liquid',\n",
       "   'Disease Models, Animal',\n",
       "   'Dose-Response Relationship, Drug',\n",
       "   'Drugs, Chinese Herbal',\n",
       "   'Infarction, Middle Cerebral Artery',\n",
       "   'Male',\n",
       "   'Medicine, Chinese Traditional',\n",
       "   'Metabolomics',\n",
       "   'Multivariate Analysis',\n",
       "   'Neuroprotective Agents',\n",
       "   'Phenotype',\n",
       "   'Rats, Sprague-Dawley',\n",
       "   'Systems Biology',\n",
       "   'Systems Integration'],\n",
       "  'final_decision': 'yes'},\n",
       " '21426566': {'QUESTION': 'Do elementary signaling modes predict the essentiality of signal transduction network components?',\n",
       "  'CONTEXTS': ['Understanding how signals propagate through signaling pathways and networks is a central goal in systems biology. Quantitative dynamic models help to achieve this understanding, but are difficult to construct and validate because of the scarcity of known mechanistic details and kinetic parameters. Structural and qualitative analysis is emerging as a feasible and useful alternative for interpreting signal transduction.',\n",
       "   \"In this work, we present an integrative computational method for evaluating the essentiality of components in signaling networks. This approach expands an existing signaling network to a richer representation that incorporates the positive or negative nature of interactions and the synergistic behaviors among multiple components. Our method simulates both knockout and constitutive activation of components as node disruptions, and takes into account the possible cascading effects of a node's disruption. We introduce the concept of elementary signaling mode (ESM), as the minimal set of nodes that can perform signal transduction independently. Our method ranks the importance of signaling components by the effects of their perturbation on the ESMs of the network. Validation on several signaling networks describing the immune response of mammals to bacteria, guard cell abscisic acid signaling in plants, and T cell receptor signaling shows that this method can effectively uncover the essentiality of components mediating a signal transduction process and results in strong agreement with the results of Boolean (logical) dynamic models and experimental observations.\"],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'This integrative method is an efficient procedure for exploratory analysis of large signaling and regulatory networks where dynamic modeling or experimental tests are impractical. Its results serve as testable predictions, provide insights into signal transduction and regulatory mechanisms and can guide targeted computational or experimental follow-up studies. The source codes for the algorithms developed in this study can be found at http://www.phys.psu.edu/~ralbert/ESM.',\n",
       "  'MESHES': ['Abscisic Acid',\n",
       "   'Computational Biology',\n",
       "   'Metabolic Networks and Pathways',\n",
       "   'Models, Biological',\n",
       "   'Plant Leaves',\n",
       "   'Receptors, Antigen, T-Cell',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '25217033': {'QUESTION': 'Do a systems approach identifies co-signaling molecules of early growth response 1 transcription factor in immobilization stress?',\n",
       "  'CONTEXTS': ['Adaptation to stress is critical for survival. The adrenal medulla, the major source of epinephrine, plays an important role in the development of the hyperadenergic state and increased risk for stress associated disorders, such as hypertension and myocardial infarction. The transcription factor Egr1 plays a central role in acute and repeated stress, however the complexity of the response suggests that other transcription factor pathways might be playing equally important roles during acute and repeated stress. Therefore, we sought to discover such factors by applying a systems approach.',\n",
       "   'Using microarrays and network analysis we show here for the first time that the transcription factor signal transducer and activator of transcription 3 (Stat3) gene is activated in acute stress whereas the prolactin releasing hormone (Prlh11) and chromogranin B (Chgb) genes are induced in repeated immobilization stress and that along with Egr1 may be critical mediators of the stress response.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our results suggest possible involvement of Stat3 and Prlh1/Chgb up-regulation in the transition from short to repeated stress activation.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Chromogranin B',\n",
       "   'Early Growth Response Protein 1',\n",
       "   'Immobilization',\n",
       "   'Microarray Analysis',\n",
       "   'Prolactin-Releasing Hormone',\n",
       "   'Rats',\n",
       "   'STAT3 Transcription Factor',\n",
       "   'Signal Transduction',\n",
       "   'Stress, Physiological',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '26438393': {'QUESTION': 'Does temporal transcriptional response to latency reversing agents identify specific factors regulating HIV-1 viral transcriptional switch?',\n",
       "  'CONTEXTS': ['Latent HIV-1 reservoirs are identified as one of the major challenges to achieve HIV-1 cure. Currently available strategies are associated with wide variability in outcomes both in patients and CD4(+) T cell models. This underlines the critical need to develop innovative strategies to predict and recognize ways that could result in better reactivation and eventual elimination of latent HIV-1 reservoirs.'],\n",
       "  'LABELS': ['BACKGROUND'],\n",
       "  'LONG_ANSWER': 'In this study, we combined genome wide transcriptome datasets post activation with Systems Biology approach (Signaling and Dynamic Regulatory Events Miner, SDREM analyses) to reconstruct a dynamic signaling and regulatory network involved in reactivation mediated by specific activators using a latent cell line. This approach identified several critical regulators for each treatment, which were confirmed in follow-up validation studies using small molecule inhibitors. Results indicate that signaling pathways involving JNK and related factors as predicted by SDREM are essential for virus reactivation by suberoylanilide hydroxamic acid. ERK1/2 and NF-κB pathways have the foremost role in reactivation with prostratin and TNF-α, respectively. JAK-STAT pathway has a central role in HIV-1 transcription. Additional evaluation, using other latent J-Lat cell clones and primary T cell model, also confirmed that many of the cellular factors associated with latency reversing agents are similar, though minor differences are identified. JAK-STAT and NF-κB related pathways are critical for reversal of HIV-1 latency in primary resting T cells.',\n",
       "  'MESHES': ['CD4-Positive T-Lymphocytes',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation, Viral',\n",
       "   'HIV-1',\n",
       "   'Humans',\n",
       "   'Hydroxamic Acids',\n",
       "   'Jurkat Cells',\n",
       "   'Male',\n",
       "   'Phorbol Esters',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology',\n",
       "   'Tumor Necrosis Factor-alpha',\n",
       "   'Virus Activation',\n",
       "   'Virus Latency',\n",
       "   'Vorinostat'],\n",
       "  'final_decision': 'yes'},\n",
       " '23945349': {'QUESTION': 'Do new network topology approaches reveal differential correlation patterns in breast cancer?',\n",
       "  'CONTEXTS': ['Analysis of genome-wide data is often carried out using standard methods such as differential expression analysis, clustering analysis and heatmaps. Beyond that, differential correlation analysis was suggested to identify changes in the correlation patterns between disease states. The detection of differential correlation is a demanding task, as the number of entries in the gene-by-gene correlation matrix is large. Currently, there is no gold standard for the detection of differential correlation and statistical validation.',\n",
       "   'We developed two untargeted algorithms (DCloc and DCglob) that identify differential correlation patterns by comparing the local or global topology of correlation networks. Construction of networks from correlation structures requires fixing of a correlation threshold. Instead of a single cutoff, the algorithms systematically investigate a series of correlation thresholds and permit to detect different kinds of correlation changes at the same level of significance: strong changes of a few genes and moderate changes of many genes. Comparing the correlation structure of 208 ER- breast carcinomas and 208 ER+ breast carcinomas, DCloc detected 770 differentially correlated genes with a FDR of 12.8%, while DCglob detected 630 differentially correlated genes with a FDR of 12.1%. In two-fold cross-validation, the reproducibility of the list of the top 5% differentially correlated genes in 140 ER- tumors and in 140 ER+ tumors was 49% for DCloc and 33% for DCglob.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We developed two correlation network topology based algorithms for the detection of differential correlations in different disease states. Clusters of differentially correlated genes could be interpreted biologically and included the marker genes hydroxyprostaglandin dehydrogenase (PGDH) and acyl-CoA synthetase medium chain 1 (ACSM1) of invasive apocrine carcinomas that were differentially correlated, but not differentially expressed. Using random subsampling and cross-validation, DCloc and DCglob were shown to identify specific and reproducible lists of differentially correlated genes.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Breast Neoplasms',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Genes, Neoplasm',\n",
       "   'Humans',\n",
       "   'Reproducibility of Results',\n",
       "   'Systems Biology',\n",
       "   'Time Factors',\n",
       "   'Transcriptome'],\n",
       "  'final_decision': 'yes'},\n",
       " '24908109': {'QUESTION': 'Does a systems biology investigation of neurodegenerative dementia reveal a pivotal role of autophagy?',\n",
       "  'CONTEXTS': ['Neurodegenerative dementia comprises chronic and progressive illnesses with major clinical features represented by progressive and permanent loss of cognitive and mental performance, including impairment of memory and brain functions. Many different forms of neurodegenerative dementia exist, but they are all characterized by death of specific subpopulation of neurons and accumulation of proteins in the brain. We incorporated data from OMIM and primary molecular targets of drugs in the different phases of the drug discovery process to try to reveal possible hidden mechanism in neurodegenerative dementia. In the present study, a systems biology approach was used to investigate the molecular connections among seemingly distinct complex diseases with the shared clinical symptoms of dementia that could suggest related disease mechanisms.',\n",
       "   'Network analysis was applied to characterize an interaction network of disease proteins and drug targets, revealing a major role of metabolism and, predominantly, of autophagy process in dementia and, particularly, in tauopathies. Different phases of the autophagy molecular pathway appear to be implicated in the individual disease pathophysiology and specific drug targets associated to autophagy modulation could be considered for pharmacological intervention. In particular, in view of their centrality and of the direct association to autophagy proteins in the network, PP2A subunits could be suggested as a suitable molecular target for the development of novel drugs.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The present systems biology investigation identifies the autophagy pathway as a central dis-regulated process in neurodegenerative dementia with a prevalent involvement in diseases characterized by tau inclusion and indicates the disease-specific molecules in the pathway that could be considered for therapy.',\n",
       "  'MESHES': ['Autophagy',\n",
       "   'Dementia',\n",
       "   'Molecular Sequence Annotation',\n",
       "   'Molecular Targeted Therapy',\n",
       "   'Signal Transduction',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '21962087': {'QUESTION': 'Does the human metabolic reconstruction Recon 1 direct hypotheses of novel human metabolic functions?',\n",
       "  'CONTEXTS': ['Metabolic network reconstructions formalize our knowledge of metabolism. Gaps in these networks pinpoint regions of metabolism where biological components and functions are \"missing.\" At the same time, a major challenge in the post genomic era involves characterisation of missing biological components to complete genome annotation.',\n",
       "   'We used the human metabolic network reconstruction RECON 1 and established constraint-based modelling tools to uncover novel functions associated with human metabolism. Flux variability analysis identified 175 gaps in RECON 1 in the form of blocked reactions. These gaps were unevenly distributed within metabolic pathways but primarily found in the cytosol and often caused by compounds whose metabolic fate, rather than production, is unknown. Using a published algorithm, we computed gap-filling solutions comprised of non-organism specific metabolic reactions capable of bridging the identified gaps. These candidate solutions were found to be dependent upon the reaction environment of the blocked reaction. Importantly, we showed that automatically generated solutions could produce biologically realistic hypotheses of novel human metabolic reactions such as of the fate of iduronic acid following glycan degradation and of N-acetylglutamate in amino acid metabolism.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'The results demonstrate how metabolic models can be utilised to direct hypotheses of novel metabolic functions in human metabolism; a process that we find is heavily reliant upon manual curation and biochemical insight. The effectiveness of a systems approach for novel biochemical pathway discovery in mammals is demonstrated and steps required to tailor future gap filling algorithms to mammalian metabolic networks are proposed.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Computational Biology',\n",
       "   'Humans',\n",
       "   'Metabolic Networks and Pathways',\n",
       "   'Models, Biological',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '21627852': {'QUESTION': 'Is hierarchical cluster-based partial least squares regression ( HC-PLSR ) an efficient tool for metamodelling of nonlinear dynamic models?',\n",
       "  'CONTEXTS': ['Deterministic dynamic models of complex biological systems contain a large number of parameters and state variables, related through nonlinear differential equations with various types of feedback. A metamodel of such a dynamic model is a statistical approximation model that maps variation in parameters and initial conditions (inputs) to variation in features of the trajectories of the state variables (outputs) throughout the entire biologically relevant input space. A sufficiently accurate mapping can be exploited both instrumentally and epistemically. Multivariate regression methodology is a commonly used approach for emulating dynamic models. However, when the input-output relations are highly nonlinear or non-monotone, a standard linear regression approach is prone to give suboptimal results. We therefore hypothesised that a more accurate mapping can be obtained by locally linear or locally polynomial regression. We present here a new method for local regression modelling, Hierarchical Cluster-based PLS regression (HC-PLSR), where fuzzy C-means clustering is used to separate the data set into parts according to the structure of the response surface. We compare the metamodelling performance of HC-PLSR with polynomial partial least squares regression (PLSR) and ordinary least squares (OLS) regression on various systems: six different gene regulatory network models with various types of feedback, a deterministic mathematical model of the mammalian circadian clock and a model of the mouse ventricular myocyte function.',\n",
       "   'Our results indicate that multivariate regression is well suited for emulating dynamic models in systems biology. The hierarchical approach turned out to be superior to both polynomial PLSR and OLS regression in all three test cases. The advantage, in terms of explained variance and prediction accuracy, was largest in systems with highly nonlinear functional relationships and in systems with positive feedback loops.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'HC-PLSR is a promising approach for metamodelling in systems biology, especially for highly nonlinear or non-monotone parameter to phenotype maps. The algorithm can be flexibly adjusted to suit the complexity of the dynamic model behaviour, inviting automation in the metamodelling of complex systems.',\n",
       "  'MESHES': ['Algorithms',\n",
       "   'Animals',\n",
       "   'Cluster Analysis',\n",
       "   'Computational Biology',\n",
       "   'Heart Ventricles',\n",
       "   'Humans',\n",
       "   'Least-Squares Analysis',\n",
       "   'Mice',\n",
       "   'Models, Theoretical',\n",
       "   'Multivariate Analysis',\n",
       "   'Muscle Cells',\n",
       "   'Phenotype',\n",
       "   'Regression Analysis',\n",
       "   'Reproducibility of Results',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '21619705': {'QUESTION': 'Does large-scale analysis of expression signatures reveal hidden links among diverse cellular processes?',\n",
       "  'CONTEXTS': ['Cells must respond to various perturbations using their limited available gene repertoires. In order to study how cells coordinate various responses, we conducted a comprehensive comparison of 1,186 gene expression signatures (gene lists) associated with various genetic and chemical perturbations.',\n",
       "   'We identified 7,419 statistically significant overlaps between various published gene lists. Most (80%) of the overlaps can be represented by a highly connected network, a \"molecular signature map,\" that highlights the correlation of various expression signatures. By dissecting this network, we identified sub-networks that define clusters of gene sets related to common biological processes (cell cycle, immune response, etc). Examination of these sub-networks has confirmed relationships among various pathways and also generated new hypotheses. For example, our result suggests that glutamine deficiency might suppress cellular growth by inhibiting the MYC pathway. Interestingly, we also observed 1,369 significant overlaps between a set of genes upregulated by factor X and a set of genes downregulated by factor Y, suggesting a repressive interaction between X and Y factors.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our results suggest that molecular-level responses to diverse chemical and genetic perturbations are heavily interconnected in a modular fashion. Also, shared molecular pathways can be identified by comparing newly defined gene expression signatures with databases of previously published gene expression signatures.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Cell Cycle',\n",
       "   'Computational Biology',\n",
       "   'Databases, Genetic',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation',\n",
       "   'Glutamine',\n",
       "   'Humans',\n",
       "   'Immune System',\n",
       "   'Mice',\n",
       "   'Models, Biological',\n",
       "   'Models, Genetic',\n",
       "   'Proto-Oncogene Proteins c-myc',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '24004452': {'QUESTION': 'Does systems analysis reveal a transcriptional reversal of the mesenchymal phenotype induced by SNAIL-inhibitor GN-25?',\n",
       "  'CONTEXTS': ['HMLEs (HMLE-SNAIL and Kras-HMLE, Kras-HMLE-SNAIL pairs) serve as excellent model system to interrogate the effect of SNAIL targeted agents that reverse epithelial-to-mesenchymal transition (EMT). We had earlier developed a SNAIL-p53 interaction inhibitor (GN-25) that was shown to suppress SNAIL function. In this report, using systems biology and pathway network analysis, we show that GN-25 could cause reversal of EMT leading to mesenchymal-to-epithelial transition (MET) in a well-recognized HMLE-SNAIL and Kras-HMLE-SNAIL models.',\n",
       "   'GN-25 induced MET was found to be consistent with growth inhibition, suppression of spheroid forming capacity and induction of apoptosis. Pathway network analysis of mRNA expression using microarrays from GN-25 treated Kras-HMLE-SNAIL cells showed an orchestrated global re-organization of EMT network genes. The expression signatures were validated at the protein level (down-regulation of mesenchymal markers such as TWIST1 and TWIST2 that was concurrent with up-regulation of epithelial marker E-Cadherin), and RNAi studies validated SNAIL dependent mechanism of action of the drug. Most importantly, GN-25 modulated many major transcription factors (TFs) such as inhibition of oncogenic TFs Myc, TBX2, NR3C1 and led to enhancement in the expression of tumor suppressor TFs such as SMAD7, DD1T3, CEBPA, HOXA5, TFEB, IRF1, IRF7 and XBP1, resulting in MET as well as cell death.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Our systems and network investigations provide convincing pre-clinical evidence in support of the clinical application of GN-25 for the reversal of EMT and thereby reducing cancer cell aggressiveness.',\n",
       "  'MESHES': ['Apoptosis',\n",
       "   'Cell Line',\n",
       "   'Cell Proliferation',\n",
       "   'Epithelial-Mesenchymal Transition',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Gene Silencing',\n",
       "   'Humans',\n",
       "   'Mesoderm',\n",
       "   'Naphthoquinones',\n",
       "   'Phenotype',\n",
       "   'RNA, Small Interfering',\n",
       "   'Snail Family Transcription Factors',\n",
       "   'Spheroids, Cellular',\n",
       "   'Systems Biology',\n",
       "   'Transcription Factors',\n",
       "   'Transcription, Genetic',\n",
       "   'Transfection'],\n",
       "  'final_decision': 'yes'},\n",
       " '21920041': {'QUESTION': 'Does computational modelling elucidate the mechanism of ciliary regulation in health and disease?',\n",
       "  'CONTEXTS': ['Ciliary dysfunction leads to a number of human pathologies, including primary ciliary dyskinesia, nephronophthisis, situs inversus pathology or infertility. The mechanism of cilia beating regulation is complex and despite extensive experimental characterization remains poorly understood. We develop a detailed systems model for calcium, membrane potential and cyclic nucleotide-dependent ciliary motility regulation.',\n",
       "   'The model describes the intimate relationship between calcium and potassium ionic concentrations inside and outside of cilia with membrane voltage and, for the first time, describes a novel type of ciliary excitability which plays the major role in ciliary movement regulation. Our model describes a mechanism that allows ciliary excitation to be robust over a wide physiological range of extracellular ionic concentrations. The model predicts the existence of several dynamic modes of ciliary regulation, such as the generation of intraciliary Ca2+ spike with amplitude proportional to the degree of membrane depolarization, the ability to maintain stable oscillations, monostable multivibrator regimes, all of which are initiated by variability in ionic concentrations that translate into altered membrane voltage.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'Computational investigation of the model offers several new insights into the underlying molecular mechanisms of ciliary pathologies. According to our analysis, the reported dynamic regulatory modes can be a physiological reaction to alterations in the extracellular environment. However, modification of the dynamic modes, as a result of genetic mutations or environmental conditions, can cause a life threatening pathology.',\n",
       "  'MESHES': ['Calcium Channels',\n",
       "   'Calcium Signaling',\n",
       "   'Cilia',\n",
       "   'Disease',\n",
       "   'Membrane Potentials',\n",
       "   'Models, Biological',\n",
       "   'Patch-Clamp Techniques',\n",
       "   'Potassium',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '21849042': {'QUESTION': 'Do a systems approach to identifying correlated gene targets for the loss of colour pigmentation in plants?',\n",
       "  'CONTEXTS': ['The numerous diverse metabolic pathways by which plant compounds can be produced make it difficult to predict how colour pigmentation is lost for different tissues and plants. This study employs mathematical and in silico methods to identify correlated gene targets for the loss of colour pigmentation in plants from a whole cell perspective based on the full metabolic network of Arabidopsis. This involves extracting a self-contained flavonoid subnetwork from the AraCyc database and calculating feasible metabolic routes or elementary modes (EMs) for it. Those EMs leading to anthocyanin compounds are taken to constitute the anthocyanin biosynthetic pathway (ABP) and their interplay with the rest of the EMs is used to study the minimal cut sets (MCSs), which are different combinations of reactions to block for eliminating colour pigmentation. By relating the reactions to their corresponding genes, the MCSs are used to explore the phenotypic roles of the ABP genes, their relevance to the ABP and the impact their eliminations would have on other processes in the cell.',\n",
       "   'Simulation and prediction results of the effect of different MCSs for eliminating colour pigmentation correspond with existing experimental observations. Two examples are: i) two MCSs which require the simultaneous suppression of genes DFR and ANS to eliminate colour pigmentation, correspond to observational results of the same genes being co-regulated for eliminating floral pigmentation in Aquilegia and; ii) the impact of another MCS requiring CHS suppression, corresponds to findings where the suppression of the early gene CHS eliminated nearly all flavonoids but did not affect the production of volatile benzenoids responsible for floral scent.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'From the various MCSs identified for eliminating colour pigmentation, several correlate to existing experimental observations, indicating that different MCSs are suitable for different plants, different cells, and different conditions and could also be related to regulatory genes. Being able to correlate the predictions with experimental results gives credence to the use of these mathematical and in silico analyses methods in the design of experiments. The methods could be used to prioritize target enzymes for different objectives to achieve desired outcomes, especially for less understood pathways.',\n",
       "  'MESHES': ['Anthocyanins',\n",
       "   'Aquilegia',\n",
       "   'Arabidopsis',\n",
       "   'Biosynthetic Pathways',\n",
       "   'Flavonoids',\n",
       "   'Genes, Regulator',\n",
       "   'Metabolic Networks and Pathways',\n",
       "   'Pigmentation',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'},\n",
       " '21861884': {'QUESTION': 'Are biological interaction networks conserved at the module level?',\n",
       "  'CONTEXTS': ['Orthologous genes are highly conserved between closely related species and biological systems often utilize the same genes across different organisms. However, while sequence similarity often implies functional similarity, interaction data is not well conserved even for proteins with high sequence similarity. Several recent studies comparing high throughput data including expression, protein-protein, protein-DNA, and genetic interactions between close species show conservation at a much lower rate than expected.',\n",
       "   'In this work we collected comprehensive high-throughput interaction datasets for four model organisms (S. cerevisiae, S. pombe, C. elegans, and D. melanogaster) and carried out systematic analyses in order to explain the apparent lower conservation of interaction data when compared to the conservation of sequence data. We first showed that several previously proposed hypotheses only provide a limited explanation for such lower conservation rates. We combined all interaction evidences into an integrated network for each species and identified functional modules from these integrated networks. We then demonstrate that interactions that are part of functional modules are conserved at much higher rates than previous reports in the literature, while interactions that connect between distinct functional modules are conserved at lower rates.'],\n",
       "  'LABELS': ['BACKGROUND', 'RESULTS'],\n",
       "  'LONG_ANSWER': 'We show that conservation is maintained between species, but mainly at the module level. Our results indicate that interactions within modules are much more likely to be conserved than interactions between proteins in different modules. This provides a network based explanation to the observed conservation rates that can also help explain why so many biological processes are well conserved despite the lower levels of conservation for the interactions of proteins participating in these processes.Accompanying website: http://www.sb.cs.cmu.edu/CrossSP.',\n",
       "  'MESHES': ['Animals',\n",
       "   'Caenorhabditis elegans',\n",
       "   'Conserved Sequence',\n",
       "   'Drosophila melanogaster',\n",
       "   'Gene Regulatory Networks',\n",
       "   'Multiprotein Complexes',\n",
       "   'Protein Interaction Maps',\n",
       "   'Saccharomyces cerevisiae',\n",
       "   'Schizosaccharomyces',\n",
       "   'Species Specificity',\n",
       "   'Systems Biology'],\n",
       "  'final_decision': 'yes'}}"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "{k : v for k, v in data.items() if 'systems biology' in [x.lower() for x in v['MESHES']]}"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "exponential-chain",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
